<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Coupling Diet and Pollen Metabarcoding with Field Surveys to Reveal Plant - pollinator Interactions</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="interaction_methodologies_files/libs/clipboard/clipboard.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="interaction_methodologies_files/libs/quarto-html/popper.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/anchor.min.js"></script>
<link href="interaction_methodologies_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="interaction_methodologies_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-bb462d781dde1847d9e3ccf7736099dd.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent quarto-light">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Coupling Diet and Pollen Metabarcoding with Field Surveys to Reveal Plant - pollinator Interactions</h1>
<p class="subtitle lead">Christian Gostout <sup>1</sup>, Luis J. Chueca <sup>1</sup>, Jon Poza, Xabier Salgado-Irazabal<sup>1</sup>, Estefanía Tobajas<sup>1</sup>, Jennifer Rose<sup>1</sup>, Brais Hermosilla<sup>1</sup>, Marion Donald<sup>2</sup>, Manpreet Dhami<sup>2</sup>, Ainhoa Magrach<sup>1,3</sup> <sup>1</sup>BC3 Basque Centre for Climate Change, Leioa, Spain, <sup>2</sup>Bioeconomy Science Institute, Lincoln, New Zealand, <sup>3</sup>IKERBASQUE, Basque Foundation for Science, Bilbao, Spain</p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><strong>Key-words:</strong> interaction, metabarcoding, network, pollinator</p>
<p><strong>Abstract</strong></p>
<p>Our understanding of plant-pollinator interaction networks hinges on the methods used to describe their nodes and links. Currently, most networks are built from field observations that may overlook many consumer–resource links. Further, interaction networks lack descriptive links that characterize interaction types and outcomes. Towards a more complete approach for building interaction networks, we compare plant interactions from the wild pollinator species, Bombus pascuorum, observed by three methodologies with different implications for interaction outcomes. We compare floral visitation interactions obtained from field observations to plant consumption interactions revealed by metabarcoding of gut contents and pollen transport interactions detected by corbicular pollen loads. Our approach adds functional context to plant–pollinator network links and reveals new interactions. We show that both metabarcoding approaches increase sampling efficiency and reveal links left unobserved by field observations of visitation, highlighting plant taxa that are not pollinator-dependent, yet constitute important dietary resources. Paired with floral diversity surveys, gut content results also reveal seasonal patterns in the spatial extent and functional diversity included in forage, which other methodologies fail to demonstrate. Metabarcoding data analyzed at the individual specimen level further reveal heterogeneity in plant resource use between pollen transport and consumption. Metabarcoding methodologies capture greater spatial, temporal, and taxonomic ranges, while field observations provide validating datasets with taxonomic precision.Our results show that integrating visitation, transport, and consumption data changes network topology and the roles of plant nodes, offering a more nuanced and complete map of interactions with clearer priorities for management. We advocate for defining links explicitly by their functions and combining methods to account for hidden structure in ecological networks.</p>
<p><strong>Introduction</strong></p>
<p>Pollination is a critical ecosystem service that is currently threatened by different global changes, including habitat loss, intensifying agriculture, pathogens, and invasive species <span class="citation" data-cites="klein2006">(<a href="#ref-klein2006" role="doc-biblioref">Klein et al., 2006</a>)</span>. Pollinators critically support the reproduction of 94% of wild flowering plants and 75% of crop species <span class="citation" data-cites="vanbergen2013">(<a href="#ref-vanbergen2013" role="doc-biblioref">Vanbergen &amp; Insect Pollinators Initiative, 2013</a>)</span>, contributing to 35% of global food production <span class="citation" data-cites="klein2006">(<a href="#ref-klein2006" role="doc-biblioref">Klein et al., 2006</a>)</span>. However, our understanding of this topic is incomplete given that, historically, the methodological approach to studying plant-pollinator interactions has been biased towards the plant side of interactions <span class="citation" data-cites="evans2020 bosch2009">(<a href="#ref-bosch2009" role="doc-biblioref">Bosch et al., 2009</a>; <a href="#ref-evans2020" role="doc-biblioref">Evans &amp; Kitson, 2020</a>)</span>. As a consequence, the well-established relationship between pollinator diversity and the productivity of plant communities <span class="citation" data-cites="artamendi2025 woodcock2019 katumo2022">(<a href="#ref-artamendi2025" role="doc-biblioref">Artamendi et al., 2025</a>; <a href="#ref-katumo2022" role="doc-biblioref">Katumo et al., 2022</a>; <a href="#ref-woodcock2019" role="doc-biblioref">Woodcock et al., 2019</a>)</span> lacks an equally developed mirrored perspective, describing the floral diversity that supports pollinator populations.&nbsp;</p>
<p>Network theory provides a useful framework to summarize patterns of plant–pollinator interaction <span class="citation" data-cites="burkle2011">(<a href="#ref-burkle2011" role="doc-biblioref">Burkle &amp; Alarcón, 2011</a>)</span>, but the strong influence of the interaction types that define links on the scope of networks has yet to be accounted for in most studies. Existing methodologies for reconstructing interaction networks tend to emphasize structural patterns, while overlooking the functional outcomes of interactions that are critical for understanding how plant communities support pollinators <span class="citation" data-cites="quintero2022">(<a href="#ref-quintero2022" role="doc-biblioref">Quintero et al., 2022</a>)</span>. In eusocial bees, for example, plant interactions may have several outcomes. Bees consume plant material, including pollen, nectar, or even plant tissue <span class="citation" data-cites="vaudo2015 pashalidou2020">(<a href="#ref-pashalidou2020" role="doc-biblioref">Pashalidou et al., 2020</a>; <a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>. They also collect pollen on their corbicula for transport to the nest for feeding drones and larvae <span class="citation" data-cites="leach2018 vaudo2015">(<a href="#ref-leach2018" role="doc-biblioref">Leach &amp; Drummond, 2018</a>; <a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>. Finally, visitation of the reproductive parts of flowers can have various outcomes for both the plant and pollinator, including pollination and pathogen transfer (Lignon et al.&nbsp;2024). Interaction networks generally represent only one of these outcomes, although each is important to understanding how plant taxa support pollinators.</p>
<p>The importance of different outcomes in plant-pollinator interactions becomes clear when considering the biodiversity necessary to support pollinators across life stages. Because the resources needed for foraging adult pollinator nutrition can be different from those needed at the larval stage, or by other colony members <span class="citation" data-cites="leach2018 vaudo2015">(<a href="#ref-leach2018" role="doc-biblioref">Leach &amp; Drummond, 2018</a>; <a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>, transported pollen may not completely represent the interactions necessary to sustain adult pollinator diets. This is especially true for bumblebees (<em>Bombus spp.</em>), which are able to evaluate pollen resource quality to make discerning forage choices <span class="citation" data-cites="leonhardt2012 timberlake2024">(<a href="#ref-leonhardt2012" role="doc-biblioref">Leonhardt &amp; Blüthgen, 2012</a>; <a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>)</span>. Bumblebees make trial-and-error floral visits in order to find adequate forage <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>)</span>, which may result in pollen transport without consumption. Conversely, consumption, or simply visitation, may occur without resulting in transport <span class="citation" data-cites="popic2012">(<a href="#ref-popic2012" role="doc-biblioref">Popic et al., 2012</a>)</span>. Accounting for different interaction outcomes, such as visitation, transport, and consumption, is a critical next step in representing the network of plant diversity used by pollinators.</p>
<p>Shifting network studies to incorporate the pollinator perspective and leveraging the contributions of different methodologies can produce a more complete image of interaction networks. Research based on microscopy and molecular analyses of pollen load samples sourced from insect specimens can identify greater plant species diversity within interaction networks compared to studies based solely on field observations of floral visitation <span class="citation" data-cites="baksay2022 bosch2009">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay et al., 2022</a>; <a href="#ref-bosch2009" role="doc-biblioref">Bosch et al., 2009</a>)</span>. Additionally, studies adopting a pollinator-centered view have revealed greater detail in forage preference trends, such as how pollinators use forage quality or quantity-based strategies <span class="citation" data-cites="timberlake2024 selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>; <a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>)</span>, seasonal changes <span class="citation" data-cites="leponiemi2023">(<a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi et al., 2023</a>)</span>, life cycle timing, and metabolic specialization <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>.</p>
<p>Genetic tools can detect plant-pollinator interactions that may be unobserved in pollen microscopy and traditional field surveys <span class="citation" data-cites="lowe2022 arstingstall2021 bell2016">Pornon et al. (<a href="#ref-pornon2017" role="doc-biblioref">2017</a>)</span>, and target specific interaction types. Amplicon sequence metabarcoding of pollen samples complements the visitation interactions observed by field studies <span class="citation" data-cites="arstingstall2021 bell2017">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>; <a href="#ref-bell2017" role="doc-biblioref">K. L. Bell et al., 2017</a>)</span>, increasing species detection by 9 - 144% <span class="citation" data-cites="milla2022 baksay2022 smart2017">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay et al., 2022</a>; <a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>; <a href="#ref-smart2017" role="doc-biblioref">Smart et al., 2017</a>)</span> and network sampling completeness up to 30%, while reducing exaggeration of specialization <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>)</span> and revealing interactions beyond the traditionally surveyed floral community <span class="citation" data-cites="milla2022 devere2017">(<a href="#ref-devere2017" role="doc-biblioref">de Vere et al., 2017</a>; <a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>)</span>. Advances in the reliability and accessibility of amplicon sequencing have made these approaches more feasible for studying plant-pollinator interactions. Field surveys of visitation can now be effectively complemented by genetic tools <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>)</span> targeting specific interaction types, enhancing our understanding of interaction diversity.</p>
<p>Most studies applying metabarcoding to pollinator-sourced samples for constructing interaction networks analyze the external pollen loads of bees or pollen stored in nest reserves of honey and beebread <span class="citation" data-cites="selva2024 devriese2024 leponiemi2023 baksay2022 leontidou2021">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay et al., 2022</a>; <a href="#ref-devriese2024" role="doc-biblioref">Devriese et al., 2024</a>; <a href="#ref-leontidou2021" role="doc-biblioref">Leontidou et al., 2021</a>; <a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi et al., 2023</a>; <a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>)</span>, despite limitations of these sampling targets. Pollen in these samples can come from the environment, even including accumulation of windborne material <span class="citation" data-cites="negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri et al., 2015</a>)</span>. To account for this, past studies have ignored detections of wind pollinated taxa&nbsp;<span class="citation" data-cites="tanaka2020">Pornon et al. (<a href="#ref-pornon2017" role="doc-biblioref">2017</a>)</span>, although this may introduce bias to results, given that many plant taxa have partial identities as wind or insect pollinated taxa <span class="citation" data-cites="saunders2018">(<a href="#ref-saunders2018" role="doc-biblioref">Saunders, 2018</a>)</span> A more fundamental issue with externally carried pollen and nest reserves is present in their restricted ability to represent interaction types. Studies of external pollen carried by eusocial bees, for example, have generally sequenced the DNA of pollen from the corbicula (e.g. <span class="citation" data-cites="shi2025">Shi et al. (<a href="#ref-shi2025" role="doc-biblioref">2025</a>)</span>). Corbicular pollen provides an easily obtained sample, containing a mixture of pollen collected for transport to the nest for brood feeding <span class="citation" data-cites="leach2018 vaudo2015">(<a href="#ref-leach2018" role="doc-biblioref">Leach &amp; Drummond, 2018</a>; <a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>, which only directly observes interactions where pollen is transported <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>)</span>. Given the role of this pollen in bees’ life cycles, it is easy to overstep the interpretative capacity of these sample types when characterizing forage networks to describe diet, or successful pollination interactions.</p>
<p>Pollinator intestinal tracts (hereafter: guts) represent an additional source for observing interactions, specifically those related to consumption of pollen and other plant material <span class="citation" data-cites="li2025 haag2023 mayr2021">(<a href="#ref-haag2023" role="doc-biblioref">Haag et al., 2023</a>; <a href="#ref-li2025" role="doc-biblioref">Li et al., 2025</a>; <a href="#ref-mayr2021" role="doc-biblioref">Mayr et al., 2021</a>)</span>. Plant DNA detected in gut contents can reveal interactions with consumption as the exclusive outcome, which, aside from flower visits, can include nectar robbing <span class="citation" data-cites="popic2012">(<a href="#ref-popic2012" role="doc-biblioref">Popic et al., 2012</a>)</span> and plant damage <span class="citation" data-cites="pashalidou2020">(<a href="#ref-pashalidou2020" role="doc-biblioref">Pashalidou et al., 2020</a>)</span>. The gut content approach can also account for environmental contamination in external pollen and nest stores by highlighting oversights resulting from the exclusion of interactions with the anemophilous and partially-anemophilous plant taxa in external pollen studies. There is an accumulating body of evidence supporting the idea that pollinators must regularly search across functional groups of the plant community to meet their nutritional needs <span class="citation" data-cites="ibiyemi2025 selva2024 milla2022 wood2022 timberlake2024 devere2017 tanaka2020 terrell1984 pojar1973">(<a href="#ref-devere2017" role="doc-biblioref">de Vere et al., 2017</a>; <a href="#ref-ibiyemi2025" role="doc-biblioref">Ibiyemi et al., 2025</a>; <a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>; <a href="#ref-pojar1973" role="doc-biblioref">Pojar, 1973</a>; <a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>; <a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al., 2020</a>; <a href="#ref-terrell1984" role="doc-biblioref">Terrell &amp; Batra, 1984</a>; <a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>; <a href="#ref-wood2022" role="doc-biblioref">Wood et al., 2022</a>)</span>, although little attention has been given to these observations as a potentially important part of plant-pollinator networks <span class="citation" data-cites="saunders2018">(<a href="#ref-saunders2018" role="doc-biblioref">Saunders, 2018</a>)</span>. This understudied component of pollinator forage together with the surprising lack of genetic analyses of pollinator gut contents, represents a clear knowledge gap and an opportunity to uncover finer detail in pollinator interaction networks.</p>
<p>Our objective is to determine whether a combined methodological approach can provide further insights into pollinator forage ecology and plant-pollinator interaction networks by expanding interaction detections and providing context to network links. We assess how metabarcoding of pollinator gut contents can complement or challenge the characterization of plant-pollinator interaction networks described by more common methodologies, including field surveys of plant pollinator interactions and external pollen load metabarcoding. To this end, we compare interaction networks constructed from each of these methodologies for a single model pollinator, <em>Bombus pascuorum</em>, an easily identified bumblebee common to most of Europe <span class="citation" data-cites="lecocq2015">(<a href="#ref-lecocq2015" role="doc-biblioref">Lecocq et al., 2015</a>)</span>. Our focus on a single pollinator species holds pollinator identity constant and attributes differences in network structure to methodology, rather than to variation among pollinator species. We hypothesize that the consumption interactions detected in gut metabarcoding will include a network of plant taxa distinct from those detected by other methodologies. Although we expect overlap between networks constructed by different methodologies, we expect to observe previously overlooked interaction network structure, including new links and significance of network links. Ideally, the resulting combination of observations will generate a network that will elevate our capacity to detect meaningful plant-pollinator interactions, and learn more about interaction types and implications for pollinator health.</p>
<p><strong>Methods</strong></p>
<p>Our sample collection was conducted in Gorbeia Natural Park, a protected area in Spain. Within Gorbeia, we selected 16 sampling sites located within the mixed zones of meadows and shrublands found at higher elevations within the park.&nbsp; We conducted fieldwork from early April to the end of July 2023 covering the main flowering period and peak annual pollinator activity in Gorbeia. On each sampling day during this timeframe, we visited field sites in pairs. Sampling days were organized into six periods, in which we sampled each site pair once per period. We conducted three types of surveys during daily peaks of pollinator activity, including floral diversity surveys (“flower counts”), interaction transect surveys, and Bombus pascuorum specimen collection for amplicon sequencing analyses.</p>
<p><em>Interaction transects and floral resource availability surveys</em>&nbsp;</p>
<p>We used the one 250 m transect at each site for both interaction transect and flower count surveys, recording observations within ~2 m of the transect line. Interaction surveys were conducted three times per day, each lasting 1 h. All insects observed contacting the reproductive parts of herbaceous flowers within the transect were recorded; for this study, we retained only Bombus pascuorum interaction data. Surveys were spaced by ~2 hours (~11:00, ~13:00, ~15:00), and transects were walked at a constant pace to cover the full length within an hour. For each site and sampling period, one flower count was conducted by recording all of the flowering herbaceous species within the transects.</p>
<p><em>Bombus pascuorum specimens</em></p>
<p>For every period visit at each site, we collected up to five <em>B. pascuorum</em> specimens for molecular analyses (N = 126). We brought specimens back from the field and froze them at -20°C until processed. In the lab, we extracted the entire gut and honey stomach of <em>B. pascuorum</em> individuals. Additionally, if present, we collected pollen pellets from the corbicula of specimens into sterile 1.5 mL centrifuge tubes. Pollen samples were stored individually by specimen sample at -20°C.</p>
<p>Gut Content DNA extraction</p>
<p>Genomic DNA was extracted from <em>B. pascuorum</em> guts using the NucleoSpin® 96 Soil kit (Macherey-Nagel, Düren, Germany) and amplified in duplicate using the DFD forward and ASDFAS reverse primers. To avoid site and period bias, all samples were randomized using a randomizer program before the DNA extraction. We followed the kit manufacturer protocol, only adjusting centrifuge times to account for the lower maximum velocity of the large centrifuge used to process large sample numbers simultaneously (See Supporting Information). To confirm successful DNA extraction, Nanodrop tests were performed on random samples.</p>
<p><em>DNA extraction from corbicular pollen pellets</em></p>
<p>DNA was extracted from pollen pellets (N = 25) using the Machery-Nagel NucleoSpin 8 Food kit, including additional initial steps recommended by the kit’s supplementary protocol for pollen DNA extraction (See Supporting Information). Qubit fluorometry tests using random samples confirmed successful DNA extractions.</p>
<p><em>Amplicon Sequencing</em>&nbsp;</p>
<p>Our metabarcoding sequence libraries were built by amplifying and sequencing the internal transcribed spacer (ITS2) region of the ribosomal DNA in our extract samples. For all samples, we used existing primers for amplification of the ITS2 region (See Supporting Information). Libraries were sequenced on an Illumina platform to generate paired-end raw reads. We used demultiplexed raw sequence data, with primer and adapter sequences removed, in further bioinformatic analyses.</p>
<p><em>Bioinformatics: taxonomic assignment and contaminant analysis</em></p>
<p>Raw Illumina sequences were processed using the DADA2 bioinformatics pipeline <span class="citation" data-cites="callahan2016">(<a href="#ref-callahan2016" role="doc-biblioref">Callahan et al., 2016</a>)</span>. Taxonomy was added to the ASVs using an existing reference sequence database <span class="citation" data-cites="bell2021">(<a href="#ref-bell2021" role="doc-biblioref">K. Bell, 2021</a>)</span>, which provided reference sequences at the species level for all but 21 of the species present in the study area, all of which were identifiable to the genus level in the database. We removed likely contaminants and misidentified ASVs from our bioinformatics results using a three-step screening process. First, ASVs were analyzed for contaminants using the decontam package in R <span class="citation" data-cites="davis2018">(<a href="#ref-davis2018" role="doc-biblioref">Davis et al., 2018</a>)</span>. Second, we conducted a BLAST search using ITS2 Database <span class="citation" data-cites="ankenbrand2015">(<a href="#ref-ankenbrand2015" role="doc-biblioref">Ankenbrand et al., 2015</a>)</span> to verify taxa that were identified by only one ASV within our results. Finally, the remaining list of taxa was assessed by a local botanist for remaining errors.</p>
<p><em>Statistical analysis</em></p>
<p>We analyzed the results of each methodology together using statistical tools for comparing interaction plant communities across methodology, time, and individual specimens. As an initial broad test of whether the methodologies detected interactions with different plant communities, we used binary presence-absence matrices to compare the communities detected by each methodology on each sampling day. Data were aggregated by sampling day for all sets of observations. Community composition was contrasted using the Raup-Crick dissimilarity index in a PERMANOVA test within the R package, vegan <span class="citation" data-cites="vegan">(<a href="#ref-vegan" role="doc-biblioref">Oksanen et al., 2024</a>)</span>with methodology as the independent variable. Further pairwise comparisons of these data were made by subsetting the dissimilarity matrix used in the first test by each unique methodology pair and using multiple PERMANOVAs to test the pairs. We also used vegan to observe beta dispersal of our data as a further means of understanding PERMANOVA results.</p>
<p>Among our <em>B. pascuorum</em> specimens, 25 provided both pollen and gut samples. Using the data from this subset of samples, we compared the plant communities detected by the two metabarcoding methodologies at the individual sample level without aggregation. As before, Raup-Crick dissimilarity matrices were calculated using binary detection data from pollen and gut detections. PERMANOVA compared both methodologies’ detected communities in strata defined by specimens of sample origin.</p>
<p><em>B. pascuorum - plant interaction Network Metrics</em>&nbsp;</p>
<p>We used interaction frequencies from the three methodologies to build <em>B. pascuorum</em>–plant interaction networks and calculate species-level metrics for plant importance and specialization. Plant importance was the proportion of all <em>B. pascuorum</em> interactions involving a given plant genus. For metabarcoding and pollen-load data, interactions were counted as the number of individual bee samples in which a plant taxon was detected; for observational data, interactions corresponded to recorded visits. Species-level specialization (<em>d’</em>) was calculated following <span class="citation" data-cites="blüthgen2006">Blüthgen et al. (<a href="#ref-blüthgen2006" role="doc-biblioref">2006</a>)</span>, as implemented in the R package bipartite <span class="citation" data-cites="bipartite">(<a href="#ref-bipartite" role="doc-biblioref">Dormann et al., 2009</a>)</span>.</p>
<p>We created a composite interaction network for <em>B. pascuorum</em>, incorporating the data of each methodology and the interaction outcome types as network metadata. Network nodes included <em>B. pascuorum</em> and the list of plant genera detected across the three interaction datasets. Single plant genera were assigned between one and three links corresponding to interaction type, depending on their detection across methodologies.</p>
<hr>
<p><strong>Results</strong></p>
<p><em>Assessment of floral resource use relative to availability</em></p>
<p>Within our flower count surveys we registered a total of 117 flowering herbaceous plant genera across the sampling season, representing the pool of floral resources available to <em>B. pascuorum</em>, which interacted with only a subset of this diversity (Fig. 1). In fact, 39 genera recorded in flower counts were absent from the interaction networks generated by any of the methodologies. Interaction transects revealed interactions with 27 genera (23% of total floral diversity), while gut content and corbicular pollen metabarcoding revealed interactions with 58% and 53% of available taxa, respectively.</p>
<p><em>Comparison of interaction detections by methodology</em>&nbsp;</p>
<p>Both metabarcoding methodologies detected multiple unique taxa (33 taxa for gut contents and 30 for corbicular pollen), while interaction transects did not detect any unique interactions (Fig. 1). The two metabarcoding methodologies shared 83 common plant genera, representing 67% of the total corbicular pollen diversity and 63% of the gut content diversity.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="manuscript_figures/venn.figure.png" class="img-fluid figure-img" width="1500"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 1</strong>: Total diversity and overlap of plant genera observed by four observation methodologies: transect surveys of floral diversity (“flower counts”) and B. pascuorum - flower interactions, and metabarcoding of plant DNA in corbicular pollen and gut contents of B. pascuorum. For the three interaction observation methodologies, the total number of taxa represents the degree of B. pascuorum in the interaction network constructed by the corresponding methodology.</em></p>
<p>Taxonomic diversity varied across sampling periods, revealing distinct temporal patterns in flowering taxa and interactions (Fig. 2). Although floral and interaction diversity increased overall from the first to the last period, flowering taxa peaked in period four, whereas interactions peaked in period six. Metabarcoding consistently detected more taxa than interaction transects, with gut-content metabarcoding outperforming all other methods. In periods one, two, and six—before and after peak flowering—gut metabarcoding detected 59% more taxa than were recorded in flower counts on average, while in periods three to five floral diversity exceeded gut-content diversity.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="manuscript_figures/interaction.diversity.periods.png" class="img-fluid figure-img" width="1006"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 2:</strong> Taxonomic diversity in Bombus pascuorum interaction networks over six sampling periods (April - August, 2023) observed through floral visitation surveys and ITS2 metabarcoding of DNA extracted from bumblebee gut contents and corbicular pollen loads. The results of each methodology correspond to samples or surveys each taken across the same 48 sampling days. The number of plant genera indicated is a cumulative raw value for each methodology and period, with no standardization for sampling effort. Interaction diversity for transects is represented by the total number of taxa observed over each transect and sampling day, for each period. For metabarcoding methodologies, interaction diversity is the total number of plant genera observed across all samples collected during the given period.</em>&nbsp;</p>
<p><em>Functional diversity observations</em></p>
<p>The design of interaction transects only included taxa from the entomophilous community, while both metabarcoding methodologies detected taxa from the anemophilous community as well, representing 28% (N = 41) of the total identified plant genera between the two methodologies. These genera included 20 genera from <em>Poaceae</em>, nine tree/woody plant genera, and 12 other herbaceous genera (See Supporting Information). During periods one, two, and six—when gut-content metabarcoding detected more taxa than the entomophilous community recorded in transects—an average of 13% of those taxa were anemophilous or partially anemophilous (See Supporting Information).</p>
<p><em>Plant community composition across methodologies</em></p>
<p>A PERMANOVA test comparing taxonomic composition of interaction plant communities between methodologies indicated a significant effect of methodology on the observed community (P &lt; 0.001, R = 0.28). In this analysis, interaction transects showed high beta-dispersal (distance to centroid = 0.62) compared to the more centered metabarcoding and flower count results (distance to centroid ≤ 0.10), and an ANOVA test of mean dispersal by methodology indicated different levels of dispersal (P &lt; 0.001) for each methodology. The communities detected by each of the methodologies were also visualized using non-metric Multidimensional Scaling (nMDS, stress = 0.17, Fig. 3). Pairwise comparisons (Table 1) showed that the plant communities detected by flower counts were different from those of all other methodologies (P &lt; 0.001, Holm-Bonferroni), although between pairs of other methodologies, no differences were observed.&nbsp;</p>
<p><strong>Table 1.</strong> Pairwise tests comparing the community composition of plant taxa detected by four methodologies. Detected communities were compared by repeating PERMANOVA tests for each methodology pair. Tests applied the Raup-Crick dissimilarity index with 9999 permutations, and adjusted p-values were calculated using the Holm–Bonferroni method. The summarized test statistics include degrees of freedom for each methodology (DF), R², test F-statistics (F) and associated p-value (p), as well as the adjusted p-value.</p>
<div class="cell">
<div class="cell-output-display">
<table class="lightable-minimal caption-top table table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th">Methodology 1</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Methodology 2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">DF1</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">DF2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">R²</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">F</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">p</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Adjusted p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">flower count</td>
<td style="text-align: left;">gut metabarcoding</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.534</td>
<td style="text-align: right;">161.69</td>
<td style="text-align: left;">&lt;0.001</td>
<td style="text-align: left;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">flower count</td>
<td style="text-align: left;">pollen metabarcoding</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.376</td>
<td style="text-align: right;">64.95</td>
<td style="text-align: left;">&lt;0.001</td>
<td style="text-align: left;">&lt;0.001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">flower count</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.230</td>
<td style="text-align: right;">37.64</td>
<td style="text-align: left;">&lt;0.001</td>
<td style="text-align: left;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">gut metabarcoding</td>
<td style="text-align: left;">pollen metabarcoding</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.130</td>
<td style="text-align: right;">9.38</td>
<td style="text-align: left;">0.038</td>
<td style="text-align: left;">0.114</td>
</tr>
<tr class="odd">
<td style="text-align: left;">gut metabarcoding</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.010</td>
<td style="text-align: right;">0.80</td>
<td style="text-align: left;">0.55</td>
<td style="text-align: left;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">pollen metabarcoding</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">-0.024</td>
<td style="text-align: right;">-1.13</td>
<td style="text-align: left;">0.997</td>
<td style="text-align: left;">1</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-5-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Figure 3:</strong> Non-metric dimensional scaled visualization of plant communities detected by three methodologies for observing <em>B. pascuorum</em> floral interactions and a flower diversity survey. Observations from each methodology are aggregated by sampling day, reduced to binary presence/absence data, and compared in ordination using the Raup-Crick dissimilarity index (ordination stress = 0.17).&nbsp;</p>
<p><em>Specimen level metabarcoding results</em></p>
<p>Comparing metabarcoding results from the same specimens, gut contents yielded fewer taxa (mean = 12 genera, sd = 9) than pollen samples (mean = 18 genera, sd = 7). On average, only 20% of taxa (mean = 6 genera, sd = 3) were shared between the two sample types. A PERMANOVA with specimen as a blocking factor indicated a difference in the plant community observed by both sample types (P &lt; 0.01, See Supporting Information) explaining 17% of the variation between gut- and pollen-based detections (See Supporting Information). Data used in this comparison were similarly dispersed (distance to centroid = 0.08), with no difference between the two groups observed by a permutest.</p>
<p><em>Species Level Interaction Network</em></p>
<p>We calculated interaction specialization of <em>B. pascuorum</em> and an importance metric for the plant taxa within interaction networks. Specialization (<em>d’</em>; <span class="citation" data-cites="blüthgen2006">Blüthgen et al. (<a href="#ref-blüthgen2006" role="doc-biblioref">2006</a>)</span>) declined over the season for transect and pollen-metabarcoding data but remained relatively stable for gut-content metabarcoding (Fig. 4), with transects indicating complete specialization in the first period. Across all methods, <em>Lotus</em> emerged as the most important plant genus, though the structure of importance differed: the two metabarcoding networks showed more evenly distributed importance values, whereas the transect network was dominated by a few top taxa (Fig. 5).</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 4:</strong> Specialization of plant interactions for B. pascuorum as indicated by networks constructed from three interaction observation methodologies. Specialization was calculated as d’ using the methodology of Blüthgen et al.&nbsp;(2006), with d’ = 1 representing perfect specialist behavior. Specialization of B. pascuorum for each period was calculated relative to interaction data from the same species in other periods, rather than other pollinator species.</em></p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-7-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-7-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 5.</strong> Plant “importance” within B. pascuorum interaction networks constructed from three interaction observation methodologies (A) interaction transects, (B) gut content metabarcoding, and (C) corbicular pollen metabarcoding. Importance was calculated as the proportion of total plant interactions observed by the given methodology represented by interactions with the specific plant genus. Importance is visualized with block size proportional to importance, and color scaled to minimum and maximum values observed by each methodology.</em></p>
<p><em>Combined interaction network</em></p>
<p>We combined the results from each interaction methodology to create an interaction network for <em>B. pascuorum</em> with links defined by interaction outcomes, including consumption, transport, and visitation (Fig. 6). This single species network included 169 nodes, increasing the number of taxa included in the network compared to individual methodology constructed networks. Additionally, each plant taxa received up to three links, including link metadata for interaction outcomes in the network. In total, the network contained 281 descriptive links.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="manuscript_figures/THE_GBP_composite_network.jpg" class="img-fluid figure-img" width="1240"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 6.</strong> Combined interaction network for B. pascuorum including all interaction plant taxa detected by three methodologies. Interaction transect observations are represented by visitation, corbicular pollen metabarcoding observation by transport, and gut content metabarcoding observations by consumption. 170 plant genera are included within the network, each with up to three links describing the outcomes of interactions with the single pollinator species. Interactions providing links represent the presence or absence of any interaction observation within the dataset of a given methodology.</em></p>
<p><strong>Discussion</strong></p>
<p>Our results show that combining methodologies yields stronger validation of plant–pollinator interactions and deeper insight into network structure. Notably, the two metabarcoding approaches revealed shared interactions with anemophilous and partially anemophilous plants for pollen consumption and transport, highlighting the complementarity of their data. Although each interaction methodology overlapped statistically at the aggregated level, the combined network resulting from each methodology increased the total nodes, and each methodology provided context to network links. Metabarcoding alone also proved efficient at capturing a broad range of links and providing detailed, specimen-level data. Important information from the function of links is missing under the current approach to characterizing interaction networks, but using multiple methodologies helps to fill these gaps.</p>
<p>We compared each methodology in terms of the diversity of detected interactions, assignment of relative importance of plant taxa and specialization of B. pascuorum within the resulting network, and observed plant community composition. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions, metabarcoding increased observed interaction diversity <span class="citation" data-cites="milla2022 baksay2022 smart2017">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay et al., 2022</a>; <a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>; <a href="#ref-smart2017" role="doc-biblioref">Smart et al., 2017</a>)</span>, in our case by more than six-fold compared to interaction transect results. Considering this, and the time dedicated to data collection for both types of methodologies, metabarcoding was a more efficient approach. Interaction transects did provide the advantage of greater taxonomic resolution, as we were able to detect interactions at the species-species level, whereas metabarcoding provided species-genus level interactions. Beyond taxonomic detection capabilities, the results from each methodology allowed for network level cross-validation.&nbsp;</p>
<p>Network topology and specialization patterns differed markedly across methodologies. Interaction transects tended to overstate both the degree of specialization and the dominance of the most frequently visited plant taxa. Although <em>B. pascuorum</em> is known to form strong early-season associations with certain plant species (Artamendi et al.&nbsp;in preparation), the metabarcoding approaches indicated much lower specialization and produced more evenly distributed network structures. These results mirrored previous interaction networks constructed for individual pollinator species, which also have shown a tendency towards representing pollinators as specialists when using field observation data versus the generalist behavior indicated by metabarcoding data <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>)</span>. Overall, the combination of our datasets across methodologies suggested a more diverse foraging niche than observational data alone would have implied.</p>
<p>The three methodologies showed complementary patterns in network composition. Flower counts and interaction transects overlapped as expected from the study design, yet differed statistically, likely due to the much larger number of taxa detected by the former. No statistical differences were found among the three interaction-focused methods, although their dispersion differed, reflecting variation in spatial and taxonomic coverage. Interaction transects are shaped by local habitat and plant-community differences, whereas metabarcoding integrates interactions across the broader landscape, producing more consistent results. Metabarcoding approaches overlapped minimally with the floral community detected by flower counts, indicating that interaction networks include taxa not captured within transects. This is unsurprising given that flower counts reflect potential, not actual, interactions and are constrained by spatial and temporal limits that do not restrict metabarcoding.</p>
<p>Between the two metabarcoding approaches, gut-content metabarcoding captured greater overall taxonomic diversity and was more efficient, given that every specimen provided a gut sample, but not necessarily a pollen sample. Pollen samples detected more taxa per individual, however, and hypothetically offered an advantage as a non-lethal sampling option. The combination of both methodologies’ results broadened the interaction network greatly, and incorporated contextualized interaction links, showing which plant genera were consumed for adult bee nutrition, and which provided pollen for transport to the colony nest. In our case, gut-content metabarcoding was particularly informative for revealing seasonal foraging patterns, detecting more consumed taxa than were flowering in the early and late parts of the season, and showing relatively stable specialization over time. Together, these results indicated that the plant community represented in consumption-based interactions differs from the floral community captured by field and pollen-based surveys.</p>
<p><em>Metabarcoding observes forage across functional groups</em></p>
<p>The diversity of plant groups observed within our metabarcoding data, especially the temporal changes in diversity observed by gut content metabarcoding, indicated that <em>B. pascuorum</em> forages on different plant taxa than previously expected. Our reference database for metabarcoding allowed us to identify taxa from functional groups beyond the floral community sampled in our transects (See Supporting Information). Through metabarcoding, we observed interactions with a variety of taxa outside of the entomophilous meadow and shrubland plant community, including trees and shrubs, grasses, and other herbaceous plants.&nbsp;</p>
<p>Our observations of interactions with the anemophilous community are supported by previously documented interactions <span class="citation" data-cites="ibiyemi2025 selva2024 milla2022 wood2022 timberlake2024 devere2017 tanaka2020 terrell1984 pojar1973">(<a href="#ref-devere2017" role="doc-biblioref">de Vere et al., 2017</a>; <a href="#ref-ibiyemi2025" role="doc-biblioref">Ibiyemi et al., 2025</a>; <a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>; <a href="#ref-pojar1973" role="doc-biblioref">Pojar, 1973</a>; <a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>; <a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al., 2020</a>; <a href="#ref-terrell1984" role="doc-biblioref">Terrell &amp; Batra, 1984</a>; <a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>; <a href="#ref-wood2022" role="doc-biblioref">Wood et al., 2022</a>)</span>, and have especially intriguing implications for bumblebee forage behavior. Previous studies using external pollen metabarcoding have removed wind-pollinated taxa from their analyses under the argument that wind-borne pollen in samples may represent false positive interactions <span class="citation" data-cites="tanaka2020 pornon2017 negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri et al., 2015</a>; <a href="#ref-pornon2017" role="doc-biblioref">Pornon et al., 2017</a>; <a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al., 2020</a>)</span>. Our gut content results, however, suggest that the practice of removing these taxa as contaminants could be a large oversight, especially if using external pollen loads as standalone proxies for forage networks.&nbsp;</p>
<p>The presence of DNA from anemophilous taxa within gut samples suggests that interactions with these taxa may be more than coincidental interactions with wind-borne pollen. Indeed, beyond consumption for adult nutrition, there are previous indications that pollen from flowering trees supports colony establishment success and low larval mortality <span class="citation" data-cites="wood2022">(<a href="#ref-wood2022" role="doc-biblioref">Wood et al., 2022</a>)</span>. Our results support the hypothesis that bumblebees forage selectively for consumption and transport of high quality pollen <span class="citation" data-cites="ruedenauer2016 timberlake2024">(<a href="#ref-ruedenauer2016" role="doc-biblioref">Ruedenauer et al., 2016</a>; <a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>)</span>, adapting their forage to take advantage of the best available resources as they change with environmental variability <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>)</span>. While it is possible that some plant material may be transported or consumed incidentally <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>)</span>, the taxa detected within <em>B. pascuorum</em> gut contents and corbicular pollen form part of the web of biodiversity that supports the species and possibly other pollinators. Our detection of DNA from anemophilous pollen sources across the metabarcoding methodologies indicates the potential for intentional forage interactions with these taxa as a means of meeting the nutritional needs for bumblebees at various lifecycle stages.&nbsp;</p>
<p>Existing hypotheses for pollinator forage adaptations in response to environmental changes have suggested that bees expand forage diversity beyond the flowering community and across habitats in order to survive annual “hunger gaps” <span class="citation" data-cites="timberlake2024a becher2024">(<a href="#ref-becher2024" role="doc-biblioref">Becher et al., 2024</a>; <a href="#ref-timberlake2024a" role="doc-biblioref">Timberlake, Tew, et al., 2024</a>)</span>, when blooming floral species are limited <span class="citation" data-cites="wood2022 morozumi2022">(<a href="#ref-morozumi2022" role="doc-biblioref">Morozumi et al., 2022</a>; <a href="#ref-wood2022" role="doc-biblioref">Wood et al., 2022</a>)</span>. Our observation of high forage diversity in gut contents before and after the floral peak, distinct interaction and flowering taxa network topologies, and consumption of taxa across functional groups, all together support these hypotheses. While the community beyond the physical area of our transects likely played a large role in these observations, the detection of anemophilous taxa in gut contents during the periods where forage diversity was higher than flowering diversity provide evidence for a community driven component as well. These observations show how the broader taxonomic detection capacity of metabarcoding allows for detection of interactions that otherwise would go unobserved by flower visitation surveys. This advantage is extended when working with metabarcoding data at the individual sample level, where greater resolution for interactions is obtainable.</p>
<p>Metabarcoding offers individual level analysis</p>
<p>Our comparative analyses understate the resolution of our metabarcoding data. We aggregated detections by sampling day to balance effort across methods, overlooking the individual-level detail that metabarcoding can provide. When we compared taxa detected from paired pollen and gut samples at the individual level, overlap was low, revealing a difference between sample sources that was not apparent in comparisons of aggregated data. This difference likely reflects the different roles of corbicular pollen and immediately consumed pollen in the nutrition needed for different life-cycle stages <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>. Taxa repeatedly detected by both methods increased confidence in their importance. For instance, the consistent appearance of <em>Vicia</em> in both sample types early in the season supports field observations of a strong association between <em>B. pascuorum</em> and <em>Vicia</em> species (Artamendi et al., unpublished data), underscoring the value of integrating field surveys with laboratory-based methods.</p>
<p><em>Conclusions</em></p>
<p>The similarities between interaction data suggest robustness between each methodology, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, although given their lower sampling efficiency, incorporating them as a validation of other surveys may be the best way to integrate this methodology into future studies. Field observations can fill gaps left by metabarcoding methodologies, such as confirmation of pollination efficacy, interaction frequency, and species-level resolution. As a direct observation of the pollen transported to the nest, corbicular pollen may also be a good starting point for identifying which plants may provide pollen with optimal macronutrients for larval development. Similarly, gut content metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity <span class="citation" data-cites="li2025">(<a href="#ref-li2025" role="doc-biblioref">Li et al., 2025</a>)</span>. Knowing which taxa are actually ingested by pollinators is especially useful for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions <span class="citation" data-cites="cullen2021 keller2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen et al., 2021</a>; <a href="#ref-keller2021" role="doc-biblioref">Keller et al., 2021</a>)</span>, including parasite and disease transfer <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon et al., 2024</a>)</span>. Although they are not equal, our research highlights overall that each methodology offers advantages and disadvantages in terms of sensitivity, sampling effort, and perspective.</p>
<p>While most of the methodologies we applied—aside from gut-content metabarcoding—have previously been used independently to characterize plant–pollinator networks <span class="citation" data-cites="magrach2023 devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese et al., 2024</a>; e.g., <a href="#ref-magrach2023" role="doc-biblioref">Magrach et al., 2023</a>)</span>, our findings highlight the added value of integrating them. Gut-content metabarcoding emerges as a promising new tool, but its greatest potential is realized when combined with established approaches. A key next step is improving our ability to quantify interaction frequencies at the individual level using metabarcoding, whether from gut contents or pollen. Overall, methodological advances are likely to come from linking complementary data sources to fill the informational gaps left by any single approach.</p>
<p><em>Data and code availability</em></p>
<p>All amplicon sequencing data will be deposited in the European Nucleotide Archive project: PRJEB105453. Other data and code will be available in the GorBEEa public GitHub repository, Comparison_interaction_methodologies. Data and code will be made publicly available upon acceptance of the manuscript.</p>


<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-ankenbrand2015" class="csl-entry" role="listitem">
Ankenbrand, M. J., Keller, A., Wolf, M., Schultz, J., &amp; Förster, F. (2015). ITS2 Database V: Twice as Much: Table 1. <em>Molecular Biology and Evolution</em>, <em>32</em>(11), 3030–3032. <a href="https://doi.org/10.1093/molbev/msv174">https://doi.org/10.1093/molbev/msv174</a>
</div>
<div id="ref-arstingstall2021" class="csl-entry" role="listitem">
Arstingstall, K. A., DeBano, S. J., Li, X., Wooster, D. E., Rowland, M. M., Burrows, S., &amp; Frost, K. (2021). Capabilities and limitations of using DNA metabarcoding to study plant<span></span>pollinator interactions. <em>Molecular Ecology</em>, <em>30</em>(20), 5266–5297. <a href="https://doi.org/10.1111/mec.16112">https://doi.org/10.1111/mec.16112</a>
</div>
<div id="ref-artamendi2025" class="csl-entry" role="listitem">
Artamendi, M., Martin, P. A., Bartomeus, I., &amp; Magrach, A. (2025). Loss of pollinator diversity consistently reduces reproductive success for wild and cultivated plants. <em>Nature Ecology &amp; Evolution</em>, <em>9</em>(2), 296–313. <a href="https://doi.org/10.1038/s41559-024-02595-2">https://doi.org/10.1038/s41559-024-02595-2</a>
</div>
<div id="ref-baksay2022" class="csl-entry" role="listitem">
Baksay, S., Andalo, C., Galop, D., Burrus, M., Escaravage, N., &amp; Pornon, A. (2022). Using metabarcoding to investigate the strength of plant-pollinator interactions from surveys of visits to DNA sequences. <em>Frontiers in Ecology and Evolution</em>, <em>10</em>. <a href="https://doi.org/10.3389/fevo.2022.735588">https://doi.org/10.3389/fevo.2022.735588</a>
</div>
<div id="ref-becher2024" class="csl-entry" role="listitem">
Becher, M. A., Twiston-Davies, G., Osborne, J. L., &amp; Lander, T. A. (2024). Resource gaps pose the greatest threat for bumblebees during the colony establishment phase. <em>Insect Conservation and Diversity</em>, <em>17</em>(4), 676–689. <a href="https://doi.org/10.1111/icad.12736">https://doi.org/10.1111/icad.12736</a>
</div>
<div id="ref-bell2021" class="csl-entry" role="listitem">
Bell, K. (2021). <em>ITS2 july 2021</em>. figshare. <a href="https://doi.org/10.6084/M9.FIGSHARE.14936004.V1">https://doi.org/10.6084/M9.FIGSHARE.14936004.V1</a>
</div>
<div id="ref-bell2016" class="csl-entry" role="listitem">
Bell, K. L., de Vere, N., Keller, A., Richardson, R. T., Gous, A., Burgess, K. S., &amp; Brosi, B. J. (2016). Pollen DNA barcoding: Current applications and future prospects. <em>Genome</em>, <em>59</em>(9), 629–640. <a href="https://doi.org/10.1139/gen-2015-0200">https://doi.org/10.1139/gen-2015-0200</a>
</div>
<div id="ref-bell2017" class="csl-entry" role="listitem">
Bell, K. L., Fowler, J., Burgess, K. S., Dobbs, E. K., Gruenewald, D., Lawley, B., Morozumi, C., &amp; Brosi, B. J. (2017). Applying pollen DNA metabarcoding to the study of plant<span></span>pollinator interactions. <em>Applications in Plant Sciences</em>, <em>5</em>(6), 1600124. <a href="https://doi.org/10.3732/apps.1600124">https://doi.org/10.3732/apps.1600124</a>
</div>
<div id="ref-blüthgen2006" class="csl-entry" role="listitem">
Blüthgen, N., Menzel, F., &amp; Blüthgen, N. (2006). Measuring specialization in species interaction networks. <em>BMC Ecology</em>, <em>6</em>(1), 9. <a href="https://doi.org/10.1186/1472-6785-6-9">https://doi.org/10.1186/1472-6785-6-9</a>
</div>
<div id="ref-bosch2009" class="csl-entry" role="listitem">
Bosch, J., Martín González, A. M., Rodrigo, A., &amp; Navarro, D. (2009). Plant<span></span>pollinator networks: adding the pollinator<span>’</span>s perspective. <em>Ecology Letters</em>, <em>12</em>(5), 409–419. <a href="https://doi.org/10.1111/j.1461-0248.2009.01296.x">https://doi.org/10.1111/j.1461-0248.2009.01296.x</a>
</div>
<div id="ref-burkle2011" class="csl-entry" role="listitem">
Burkle, L. A., &amp; Alarcón, R. (2011). The future of plant<span></span>pollinator diversity: Understanding interaction networks across time, space, and global change. <em>American Journal of Botany</em>, <em>98</em>(3), 528–538. <a href="https://doi.org/10.3732/ajb.1000391">https://doi.org/10.3732/ajb.1000391</a>
</div>
<div id="ref-callahan2016" class="csl-entry" role="listitem">
Callahan, B. J., McMurdie, P. J., Rosen, M. J., Han, A. W., Johnson, A. J. A., &amp; Holmes, S. P. (2016). DADA2: High resolution sample inference from illumina amplicon data. <em>Nature Methods</em>, <em>13</em>(7), 581–583. <a href="https://doi.org/10.1038/nmeth.3869">https://doi.org/10.1038/nmeth.3869</a>
</div>
<div id="ref-cullen2021" class="csl-entry" role="listitem">
Cullen, N., Fetters, A., &amp; Ashman, T.-L. (2021). Integrating microbes into pollination. <em>Current Opinion in Insect Science</em>, <em>44</em>, 48–54. <a href="https://doi.org/10.1016/j.cois.2020.11.002">https://doi.org/10.1016/j.cois.2020.11.002</a>
</div>
<div id="ref-davis2018" class="csl-entry" role="listitem">
Davis, N. M., Proctor, D. M., Holmes, S. P., Relman, D. A., &amp; Callahan, B. J. (2018). Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data. <em>Microbiome</em>, <em>6</em>(1), 226. <a href="https://doi.org/10.1186/s40168-018-0605-2">https://doi.org/10.1186/s40168-018-0605-2</a>
</div>
<div id="ref-devere2017" class="csl-entry" role="listitem">
de Vere, N., Jones, L. E., Gilmore, T., Moscrop, J., Lowe, A., Smith, D., Hegarty, M. J., Creer, S., &amp; Ford, C. R. (2017). Using DNA metabarcoding to investigate honey bee foraging reveals limited flower use despite high floral availability. <em>Scientific Reports</em>, <em>7</em>(1), 42838. <a href="https://doi.org/10.1038/srep42838">https://doi.org/10.1038/srep42838</a>
</div>
<div id="ref-devriese2024" class="csl-entry" role="listitem">
Devriese, A., Peeters, G., Brys, R., &amp; Jacquemyn, H. (2024). The impact of extraction method and pollen concentration on community composition for pollen metabarcoding. <em>Applications in Plant Sciences</em>, <em>12</em>(5), e11601. <a href="https://doi.org/10.1002/aps3.11601">https://doi.org/10.1002/aps3.11601</a>
</div>
<div id="ref-bipartite" class="csl-entry" role="listitem">
Dormann, C. F., Fruend, J., Bluethgen, N., &amp; Gruber, B. (2009). <em>Indices, graphs and null models: Analyzing bipartite ecological networks.</em> <em>2</em>, 7–24.
</div>
<div id="ref-evans2020" class="csl-entry" role="listitem">
Evans, D., &amp; Kitson, J. (2020). Molecular ecology as a tool for understanding pollination and other plant<span></span>insect interactions. <em>Current Opinion in Insect Science</em>, <em>38</em>, 26–33. <a href="https://doi.org/10.1016/j.cois.2020.01.005">https://doi.org/10.1016/j.cois.2020.01.005</a>
</div>
<div id="ref-haag2023" class="csl-entry" role="listitem">
Haag, K. L., Caesar, L., Silveira Regueira-Neto, M. da, Sousa, D. R. de, Montenegro Marcelino, V., Queiroz Balbino, V. de, &amp; Torres Carvalho, A. (2023). Temporal Changes in Gut Microbiota Composition and Pollen Diet Associated with Colony Weakness of a Stingless Bee. <em>Microbial Ecology</em>, <em>85</em>(4), 1514–1526. <a href="https://doi.org/10.1007/s00248-022-02027-3">https://doi.org/10.1007/s00248-022-02027-3</a>
</div>
<div id="ref-ibiyemi2025" class="csl-entry" role="listitem">
Ibiyemi, D., Harris-Shultz, K., Jespersen, D., &amp; Joseph, S. V. (2025). Understanding the Foraging Behavior of Sweat Bees, Bumble Bees, and Honey Bees on Centipedegrass for Conservation Strategies. <em>Journal of Insect Behavior</em>, <em>38</em>(2), 29. <a href="https://doi.org/10.1007/s10905-025-09893-y">https://doi.org/10.1007/s10905-025-09893-y</a>
</div>
<div id="ref-katumo2022" class="csl-entry" role="listitem">
Katumo, D. M., Liang, H., Ochola, A. C., Lv, M., Wang, Q.-F., &amp; Yang, C.-F. (2022). Pollinator diversity benefits natural and agricultural ecosystems, environmental health, and human welfare. <em>Plant Diversity</em>, <em>44</em>(5), 429–435. <a href="https://doi.org/10.1016/j.pld.2022.01.005">https://doi.org/10.1016/j.pld.2022.01.005</a>
</div>
<div id="ref-keller2021" class="csl-entry" role="listitem">
Keller, A., McFrederick, Q., &amp; Leonhardt, S. (2021). (More than) hitchhikers through the network: The shared microbiome of bees and flowers. <em>Current Opinion in Insect Science</em>, <em>44</em>, 8–15. <a href="https://doi.org/10.1016/j.cois.2020.09.007">https://doi.org/10.1016/j.cois.2020.09.007</a>
</div>
<div id="ref-klein2006" class="csl-entry" role="listitem">
Klein, A.-M., Vaissière, B. E., Cane, J. H., Steffan-Dewenter, I., Cunningham, S. A., Kremen, C., &amp; Tscharntke, T. (2006). Importance of pollinators in changing landscapes for world crops. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>274</em>(1608), 303–313. <a href="https://doi.org/10.1098/rspb.2006.3721">https://doi.org/10.1098/rspb.2006.3721</a>
</div>
<div id="ref-leach2018" class="csl-entry" role="listitem">
Leach, M. E., &amp; Drummond, F. (2018). A Review of Native Wild Bee Nutritional Health. <em>International Journal of Ecology</em>, <em>2018</em>(1), 9607246. <a href="https://doi.org/10.1155/2018/9607246">https://doi.org/10.1155/2018/9607246</a>
</div>
<div id="ref-lecocq2015" class="csl-entry" role="listitem">
Lecocq, T., Brasero, N., Martinet, B., Valterovà, I., &amp; Rasmont, P. (2015). Highly polytypic taxon complex: interspecific and intraspecific integrative taxonomic assessment of the widespread pollinator ombus pascuorum Scopoli 1763 (Hymenoptera: Apidae). <em>Systematic Entomology</em>, <em>40</em>(4), 881–890. <a href="https://doi.org/10.1111/syen.12137">https://doi.org/10.1111/syen.12137</a>
</div>
<div id="ref-leonhardt2012" class="csl-entry" role="listitem">
Leonhardt, S. D., &amp; Blüthgen, N. (2012). The same, but different: pollen foraging in honeybee and bumblebee colonies. <em>Apidologie</em>, <em>43</em>(4), 449–464. <a href="https://doi.org/10.1007/s13592-011-0112-y">https://doi.org/10.1007/s13592-011-0112-y</a>
</div>
<div id="ref-leontidou2021" class="csl-entry" role="listitem">
Leontidou, K., Vokou, D., Sandionigi, A., Bruno, A., Lazarina, M., De Groeve, J., Li, M., Varotto, C., Girardi, M., Casiraghi, M., &amp; Cristofori, A. (2021). Plant biodiversity assessment through pollen DNA metabarcoding in Natura 2000 habitats (Italian Alps). <em>Scientific Reports</em>, <em>11</em>(1), 18226. <a href="https://doi.org/10.1038/s41598-021-97619-3">https://doi.org/10.1038/s41598-021-97619-3</a>
</div>
<div id="ref-leponiemi2023" class="csl-entry" role="listitem">
Leponiemi, M., Freitak, D., Moreno-Torres, M., Pferschy-Wenzig, E.-M., Becker-Scarpitta, A., Tiusanen, M., Vesterinen, E. J., &amp; Wirta, H. (2023). Honeybees<span>’</span> foraging choices for nectar and pollen revealed by DNA metabarcoding. <em>Scientific Reports</em>, <em>13</em>(1), 14753. <a href="https://doi.org/10.1038/s41598-023-42102-4">https://doi.org/10.1038/s41598-023-42102-4</a>
</div>
<div id="ref-li2025" class="csl-entry" role="listitem">
Li, Y., Liu, C., Wang, Y., Li, M., Zou, S., Hu, X., Chen, Z., Li, M., Ma, C., Obi, C. J., Zhou, X., Zou, Y., &amp; Tang, M. (2025). Urban wild bee well-being revealed by gut metagenome data: A mason bee model. <em>Insect Science</em>, <em>n/a</em>(n/a). <a href="https://doi.org/10.1111/1744-7917.70051">https://doi.org/10.1111/1744-7917.70051</a>
</div>
<div id="ref-lignon2024" class="csl-entry" role="listitem">
Lignon, V. A., Mas, F., Jones, E. E., Kaiser, C., &amp; Dhami, M. K. (2024). The floral interface: a playground for interactions between insect pollinators, microbes, and plants. <em>New Zealand Journal of Zoology</em>, 1–20. <a href="https://doi.org/10.1080/03014223.2024.2353285">https://doi.org/10.1080/03014223.2024.2353285</a>
</div>
<div id="ref-lowe2022" class="csl-entry" role="listitem">
Lowe, A., Jones, L., Witter, L., Creer, S., &amp; Vere, N. de. (2022). Using DNA Metabarcoding to Identify Floral Visitation by Pollinators. <em>Diversity</em>, <em>14</em>(4), 236. <a href="https://doi.org/10.3390/d14040236">https://doi.org/10.3390/d14040236</a>
</div>
<div id="ref-magrach2023" class="csl-entry" role="listitem">
Magrach, A., Artamendi, M., Lapido, P. D., Parejo, C., &amp; Rubio, E. (2023). Indirect interactions between pollinators drive interaction rewiring through space. <em>Ecosphere</em>, <em>14</em>(6), e4521. <a href="https://doi.org/10.1002/ecs2.4521">https://doi.org/10.1002/ecs2.4521</a>
</div>
<div id="ref-mayr2021" class="csl-entry" role="listitem">
Mayr, A. V., Keller, A., Peters, M. K., Grimmer, G., Krischke, B., Geyer, M., Schmitt, T., &amp; Steffan-Dewenter, I. (2021). Cryptic species and hidden ecological interactions of halictine bees along an elevational gradient. <em>Ecology and Evolution</em>, <em>11</em>(12), 7700–7712. <a href="https://doi.org/10.1002/ece3.7605">https://doi.org/10.1002/ece3.7605</a>
</div>
<div id="ref-milla2022" class="csl-entry" role="listitem">
Milla, L., Schmidt-Lebuhn, A., Bovill, J., &amp; Encinas-Viso, F. (2022). Monitoring of honey bee floral resources with pollen DNA metabarcoding as a complementary tool to vegetation surveys. <em>Ecological Solutions and Evidence</em>, <em>3</em>(1), e12120. <a href="https://doi.org/10.1002/2688-8319.12120">https://doi.org/10.1002/2688-8319.12120</a>
</div>
<div id="ref-morozumi2022" class="csl-entry" role="listitem">
Morozumi, C., Loy, X., Reynolds, V., Schiffer, A., Morrison, B., Savage, J., &amp; Brosi, B. (2022). Simultaneous niche expansion and contraction in plant<span></span>pollinator networks under drought. <em>Oikos</em>, <em>2022</em>(11), e09265. <a href="https://doi.org/10.1111/oik.09265">https://doi.org/10.1111/oik.09265</a>
</div>
<div id="ref-negri2015" class="csl-entry" role="listitem">
Negri, I., Mavris, C., Prisco, G. D., Caprio, E., &amp; Pellecchia, M. (2015). Honey Bees (Apis mellifera, L.) as Active Samplers of Airborne Particulate Matter. <em>PLOS ONE</em>, <em>10</em>(7), e0132491. <a href="https://doi.org/10.1371/journal.pone.0132491">https://doi.org/10.1371/journal.pone.0132491</a>
</div>
<div id="ref-vegan" class="csl-entry" role="listitem">
Oksanen, J., Simpson, G. L., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R., O’Hara, R. B., Solymos, P., Stevens, M. H. H., Szoecs, E., Wagner, H., Barbour, M., Bedward, M., Bolker, B., Borcard, D., Carvalho, G., Chirico, M., De Caceres, M., Durand, S., … Weedon, J. (2024). <em>Vegan: Community ecology package</em>. <a href="https://CRAN.R-project.org/package=vegan">https://CRAN.R-project.org/package=vegan</a>
</div>
<div id="ref-pashalidou2020" class="csl-entry" role="listitem">
Pashalidou, F. G., Lambert, H., Peybernes, T., Mescher, M. C., &amp; De Moraes, C. M. (2020). Bumble bees damage plant leaves and accelerate flower production when pollen is scarce. <em>Science</em>, <em>368</em>(6493), 881–884. <a href="https://doi.org/10.1126/science.aay0496">https://doi.org/10.1126/science.aay0496</a>
</div>
<div id="ref-pojar1973" class="csl-entry" role="listitem">
Pojar, J. (1973). Pollination of typically anemophilous salt marsh plants by bumble bees, bombus terricola occidentalis grne. <em>The American Midland Naturalist</em>, <em>89</em>(2), 448–451. <a href="https://doi.org/10.2307/2424049">https://doi.org/10.2307/2424049</a>
</div>
<div id="ref-popic2012" class="csl-entry" role="listitem">
Popic, T. J., Wardle, G. M., &amp; Davila, Y. C. (2012). Flower<span>-</span>visitor networks only partially predict the function of pollen transport by bees. <em>Austral Ecology</em>, <em>38</em>(1), 76–86. <a href="https://doi.org/10.1111/j.1442-9993.2012.02377.x">https://doi.org/10.1111/j.1442-9993.2012.02377.x</a>
</div>
<div id="ref-pornon2017" class="csl-entry" role="listitem">
Pornon, A., Andalo, C., Burrus, M., &amp; Escaravage, N. (2017). DNA metabarcoding data unveils invisible pollination networks. <em>Scientific Reports</em>, <em>7</em>(1), 16828. <a href="https://doi.org/10.1038/s41598-017-16785-5">https://doi.org/10.1038/s41598-017-16785-5</a>
</div>
<div id="ref-quintero2022" class="csl-entry" role="listitem">
Quintero, E., Isla, J., &amp; Jordano, P. (2022). Methodological overview and data-merging approaches in the study of plant<span></span>frugivore interactions. <em>Oikos</em>, <em>2022</em>(2). <a href="https://doi.org/10.1111/oik.08379">https://doi.org/10.1111/oik.08379</a>
</div>
<div id="ref-ruedenauer2016" class="csl-entry" role="listitem">
Ruedenauer, F. A., Spaethe, J., &amp; Leonhardt, S. D. (2016). Hungry for quality<span></span>individual bumblebees forage flexibly to collect high-quality pollen. <em>Behavioral Ecology and Sociobiology</em>, <em>70</em>(8), 1209–1217. <a href="https://doi.org/10.1007/s00265-016-2129-8">https://doi.org/10.1007/s00265-016-2129-8</a>
</div>
<div id="ref-saunders2018" class="csl-entry" role="listitem">
Saunders, M. E. (2018). Insect pollinators collect pollen from wind-pollinated plants: implications for pollination ecology and sustainable agriculture. <em>Insect Conservation and Diversity</em>, <em>11</em>(1), 13–31. <a href="https://doi.org/10.1111/icad.12243">https://doi.org/10.1111/icad.12243</a>
</div>
<div id="ref-selva2024" class="csl-entry" role="listitem">
Selva, S., Moretti, M., Ruedenauer, F., Keller, A., Fournier, B., Leonhardt, S. D., Eggenberger, H. A., &amp; Abella, J. C. (2024). <em>Urban bumblebees diversify their foraging strategy to maintain nutrient intake</em>. <a href="https://ecoevorxiv.org/repository/view/7812/">https://ecoevorxiv.org/repository/view/7812/</a>
</div>
<div id="ref-shi2025" class="csl-entry" role="listitem">
Shi, H., Ratering, S., Schneider, B., &amp; Schnell, S. (2025). Microbiome of honey bee corbicular pollen: Factors influencing its structure and potential for studying pathogen transmission. <em>Science of The Total Environment</em>, <em>958</em>, 178107. <a href="https://doi.org/10.1016/j.scitotenv.2024.178107">https://doi.org/10.1016/j.scitotenv.2024.178107</a>
</div>
<div id="ref-smart2017" class="csl-entry" role="listitem">
Smart, M. D., Cornman, R. S., Iwanowicz, D. D., McDermott-Kubeczko, M., Pettis, J. S., Spivak, M. S., &amp; Otto, C. R. V. (2017). A comparison of honey bee-collected pollen from working agricultural lands using light microscopy and ITS metabarcoding. <em>Environmental Entomology</em>, <em>46</em>(1), 38–49. <a href="https://doi.org/10.1093/ee/nvw159">https://doi.org/10.1093/ee/nvw159</a>
</div>
<div id="ref-tanaka2020" class="csl-entry" role="listitem">
Tanaka, K., Nozaki, A., Nakadai, H., Shiwa, Y., &amp; Shimizu-Kadota, M. (2020). Using pollen DNA metabarcoding to profile nectar sources of urban beekeeping in K<span>ō</span>t<span>ō</span>-ku, Tokyo. <em>BMC Research Notes</em>, <em>13</em>(1), 515. <a href="https://doi.org/10.1186/s13104-020-05361-2">https://doi.org/10.1186/s13104-020-05361-2</a>
</div>
<div id="ref-terrell1984" class="csl-entry" role="listitem">
Terrell, E. E., &amp; Batra, S. W. T. (1984). Insects collect pollen of eastern wildrice, zizania aquatica (poaceae). <em>Castanea</em>, <em>49</em>(1), 31–34. <a href="https://www.jstor.org/stable/4033059">https://www.jstor.org/stable/4033059</a>
</div>
<div id="ref-timberlake2024a" class="csl-entry" role="listitem">
Timberlake, T. P., Tew, N. E., &amp; Memmott, J. (2024). Gardens reduce seasonal hunger gaps for farmland pollinators. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>291</em>(2033). <a href="https://doi.org/10.1098/rspb.2024.1523">https://doi.org/10.1098/rspb.2024.1523</a>
</div>
<div id="ref-timberlake2024" class="csl-entry" role="listitem">
Timberlake, T. P., Vere, N. de, Jones, L. E., Vaughan, I. P., Baude, M., &amp; Memmott, J. (2024). Ten-a-day: Bumblebee pollen loads reveal high consistency in foraging breadth among species, sites and seasons. <em>Ecological Solutions and Evidence</em>, <em>5</em>(3), e12360. <a href="https://doi.org/10.1002/2688-8319.12360">https://doi.org/10.1002/2688-8319.12360</a>
</div>
<div id="ref-vanbergen2013" class="csl-entry" role="listitem">
Vanbergen, A. J., &amp; Insect Pollinators Initiative, the. (2013). Threats to an ecosystem service: pressures on pollinators. <em>Frontiers in Ecology and the Environment</em>, <em>11</em>(5), 251–259. <a href="https://doi.org/10.1890/120126">https://doi.org/10.1890/120126</a>
</div>
<div id="ref-vaudo2015" class="csl-entry" role="listitem">
Vaudo, A. D. (2015). Bee nutrition and floral resource restoration. <em>Current Opinion in Insect Science</em>, <em>10</em>, 133–141. <a href="https://doi.org/10.1016/j.cois.2015.05.008">https://doi.org/10.1016/j.cois.2015.05.008</a>
</div>
<div id="ref-wood2022" class="csl-entry" role="listitem">
Wood, T. J., Vanderplanck, M., Vastrade, M., Vaudo, A. D., &amp; Michez, D. (2022). Trees for bees: could woody plant pollen be used as a consistent resource in bee-focused agri-environment schemes? <em>Entomologia Generalis</em>, <em>42</em>(3), 361. <a href="https://doi.org/10.1127/entomologia/2021/1241">https://doi.org/10.1127/entomologia/2021/1241</a>
</div>
<div id="ref-woodcock2019" class="csl-entry" role="listitem">
Woodcock, B. A., Garratt, M. P. D., Powney, G. D., Shaw, R. F., Osborne, J. L., Soroka, J., Lindström, S. a. M., Stanley, D., Ouvrard, P., Edwards, M. E., Jauker, F., McCracken, M. E., Zou, Y., Potts, S. G., Rundlöf, M., Noriega, J. A., Greenop, A., Smith, H. G., Bommarco, R., … Pywell, R. F. (2019). Meta-analysis reveals that pollinator functional diversity and abundance enhance crop pollination and yield. <em>Nature Communications</em>, <em>10</em>(1), 1481. <a href="https://doi.org/10.1038/s41467-019-09393-6">https://doi.org/10.1038/s41467-019-09393-6</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>