<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Interaction methodologies</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="interaction_methodologies_files/libs/clipboard/clipboard.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="interaction_methodologies_files/libs/quarto-html/popper.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/anchor.min.js"></script>
<link href="interaction_methodologies_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="interaction_methodologies_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-bb462d781dde1847d9e3ccf7736099dd.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent quarto-light">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Interaction methodologies</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><strong>Introduction</strong></p>
<p>The global importance and threatened loss of pollination as a critical ecosystem service is clear within the existing research on pollinator communities <span class="citation" data-cites="klein2006">(<a href="#ref-klein2006" role="doc-biblioref">Klein <em>et al.</em> 2006</a>)</span> <span class="citation" data-cites="vanbergen2013">(<a href="#ref-vanbergen2013" role="doc-biblioref">Vanbergen &amp; Insect Pollinators Initiative 2013</a>)</span>. Our understanding of this topic is incomplete, however, given that historically the methodological approach to studying plant-pollinator interactions has been biased towards the plant side of the interaction <span class="citation" data-cites="evans2020">(<a href="#ref-evans2020" role="doc-biblioref">Evans &amp; Kitson 2020</a>)</span> <span class="citation" data-cites="bosch2009">(<a href="#ref-bosch2009" role="doc-biblioref">Bosch <em>et al.</em> 2009</a>)</span>. Pollination maintains plant community biodiversity, which is essential to ecosystem functioning <span class="citation" data-cites="ollerton2017">(<a href="#ref-ollerton2017" role="doc-biblioref">Ollerton 2017</a>)</span> across global ecosystems and agroecosystems. Pollinators critically support the reproduction of 94% of wild flowering plants and 75% of crop species <span class="citation" data-cites="vanbergen2013">(<a href="#ref-vanbergen2013" role="doc-biblioref">Vanbergen &amp; Insect Pollinators Initiative 2013</a>)</span>, contributing to 35% of global food production <span class="citation" data-cites="klein2006">(<a href="#ref-klein2006" role="doc-biblioref">Klein <em>et al.</em> 2006</a>)</span>. The well established relationship between pollinator diversity and the productivity of plant communities <span class="citation" data-cites="artamendi2025">(<a href="#ref-artamendi2025" role="doc-biblioref">Artamendi <em>et al.</em> 2025</a>)</span> <span class="citation" data-cites="woodcock2019">(<a href="#ref-woodcock2019" role="doc-biblioref">Woodcock <em>et al.</em> 2019</a>)</span> <span class="citation" data-cites="katumo2022">(<a href="#ref-katumo2022" role="doc-biblioref">Katumo <em>et al.</em> 2022</a>)</span> lacks an equally developed mirrored perspective, describing the floral diversity that supports pollinators. Our understanding of plant-pollinator interaction networks is dependent on the perspective from which nodes and links are observed, and the functional outcomes of observed interactions. The current set of methodologies for reconstructing interaction networks overlooks consumer–resource links and interaction outcomes that are critical for understanding how interaction networks support pollinators. Shifting network studies to incorporate the pollinator perspective, and leveraging the contributions of different methodologies is essential to creating a more complete image of interaction networks.</p>
<p>Approaching the study of plant-pollinator interactions from the pollinator perspective provides valuable insights on pollinators’ needs. Research based on microscopy and molecular analyses of pollen load samples from different pollinator specimens has shown that this perspective allows for identification of greater plant species diversity within interaction networks compared to studies based solely on field observations <span class="citation" data-cites="bosch2009">(<a href="#ref-bosch2009" role="doc-biblioref">Bosch <em>et al.</em> 2009</a>)</span>. Additionally, studies based in the pollinator perspective have revealed greater detail in forage preference trends, such as how different bee species may preferentially forage using either quality or quantity based strategies <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> based on ability to detect pollen quality <span class="citation" data-cites="leonhardt2012">(<a href="#ref-leonhardt2012" role="doc-biblioref">Leonhardt &amp; Blüthgen 2012</a>)</span>, life cycle timing, or metabolic specialization <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Advances in reliability and accessibility of molecular techniques have made these approaches more feasible for studying plant-pollinator interactions. Field surveys <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span> and pollen grain analyses <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="baksay2020">(<a href="#ref-baksay2020" role="doc-biblioref">Baksay <em>et al.</em> 2020</a>)</span> <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">Bell <em>et al.</em> 2016</a>)</span> <span class="citation" data-cites="smart2017">(<a href="#ref-smart2017" role="doc-biblioref">Smart <em>et al.</em> 2017</a>)</span> can now be effectively complemented by genetic tools, enhancing our understanding of these complex interactions.</p>
<p>Genetic tools have a great capacity to detect plant-pollinator interactions that may be unobserved by pollen microscopy and traditional field surveys <span class="citation" data-cites="lowe2022">(<a href="#ref-lowe2022" role="doc-biblioref">Lowe <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall <em>et al.</em> 2021</a>)</span> <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span> <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">Bell <em>et al.</em> 2016</a>)</span>. Tools such as metabarcoding of pollen samples, whether carried externally by pollinators <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese <em>et al.</em> 2024</a>)</span>, or deposited in beehive stores of honey or beebread <span class="citation" data-cites="leponiemi2023">(<a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere <em>et al.</em> 2017</a>)</span> <span class="citation" data-cites="anderson2013">(<a href="#ref-anderson2013" role="doc-biblioref">Anderson <em>et al.</em> 2013</a>)</span> complement the pollination interactions observed by field studies <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall <em>et al.</em> 2021</a>)</span> <span class="citation" data-cites="bell2017">(<a href="#ref-bell2017" role="doc-biblioref">Bell <em>et al.</em> 2017</a>)</span>. The parallel application of pollen metabarcoding techniques to traditional pollination network studies has demonstrated that genetic surveys expand capacity of network studies in species detection by 9 - 144% <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="smart2017">(<a href="#ref-smart2017" role="doc-biblioref">Smart <em>et al.</em> 2017</a>)</span>, network sampling completeness up to 30% <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall <em>et al.</em> 2021</a>)</span>, and revealed interactions beyond the traditionally sampled entomophilous community <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere <em>et al.</em> 2017</a>)</span>. Metabarcoding analyses have further informed knowledge of forage seasonality <span class="citation" data-cites="leponiemi2023">(<a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi <em>et al.</em> 2023</a>)</span>, specialist vs.&nbsp;generalist behaviour and quality vs.&nbsp;quantity based forage preferences <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span>. Beyond inferring plant-pollinator interactions, molecular tools have also been used to uncover the exchange of microbial communities between pollinators, facilitated by flower nectar <span class="citation" data-cites="donald2023">(<a href="#ref-donald2023" role="doc-biblioref">Donald <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="donald2022">(<a href="#ref-donald2022" role="doc-biblioref">Donald <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="devega2021">(<a href="#ref-devega2021" role="doc-biblioref">Vega <em>et al.</em> 2021</a>)</span>, revealing transmission pathways for microbes shared by social pollinators <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="hammer2021">(<a href="#ref-hammer2021" role="doc-biblioref">Hammer <em>et al.</em> 2021</a>)</span>. Most of the studies applying metagenomics to pollinator-sourced samples for interaction network construction have used external pollen loads of bees as a sample source <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="leponiemi2023">(<a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="leontidou2021">(<a href="#ref-leontidou2021" role="doc-biblioref">Leontidou <em>et al.</em> 2021</a>)</span>, despite clear limitations of this approach.</p>
<p>The pollen carried on a pollinator’s body relates to limited specific interaction types, and is vulnerable to producing false-positive interaction detections. For bees, corbicular pollen is collected with the intent of transport to the colony nest for storage and larval feeding <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Prior studies have used external pollen loads to characterize forage networks and interpret forage preferences without acknowledging that these observations are limited to interactions aimed at pollen transport [<span class="citation" data-cites="selva2024">Selva <em>et al.</em> (<a href="#ref-selva2024" role="doc-biblioref">2024</a>)</span>]<span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> <span class="citation" data-cites="leponiemi2023">(<a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span>. These interpretations based on the standalone use of external pollen as a sample source are problematic because they overstep the interpretative capacity of the sample source. Because the resources needed for adult pollinator nutrition are different from those needed at the larval stage <span class="citation" data-cites="leach2018">(<a href="#ref-leach2018" role="doc-biblioref">Leach &amp; Drummond 2018</a>)</span> <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>, transported pollen may not completely represent the interactions necessary for maintenance of pollinator diets. This is especially true for bumblebees (<em>Bombus spp.</em>), which are able to discern pollen resource quality and make discerning forage choices <span class="citation" data-cites="leonhardt2012">(<a href="#ref-leonhardt2012" role="doc-biblioref">Leonhardt &amp; Blüthgen 2012</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span>. Apart from intentional interactions, bumblebees make many trial-and-error floral visits in order to find adequate forage <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span>, which may result in pollen transport without consumption. Finally, externally carried pollen can come from beyond floral interactions, even including accumulation of windborne material <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span> <span class="citation" data-cites="negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri <em>et al.</em> 2015</a>)</span>. Accounting for different interaction types is a critical next step in disentangling the network of floral diversity used by pollinators and understanding the functions of network links.</p>
<p>Pollinator intestinal tracts (“guts”) represent an additional source for observing what pollen from which plant species pollinators consume <span class="citation" data-cites="li2025">(<a href="#ref-li2025" role="doc-biblioref">Li <em>et al.</em> 2025</a>)</span> <span class="citation" data-cites="haag2023">(<a href="#ref-haag2023" role="doc-biblioref">Haag <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="mayr2021">(<a href="#ref-mayr2021" role="doc-biblioref">Mayr <em>et al.</em> 2021</a>)</span>. The gut content approach can account for the false positive and negative forage interactions apparent in external pollen analyses, and reveal important implications of interactions that are overlooked by most plant-pollinator network studies, such interactions with anemophilous plants. Although these interactions are intentionally excluded from meteagenomic network studies <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span> <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span>, here is an accumulating body of literature supporting the idea that pollinators must regularly search across functional groups of the plant community to meet their nutritional needs <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere <em>et al.</em> 2017</a>)</span> <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span> <span class="citation" data-cites="terrell1984">(<a href="#ref-terrell1984" role="doc-biblioref">Terrell &amp; Batra 1984</a>)</span> <span class="citation" data-cites="pojar1973">(<a href="#ref-pojar1973" role="doc-biblioref">Pojar 1973</a>)</span>. This topic has largely been neglected in the literature on pollinator ecology <span class="citation" data-cites="saunders2018">(<a href="#ref-saunders2018" role="doc-biblioref">Saunders 2018</a>)</span>. This understudied component of pollinator forage combined with the surprising lack of genetic analyses on pollinator gut contents together represent a clear knowledge gap and opportunity to reveal more about pollinator forage ecology.&nbsp;</p>
<p>We want to know how metabarcoding of pollinator gut contents can complement or challenge the characterization of plant-pollinator interaction networks described by other more common methodologies, including field surveys of plant pollinator interactions, and external pollen load metabarcoding. To address this, we compare interaction networks constructed from each of these methodologies as constructed for a single model pollinator, <em>Bombus pascuorum</em>. We expect that the feeding interactions detected in gut metabarcoding will include a network of plant taxa distinct from those detected by other methodologies. Ideally, the addition of data from pollinator gut contents will elevate our capacity to detect meaningful plant-pollinator interactions, and learn more about interaction types and implications for pollinator health. Our objective is to determine whether this combined methodological approach can provide further insights into pollinator forage ecology and plant-pollinator interaction networks.</p>
<p><strong>Methods</strong></p>
<p><em>Study area</em></p>
<p>Our data come from fieldwork and sample collection conducted in Gorbeia Natural Park (“Gorbeia”), a protected area in the Basque Autonomous Region of Northern Spain. Gorbeia is a temperate mountainous area that receives more than 150 days of rain, and 1,800 - 2,200 mm total of precipitation annually <span class="citation" data-cites="diputaciónforaldebizkaia2023">(<a href="#ref-diputaciónforaldebizkaia2023" role="doc-biblioref">Diputación Foral de Bizkaia 2023</a>)</span>. Mean temperatures during the study period in 2023 ranged from 10 to 20 °C <span class="citation" data-cites="R-climaemet">(<a href="#ref-R-climaemet" role="doc-biblioref">Pizarro <em>et al.</em> 2021</a>)</span>. Altitude, geomorphology, and varying sun exposure together create a diverse plant community across the sites. This diversity is exaggerated by Gorbeia’s location in the transitional margin of the Mediterranean and Atlantic drainages. As a result of this location, the resulting floral community includes a broad mix of species including taxa from Mediterranean, Basque-Cantabrian, and North-central European communities <span class="citation" data-cites="loidi2011">(<a href="#ref-loidi2011" role="doc-biblioref">Loidi <em>et al.</em> 2011</a>)</span>.</p>
<p>Within Gorbeia, we selected 16 sampling sites located within mixed zones of meadows and shrublands common to rocky open areas found at higher elevations within the park. All of the sites were located at similar elevations (&gt;800 m a.s.l).&nbsp; Most of the field sites were adjacent to forests, including mixed stands of beech and oak, and plantations of foreign conifer and eucalyptus.</p>
<p>We conducted fieldwork from early April to the end of July, 2023, covering the main flowering period and peak annual pollinator activity in Gorbeia. We limited fieldwork to days with clear weather, avoiding rainy or windy days which suppressed pollinator activity. On each sampling day during this timeframe, we visited field sites in pairs.&nbsp; Sampling sites were organized in eight pairs separated by around one km, with a distance between site pairs of around three km. We sampled two sites simultaneously each day in separate field teams. Sampling days were organized into six periods, in which we sampled each site once per period. A period ended once all surveys and samples were collected for each site. Period duration typically amounted to two to three weeks, depending on weather, with samplings occasionally split between days for specific sites as demanded by weather. We conducted three types of surveys during daily peaks of pollinator activity, between 10 am and 5 pm. Surveys included floral resource availability (“flower counts”), interaction transect surveys, and <em>Bombus pascuorum</em> specimen collection for both pollen load and gut content analysis.</p>
<p><em>Interaction transects and floral resource availability surveys</em>&nbsp;</p>
<p>We used the same transect at each site for both interaction transect surveys and flower counts. The transects measured 250 m in length, and we recorded observations within the approximately two meter margin containing the transect line, which was visually defined by an animal trail or flag markers. For interaction surveys, we walked the transect 3 times a day for one hour of observation. Every insect observed interacting with the reproductive parts of the flower of an herbaceous plant species inside the transect area was recorded. For this study, we filtered the data from B. pascuorum interactions from these survey data. Transect observation sessions were separated by approximately two hour gaps (i.e.&nbsp;first transect at ~11:00, second at ~13:00, and the third at ~15:00). During the surveys, we walked the transects at a constant pace, covering the entire length of the transect within an hour. Within the transect we also counted the number of individual flowers of each floral species once each day.&nbsp;</p>
<p><em>Bombus pascuorum</em> specimens</p>
<p>The bumblebee species <em>Bombus pascuorum</em> served as the sampling target and model pollinator for our study. <em>B. pascuorum</em> is common in Gorbeia and across Europe <span class="citation" data-cites="lecocq2015">(<a href="#ref-lecocq2015" role="doc-biblioref">Lecocq <em>et al.</em> 2015</a>)</span>. As an abundant and easily identifiable pollinator active during the field sampling period, the species was chosen as a target for genetic studies. For every period visit at each site, we captured <em>B. pascuorum</em> specimens for molecular analyses. We used clean specimen tubes to capture individuals when possible, and otherwise used insect nets to trap individuals before isolation in tubes. We collected up to five individuals for later use in DNA analyses at each site and period (N = 126). We brought specimens back from the field to BC3, and froze them at -22°C until processed.</p>
<p><em>Lab processing of B. pascuorum specimens, DNA extracts from guts and pollen</em></p>
<p>We extracted corbicular pollen and the digestive tract, or “gut”, of <em>B. pascuorum</em> individuals in the laboratory for molecular analyses. Before processing, we inspected all specimens for sex determination by observing genitalia, and we recorded specimen size by measuring the intertegular distance between wing bases using calipers. For each specimen, we harvested the entire insect gut, including the foregut, mid and hindgut, as well as the honey stomach. Additionally, if present, we collected pollen pellets from the corbicula of female specimens using clean forceps into 1.5 mL centrifuge tubes. Pollen samples were stored individually by specimen sample at -22°C. After measurement, gut and pollen extraction, we stored specimens in ethanol in 8 ml vials for storage at room temperature.</p>
<p><em>B. pascuorum specimen gut extraction</em></p>
<p>To extract the intestinal tract of bumblebee specimens, clean forceps were used to pull on the bumblebee stinger or genitalia, detaching the last segment of the abdomen, which was discarded. The forceps were then used to widen the abdomen cavity opening. By pulling slowly on the stinger or genitalia, the gut was then removed. If unable to remove up to the foregut as one continuous piece, the gut was removed section by section. If the intact nectar honey stomach was found intact, we also harvested it. Extracted gut contents were suspended in ethanol (&gt;96%), in 1.5 mL centrifuge tubes at -22°C until processed for DNA extraction.&nbsp;</p>
<p><em>Gut Content DNA extraction</em></p>
<p>Genomic DNA was extracted from <em>B. pascuorum</em> guts using the NucleoSpin Soil kit, using a modified version of the standard protocol. Guts stored in ethanol were first dried by briefly touching the gut to a clean piece of filter paper, then placing the gut in the kit provided tube used for the first extraction step (lysis). Tubes were left open at 22- 23°C for 5 minutes under hood controlled air circulation without entering the hood area in order to complete the drying process. Immediately afterwards, 700 µL of reagent SL1 was added to the tube as specified by the kit protocol before sealing the tube, without adding Enhancer SX. Gut samples were mechanically homogenized for ten minutes either using a Scientific Industries Vortex Genie 2, at the maximum setting, or sample homogenizer (Retsch MM 400), set to 30 Hz, with up to five minutes of additional disruption time added if gut samples remained unbroken after the first ten minutes. For all centrifuge steps, the centrifuge was set to 2,250 x g, and samples were centrifuged for 13 minutes where the protocol specified five minutes, and five minutes where the protocol specified two minutes. Apart from the specified changes, the recommended protocol was followed to obtain final DNA elutions of 100 µL with ideal DNA concentrations of greater than 25 ng/µL.</p>
<p><em>DNA extraction from corbicular pollen pellets</em></p>
<p>DNA was extracted from pollen pellets using the Machery-Nagel NucleoSpin 8 Food kit, including the additional initial steps of the kit’s supplementary protocol for pollen DNA extraction. The <em>B. pascuorum</em> specimens (N = 25) used for collecting pollen pellets typically carried 1-12 mg of pollen total between one or two pellets. Pollen was suspended in 200 µL of sterile water in a 2 mL reaction tube with metal beads before disruption. Samples were physically disrupted in sample homogenizer (Retsch MM 400) for two minutes at 30 Hz. After homogenization, the kit protocol was followed as recommended.&nbsp; We obtained final DNA elutions with a target volume of 60 µL, with ideal DNA concentrations greater than 10 ng/µL.</p>
<p><em>ITS2 Sequencing</em>&nbsp;</p>
<p>Our metabarcoding sequence library was built by amplifying and sequencing the internal transcribed spacer (ITS2) region of the ribosomal DNA in our extract samples. Samples were processed for sequencing library preparation and sequenced externally. We used existing primers for amplification of the ITS2 region <span class="citation" data-cites="cheng2016">White <em>et al.</em> (<a href="#ref-white1990" role="doc-biblioref">1990</a>)</span> <strong>(See a table? or supplementary material?)</strong> Libraries were sequenced on an Illumina platform to generate paired-end raw reads. We used demultiplexed raw sequence data, with primer and adapter sequences removed, in further bioinformatic analyses.</p>
<p><em>Bioinformatics: taxonomic assignment and contaminant analysis</em></p>
<p>Raw illumina sequences were processed using the DADA2 bioinformatics pipeline <span class="citation" data-cites="callahan2016">(<a href="#ref-callahan2016" role="doc-biblioref">Callahan <em>et al.</em> 2016</a>)</span>. Based on gut DNA sequence quality profiles, forward reads were&nbsp; trimmed at 240 base pairs and reverse reads at 210 bp. For consistency, all of the same parameters were used for pollen sequence bioinformatic processing. Maximum expected errors for forward reads was set to two, and a maximum of six was set for reverse reads. Minimum read overlap was set to 15 bp. In generating sequence tables, non-target-length sequences were filtered by removing sequences outside of the 240-450 bp range, and chimeras were removed using the default consensus method without pooling. Taxonomy was added to the ASVs using the reference sequence database from Bell (2021), which provided reference sequences at the species level for all but 21 of the species present in the study area, all of which were identifiable at least to genus level in the database. This database allowed for taxonomic assignment in two steps, using the assignTaxonomy() and addSpecies() functions of DADA2, although only identifications to the genus level were considered in further analysis. Taxonomy was classified using the pipeline’s internal method, “dada”, and a minimum bootstrap confidence of 50 was used for assigning taxonomic level.</p>
<p>ASV counts and taxonomy assigned in DADA2 were analyzed for contaminants using the decontam package in R <span class="citation" data-cites="davis2018">(<a href="#ref-davis2018" role="doc-biblioref">Davis <em>et al.</em> 2018</a>)</span>. ASVs were analyzed using the prevalence test, in which gut sequence samples were compared to six negative sequenced controls, and pollen sequence samples were compared to one negative sequence control. Given the low control sample number in the pollen sample analysis, the frequency test was also applied as the first screening measure. For the prevalence test, we set a contaminant determination threshold of 0.5. With this threshold value, any ASV with higher read frequency in the negative samples than in experimental samples was removed as a contaminant. Further, taxa detected in only one ASV of our sequence data were verified with a BLAST search, executed within the ITS2 Database <span class="citation" data-cites="ankenbrand2015">(<a href="#ref-ankenbrand2015" role="doc-biblioref">Ankenbrand <em>et al.</em> 2015</a>)</span>. The taxonomic identifications from the “decontaminated” ASVs were finally screened by comparing the list of detected taxa to the list of documented plant species within the study area. This screening was conducted under the supervision of a local botanist, incorporating expert knowledge into the evaluation.</p>
<p><em>Analysis of Networks at the Species Level</em></p>
<p>We calculated the values of network metrics at the species level for <em>B. pascuorum</em> as provided by the different methodologies. The degree of <em>B. pascuorum</em> as a network node was calculated as the number of plant genera detected across the 2023 sampling period by individual methodology. Centrality of plant taxa within the <em>B. pascuorum</em> interaction network was calculated for each methodology individually as the number of interactions <em>B. pascuorum</em> made with a specific taxon over the season divided by the total number of interactions observed over the season. For metabarcoding methodologies, interactions were quantified as the number of samples in which a plant taxon was detected. Finally, interaction specialization (<em>d’</em>) was calculated for <em>B. pascuorum</em> over sampling periods using the method of Blüthgen et al. <span class="citation" data-cites="blüthgen2006">(<a href="#ref-blüthgen2006" role="doc-biblioref">Blüthgen <em>et al.</em> 2006</a>)</span> as integrated into the R package bipartite <span class="citation" data-cites="bipartite">(<a href="#ref-bipartite" role="doc-biblioref">Dormann <em>et al.</em> 2009</a>)</span>.</p>
<p><strong>Results</strong></p>
<p><em>Interaction networks compared to floral diversity</em>&nbsp;</p>
<p>Comparing flower count surveys to the plant networks detected by the three interaction methodologies, we observed that B. pascuorum did not interact with the entirety of the available floral diversity within our transects (Figure 1). Flower counts identified a total of 117 flowering plant genera across the sampling season. Among these, 39 were not included within the interaction networks of any of the other methodologies. By inherent design, the 27 genera identified by interaction transects were a subset of the diversity observed by flower counts, amounting to 23% of the floral taxa identified across the season. Gut content metabarcoding detected 58% of the taxa observed by flower counts, and corbicular pollen metabarcoding detected 53%. Unlike the results of interaction transects, the taxa detected by metabarcoding methodologies were not restricted to those included within the potential interaction network indicated by flower counts.</p>
<p><em>Comparison of interaction methodology sensitivity</em></p>
<p>Both metabarcoding methodologies detected multiple unique taxa, while interaction transects detected fewer total taxa, with zero unique interactions (Figure 1). Gut content metabarcoding detected interactions with 131 total plant genera, including 33 taxa uniquely identified by this methodology. The total diversity identified by this methodology included all of the genera identified by interaction transects. Corbicular pollen metabarcoding detected 123 plant genera, with 31 uniquely detected genera. Again, these genera included all of those identified by interaction transects. The two metabarcoding methodologies shared 83 common plant genera, representing 67% of the total corbicular pollen diversity and 63% of the gut content diversity.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="../results/venn.figure.png" class="img-fluid figure-img"></p>
</figure>
</div>
</div>
</div>
<p>Taxonomic diversity observed by sampling period revealed different temporal patterns in flowering taxa and interaction diversity over the sampling season (Figure 2). While there was a net increase in available floral taxa and interactions from the first to last period, peak floral diversity was observed in period four, while peak interactions were observed in period six. Metabarcoding methodologies consistently detected more taxa than interaction transects, with gut content metabarcoding detecting more taxa than the other interaction methodologies in all periods. In periods one, two, and six, before and after the peak observed floral diversity, gut content metabarcoding detected more taxa than were observed in flower counts. During periods three through five, floral diversity was greater than gut content diversity</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-3-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><em>Functional diversity observations</em></p>
<p>While the taxa included in the interaction networks of flower counts and interaction transects only included taxa from the entomophilous community, both metabarcoding methodologies detected taxa from the anemophilous community as well. Anemophilous plant taxa represented 18% of the total identified plant taxa between the two methodologies, including 15 genera from Poaceae, nine tree genera, and seven other herbaceous genera <strong>(Supplementary Figure 1, Supplementary Table 1)</strong>.&nbsp;During the first, second, and sixth periods, when gut content metabarcoding detected more taxa than were observed in the entomophilous community of the transects, 15% of the taxa observed by gut metabarcoding were anemophilous or partially anemophilous <strong>(Supplementary data x).</strong></p>
<p><em>Plant community composition across methodologies</em></p>
<p>Plant communities detected by each methodology were compared for statistical evaluation of the differences between results. Binary presence-absence matrices were used to compare the communities detected by each methodology, aggregated by sampling day. Community composition was contrasted using the Raup-Crick dissimilarity index within a PERMANOVA test, including methodology as the independent variable (Table 1 - all results of PERMANOVAs). A PERMANOVA test <span class="citation" data-cites="vegan">(<a href="#ref-vegan" role="doc-biblioref">Oksanen <em>et al.</em> 2024</a>)</span> of the differences in taxonomic composition across methodologies&nbsp;indicated that methodology has a significant effect (P &lt; 0.001) on the detected plant community involved in pollinator interactions, accounting for 28% of the variation in detected interaction networks. In pairwise comparisons, the flower count communities differed significantly (P = 0.001) from those of all other methodologies. Between pairs of other methodologies, no significant differences were observed. Differences in methodologies were also assessed in an ordination using Non-metric Multidimensional Scaling (Figure 3, stress = 0.171). Average distances to the group centroid indicated by beta dispersal showed the highest dispersal of data in interaction transect results (Table? Supplementary?). An ANOVA test of mean dispersal by methodology indicated a significant difference (P &lt; 0.001) in dispersal attributed to methodology.&nbsp;</p>
<div class="cell">
<div class="cell-output-display">
<table class="lightable-minimal caption-top table table-sm table-striped small" data-quarto-postprocess="true">
<caption>PERMANOVA analysis of interaction methodology's effect on observed plant community</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th"></th>
<th style="text-align: right;" data-quarto-table-cell-role="th">Df</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">SumOfSqs</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">R2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">F</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">Pr(&gt;F)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Model</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">0.217003</td>
<td style="text-align: right;">0.0134103</td>
<td style="text-align: right;">0.6524416</td>
<td style="text-align: right;">0.6477</td>
</tr>
<tr class="even">
<td style="text-align: left;">Residual</td>
<td style="text-align: right;">96</td>
<td style="text-align: right;">15.964866</td>
<td style="text-align: right;">0.9865897</td>
<td style="text-align: right;">NA</td>
<td style="text-align: right;">NA</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Total</td>
<td style="text-align: right;">98</td>
<td style="text-align: right;">16.181869</td>
<td style="text-align: right;">1.0000000</td>
<td style="text-align: right;">NA</td>
<td style="text-align: right;">NA</td>
</tr>
</tbody>
</table>
</div>
<div class="cell-output-display">
<table class="lightable-minimal caption-top table table-sm table-striped small" data-quarto-postprocess="true">
<caption>Pairwise PERMANOVA Results</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th">Group 1</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Group 2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">R²</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">F value</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">df1</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">df2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">p value</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">Adjusted p value</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Significance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">count</td>
<td style="text-align: left;">gut.metabarcoding</td>
<td style="text-align: right;">0.534</td>
<td style="text-align: right;">161.78</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.000</td>
<td style="text-align: right;">0.001</td>
<td style="text-align: left;">***</td>
</tr>
<tr class="even">
<td style="text-align: left;">count</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">0.230</td>
<td style="text-align: right;">37.67</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.000</td>
<td style="text-align: right;">0.001</td>
<td style="text-align: left;">***</td>
</tr>
<tr class="odd">
<td style="text-align: left;">count</td>
<td style="text-align: left;">pollen.metabarcoding</td>
<td style="text-align: right;">0.372</td>
<td style="text-align: right;">63.87</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.000</td>
<td style="text-align: right;">0.001</td>
<td style="text-align: left;">***</td>
</tr>
<tr class="even">
<td style="text-align: left;">gut.metabarcoding</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">0.010</td>
<td style="text-align: right;">0.79</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.555</td>
<td style="text-align: right;">1.000</td>
<td style="text-align: left;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;">gut.metabarcoding</td>
<td style="text-align: left;">pollen.metabarcoding</td>
<td style="text-align: right;">0.124</td>
<td style="text-align: right;">8.89</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.042</td>
<td style="text-align: right;">0.125</td>
<td style="text-align: left;"></td>
</tr>
<tr class="even">
<td style="text-align: left;">interaction</td>
<td style="text-align: left;">pollen.metabarcoding</td>
<td style="text-align: right;">-0.025</td>
<td style="text-align: right;">-1.16</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.998</td>
<td style="text-align: right;">1.000</td>
<td style="text-align: left;"></td>
</tr>
</tbody>
</table>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><em>Specimen level metabarcoding results</em></p>
<p>Taxonomic detections from the two metabarcoding methodologies were compared by individual specimens. For specimens providing both types of samples, fewer taxa were detected in gut content samples (mean = 12 genera, sd = 9) than in pollen samples (mean = 18 genera, sd = 7). Comparing the number of taxa identified by each specimen for both sample sources, 20% of the detected taxa (mean = 6 genera, sd = 3) were shared, on average. <strong>A PERMANOVA test of the differences in taxonomic composition between the two showed a significant effect (P &lt; 0.001) of metabarcoding sample source on the detected plant community within the resulting individual interaction networks. This effect accounted for 11% of the variation in detected interaction networks.</strong>&nbsp;</p>
<p><em>Species Level Interaction Network Metrics</em></p>
<p>We calculated the values of network metrics at the species level for <em>B. pascuorum</em> and the plant taxa within its interaction networks as constructed based on different the three interaction methodologies. <em>B. pascuorum</em> as interaction network node had the highest degree of 131 within the gut content metabarcoding network, followed by 123 in the corbicular pollen metabarcoding network. <em>B. pascuorum</em> had the lowest degree in the interaction transect network, with a value of 27. <em>B. pascuorum</em> interaction specialization, calculated as <em>d’,</em> decreased across the sampling season when calculated based on interaction transect results and pollen metabarcoding results, while specialization maintained more or less stable for gut content metabarcoding results <strong>(Figure ).</strong> Interaction transect data indicated complete specialization in the first period. Centrality of the plant taxa within the <em>B. pascuorum</em> interaction networks from different interaction methodologies identified the <em>Lotus</em> as the most central plant genus in the networks from each methodology. Apart from this common result, the most central species in each network varied (<strong>FIGURE)</strong>. The two metabarcoding networks showed weaker centrality around the top plant genera relative to the interaction transect network.</p>
<p><strong>Discussion</strong></p>
<p>Bumblebees (<em>Bombus spp.</em>) are excellent model pollinators for numerous reasons. On top of the importance of bees in pollen transport relative to other pollinators <span class="citation" data-cites="alarcón2010">(<a href="#ref-alarcón2010" role="doc-biblioref">Alarcón 2010</a>)</span>, bumblebees specifically offer the advantage of abundance and ease of observation in the wild, as well as relevance due to their common use in agricultural settings <span class="citation" data-cites="velthuis2006">(<a href="#ref-velthuis2006" role="doc-biblioref">Velthuis &amp; Doorn 2006</a>)</span> and climate dependence <span class="citation" data-cites="sirois-delisle2018">(<a href="#ref-sirois-delisle2018" role="doc-biblioref">Sirois-Delisle &amp; Kerr 2018</a>)</span> <span class="citation" data-cites="maebe2021">(<a href="#ref-maebe2021" role="doc-biblioref">Maebe <em>et al.</em> 2021</a>)</span>. Given the quantity of existing research on bumblebees and their shared ecology with other pollinators, their environmental and nutritional needs are already broadly defined. Bees generally need a combination of pollen and nectar to meet their nutritional needs <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Pollen is the main source of protein and fats in the bee diet [<span class="citation" data-cites="vaudo2015">Vaudo (<a href="#ref-vaudo2015" role="doc-biblioref">2015</a>)</span>]<span class="citation" data-cites="steffan2019">(<a href="#ref-steffan2019" role="doc-biblioref">Steffan <em>et al.</em> 2019</a>)</span><span class="citation" data-cites="dharampal2019">(<a href="#ref-dharampal2019" role="doc-biblioref">Dharampal <em>et al.</em> 2019</a>)</span>, and diversity of pollen sources promotes healthy individual physiology <span class="citation" data-cites="dipasquale2016">(<a href="#ref-dipasquale2016" role="doc-biblioref">Di Pasquale <em>et al.</em> 2016</a>)</span>. For this reason it is generally understood that plant diversity supports bee populations, a notion that is supported by the observed effect of landscape composition on the bee microbiome and its implications for individual health [<span class="citation" data-cites="bosmans2018">Bosmans <em>et al.</em> (<a href="#ref-bosmans2018" role="doc-biblioref">2018</a>)</span>]<span class="citation" data-cites="jones2018">(<a href="#ref-jones2018" role="doc-biblioref">Jones <em>et al.</em> 2018</a>)</span>. While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interaction networks, including the implications of different interaction types for plants and pollinators, interaction patterns over time and space, and the functions of interaction participants. Our analysis uses the bumblebee, <em>Bombus pascuorum</em> as a model species for investigating how plant-pollinator interaction observation methodologies can provide insight on interaction specifics.</p>
<p>We evaluated the independent and complementary abilities of three pollinator-perspective based methodologies for observation of <em>B. pascuorum</em> plant interactions, contextualized by surveys of floral resource availability. We compared each methodology in terms of the diversity of interactions and our relative sampling effort in obtaining these results. Both metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity. Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Comparing the two metabarcoding methodologies, gut content metabarcoding outperformed corbicular pollen metabarcoding in terms of taxonomic diversity captured for each sampling period, and over all. The number of gut samples in 2023 was equal to the number of individuals captured (n = 126), while not all captured individuals carried sufficient pollen for genomic DNA extraction (n = 25). Given this, gut content metabarcoding was the more efficient methodology. The temporal distribution of taxonomic diversity observed by gut content metabarcoding was also the key dataset in observing seasonal forage patterns. We observed more forage taxa present in the gut contents than were flowering within our transects in the early and late parts of the sampling season (Figure 2), on either end of the mid-summer floral peak. This suggested that the composition of the plant community present in the forage network was likely distinct from that of the floral community present within our transects.</p>
<p>We also compared interaction network topology as described by each methodology, analyzing the difference in the plant taxa included in the interaction network of <em>B. pascuorum</em> as detected by each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Between the three interaction methodologies, no statistical differences were found, although the data dispersal differences were significant between the interaction observations and metabarcoding methodologies. <strong>This was likely a result of..</strong>. Both metabarcoding methodologies overlapped minimally with the flower count community and showed a statistical difference to the results of this methodology. This supported the notion that the interaction network included parts of the plant community beyond the floral taxa observed within our transects. This was partially to be expected, given that the metabarcoding methodologies and flower counts had a core difference in purpose. The latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.</p>
<p>Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span>, <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay <em>et al.</em> 2022</a>)</span>,<span class="citation" data-cites="smart2017">(<a href="#ref-smart2017" role="doc-biblioref">Smart <em>et al.</em> 2017</a>)</span>, metabarcoding increased observed interaction diversity by more than six-fold compared to interaction transect results. Metabarcoding also detected 93 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This, along with the temporal data, suggests that a large number of taxa within the <em>B. pascuorum</em> interaction network lie either beyond the spatial extent of our transects or outside of the plant community traditionally sampled in plant-pollinator network surveys. Our reference database for metabarcode sequences included 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences <span class="citation" data-cites="bell2021">(<a href="#ref-bell2021" role="doc-biblioref">Bell 2021</a>)</span>, allowing metabarcoding to identify taxa from functional groups beyond the floral community sampled in our transects. Through metabarcoding we observed interactions with a variety of taxa outside of the entomophilous meadow and shrublant plant community. Among these were included trees and shrubs (n = 9), grasses (n =15), and other herbaceous plants (n = 7), indicating a different foraging pattern than was expected for <em>B. pascuorum.</em> The diversity of plant types observed within our metabarcoding data indicated that <em>B. pascuorum</em> forages a greater diversity and spatial range than previously expected.</p>
<p>Our observations of interactions with the anemophilous community are supported by similar previously documented interactions <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere <em>et al.</em> 2017</a>)</span> <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span> [<span class="citation" data-cites="terrell1984">Terrell &amp; Batra (<a href="#ref-terrell1984" role="doc-biblioref">1984</a>)</span>]<span class="citation" data-cites="pojar1973">(<a href="#ref-pojar1973" role="doc-biblioref">Pojar 1973</a>)</span>, and have especially intriguing implications in the context of gut content metabarcoding observations. Previous studies using external pollen metabarcoding have removed wind-pollinated taxa from their analyses (e.g.<span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span>, <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span>). This is understandable, given the argument that wind-borne pollen in samples may represent false positive interactions <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span>, <span class="citation" data-cites="negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri <em>et al.</em> 2015</a>)</span>. Our gut content results, however, suggest that the practice removing these taxa as contaminants can be a large oversight, especially when using external pollen loads as standalone proxies for forage networks. Our observation of DNA from anemophilous taxa within gut samples suggest that interactions with these taxa are more than coincidental interactions with wind-borne pollen. Bumblebees are selective foragers that can assess the nutritional content of pollen, and make forage choices based on these assessments <span class="citation" data-cites="leonhardt2012">(<a href="#ref-leonhardt2012" role="doc-biblioref">Leonhardt &amp; Blüthgen 2012</a>)</span>. Multiple studies have concluded that bumblebees use this selective ability to forage for pollen quality, rather than quantity <span class="citation" data-cites="ruedenauer2016">(<a href="#ref-ruedenauer2016" role="doc-biblioref">Ruedenauer <em>et al.</em> 2016</a>)</span>,<span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span>, which has been used as an explanation for the adaptability of Bumblebee diets in response to environmental variability <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span>. Our detection of DNA from anemophilous pollen sources indicate intentional forage interactions with these taxa as a means of meeting the nutritional needs for adult bumblebees.</p>
<p>Existing hypotheses for pollinator forage adaptations in response to environmental changes have suggested that bees expand forage diversity beyond the flowering community and across habitats in order to survive annual “hunger gaps” <span class="citation" data-cites="timberlake2024b">(<a href="#ref-timberlake2024b" role="doc-biblioref">Timberlake <em>et al.</em> 2024a</a>)</span> <span class="citation" data-cites="becher2024">(<a href="#ref-becher2024" role="doc-biblioref">Becher <em>et al.</em> 2024</a>)</span>, when blooming floral species are limited <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span>. Our observation of high forage diversity in gut contents before and after the floral peak, distinct interaction and flowering taxa network topologies, and consumption of taxa across functional groups, all together support these hypotheses. While the community beyond the physical area of our transects likely played a large role in these observations, the detection of anemophilous taxa in gut contents during the periods where forage diversity was higher than flowering diversity provide evidence for a community driven component as well. These observations show how the broader taxonomic detection capacity of metabarcoding allows for detection of interactions that otherwise would go unobserved by flower visitation surveys. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.</p>
<p>Our comparative analysis understates the resolution of our metabarcoding data. We chose to aggregate our metabarcoding data by sampling day while comparing community composition across methodologies in order to balance sampling effort. By doing so, we disregarded the opportunity to study interactions at the individual level offered by metabarcoding <span class="citation" data-cites="gous2019">(<a href="#ref-gous2019" role="doc-biblioref">Gous <em>et al.</em> 2019</a>)</span> <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">Bell <em>et al.</em> 2016</a>)</span>. To demonstrate the value of this resolution, we compared the interaction data from individual specimens with both pollen and gut content samples. The low proportion of shared taxa between both sample sources at the individual level indicated a difference in the communities detected within both sample sources that was previously unobserved when methodologies were compared at the sampling day level. To clarify the different indications of these results, we also tested the difference in community composition between both methodologies at the individual level, this time observing a significant difference between the two methodologies (Table _ ).</p>
<p>The differences and commonalities in the communities detected by each methodology may reflect the result of selective interactions made by <em>B. pascuorum</em>. Bumblebees may forage pollen for immediate consumption, or transport pollen on the corbiculae to the colony nest for larval consumption <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Focusing on similarities between the two individual level datasets, the repeated detection of certain taxa by both methodologies increased confidence in these identifications, again demonstrating the complementary potential of these methodologies. For example, <em>Vicia</em> was observed by both methodologies across all specimens with both sample types in the first three periods. This adds to our previous knowledge from interaction transects of the strong relationship between <em>Vicia pyrenaica</em> and <em>B. pascuorum</em> (Artamendi, unpublished data), demonstrating the value of field surveys in validating laboratory based methodologies.</p>
<p>Our results suggest complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based <span class="citation" data-cites="mantegna2024">(<a href="#ref-mantegna2024" role="doc-biblioref">Mantegna 2024</a>)</span> <span class="citation" data-cites="nanglu2023">(<a href="#ref-nanglu2023" role="doc-biblioref">Nanglu <em>et al.</em> 2023</a>)</span>, and as molecular tools mature in their reliability <span class="citation" data-cites="richardson2015">(<a href="#ref-richardson2015" role="doc-biblioref">Richardson <em>et al.</em> 2015</a>)</span> <strong>NOT the best source for this</strong>. Given interaction transects’ disadvantage in sampling efficiency, incorporating this methodology as a validation of others may be the best way to integrate this methodology into future studies. Field observations can fill gaps left by metabarcoding methodologies, such as confirmation of pollination efficacy in interactions, or interaction frequency. As the external pollen load is a direct observation of the pollen that can be transferred between flowers, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. Pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval growth, given that external pollen will be returned to the nest to feed developing larvae <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. In a similar manner, gut content metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity <span class="citation" data-cites="li2025">(<a href="#ref-li2025" role="doc-biblioref">Li <em>et al.</em> 2025</a>)</span>. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen <em>et al.</em> 2021</a>)</span>, including parasite and disease transfer <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon <em>et al.</em> 2024</a>)</span>. Although they are not equal, our research highlights overall that each methodology offers advantages and disadvantages in terms of sensitivity, sampling effort, and perspective.</p>
<p>Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. <span class="citation" data-cites="magrach2023">(<a href="#ref-magrach2023" role="doc-biblioref">Magrach <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese <em>et al.</em> 2024</a>)</span>). We present gut content metabarcoding as a promising new methodology for observing plant-pollinator interactions, but we advocate for pairing this methodology with others. Apart from pairing with other plant-pollinator interaction methodologies, the ability to pair gut content metabarcoding with other microbiome studies is highly intriguing for revealing linked interactions within the plant-pollinator-microbe network. The microbial exchange involved in plant pollinator interactions is an important component of pollinator interactions that is yet to be fully explored <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen <em>et al.</em> 2021</a>)</span>. Further application of gut content metabarcoding to plant-pollinator interaction studies will hopefully uncover new cases of synergy with other analyses, as well as further methods of interpreting the results of this methodology. For example, the ability to quantify interaction frequency using metabarcoding at the individual level would be a large advance in the applicability of metabarcoding for gut content or pollen samples. Given our observations, we conclude that the keys to making such methodological advances are likely to lie in finding the connections with similar data sources that can fill the gaps in the information provided by our current methodologies.</p>


<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-alarcón2010" class="csl-entry" role="listitem">
Alarcón, R. (2010). <a href="https://doi.org/10.1111/j.1600-0706.2009.17694.x">Congruence between visitation and pollen-transport networks in a California plant<span></span>pollinator community</a>. <em>Oikos</em>, <strong>119</strong>, 35–44.
</div>
<div id="ref-anderson2013" class="csl-entry" role="listitem">
Anderson, K.E., Sheehan, T.H., Mott, B.M., Maes, P., Snyder, L., Schwan, M.R., Walton, A., Jones, B.M. &amp; Corby-Harris, V. (2013). <a href="https://doi.org/10.1371/journal.pone.0083125">Microbial Ecology of the Hive and Pollination Landscape: Bacterial Associates from Floral Nectar, the Alimentary Tract and Stored Food of Honey Bees (Apis mellifera)</a>. <em>PLOS ONE</em>, <strong>8</strong>, e83125.
</div>
<div id="ref-ankenbrand2015" class="csl-entry" role="listitem">
Ankenbrand, M.J., Keller, A., Wolf, M., Schultz, J. &amp; Förster, F. (2015). <a href="https://doi.org/10.1093/molbev/msv174">ITS2 Database V: Twice as Much: Table 1.</a> <em>Molecular Biology and Evolution</em>, <strong>32</strong>, 3030–3032.
</div>
<div id="ref-arstingstall2021" class="csl-entry" role="listitem">
Arstingstall, K.A., DeBano, S.J., Li, X., Wooster, D.E., Rowland, M.M., Burrows, S. &amp; Frost, K. (2021). <a href="https://doi.org/10.1111/mec.16112">Capabilities and limitations of using DNA metabarcoding to study plant<span></span>pollinator interactions</a>. <em>Molecular Ecology</em>, <strong>30</strong>, 5266–5297.
</div>
<div id="ref-artamendi2025" class="csl-entry" role="listitem">
Artamendi, M., Martin, P.A., Bartomeus, I. &amp; Magrach, A. (2025). <a href="https://doi.org/10.1038/s41559-024-02595-2">Loss of pollinator diversity consistently reduces reproductive success for wild and cultivated plants</a>. <em>Nature Ecology &amp; Evolution</em>, <strong>9</strong>, 296–313.
</div>
<div id="ref-baksay2022" class="csl-entry" role="listitem">
Baksay, S., Andalo, C., Galop, D., Burrus, M., Escaravage, N. &amp; Pornon, A. (2022). <a href="https://doi.org/10.3389/fevo.2022.735588">Using metabarcoding to investigate the strength of plant-pollinator interactions from surveys of visits to DNA sequences</a>. <em>Frontiers in Ecology and Evolution</em>, <strong>10</strong>.
</div>
<div id="ref-baksay2020" class="csl-entry" role="listitem">
Baksay, S., Pornon, A., Burrus, M., Mariette, J., Andalo, C. &amp; Escaravage, N. (2020). <a href="https://doi.org/10.1038/s41598-020-61198-6">Experimental quantification of pollen with DNA metabarcoding using ITS1 and trnL</a>. <em>Scientific Reports</em>, <strong>10</strong>, 4202.
</div>
<div id="ref-becher2024" class="csl-entry" role="listitem">
Becher, M.A., Twiston-Davies, G., Osborne, J.L. &amp; Lander, T.A. (2024). <a href="https://doi.org/10.1111/icad.12736">Resource gaps pose the greatest threat for bumblebees during the colony establishment phase</a>. <em>Insect Conservation and Diversity</em>, <strong>17</strong>, 676–689.
</div>
<div id="ref-bell2021" class="csl-entry" role="listitem">
Bell, K. (2021). <a href="https://doi.org/10.6084/M9.FIGSHARE.14936004.V1">ITS2 july 2021</a>.
</div>
<div id="ref-bell2017" class="csl-entry" role="listitem">
Bell, K.L., Fowler, J., Burgess, K.S., Dobbs, E.K., Gruenewald, D., Lawley, B., Morozumi, C. &amp; Brosi, B.J. (2017). <a href="https://doi.org/10.3732/apps.1600124">Applying pollen DNA metabarcoding to the study of plant<span></span>pollinator interactions</a>. <em>Applications in Plant Sciences</em>, <strong>5</strong>, 1600124.
</div>
<div id="ref-bell2016" class="csl-entry" role="listitem">
Bell, K.L., Vere, N. de, Keller, A., Richardson, R.T., Gous, A., Burgess, K.S. &amp; Brosi, B.J. (2016). <a href="https://doi.org/10.1139/gen-2015-0200">Pollen DNA barcoding: Current applications and future prospects</a>. <em>Genome</em>, <strong>59</strong>, 629–640.
</div>
<div id="ref-blüthgen2006" class="csl-entry" role="listitem">
Blüthgen, N., Menzel, F. &amp; Blüthgen, N. (2006). <a href="https://doi.org/10.1186/1472-6785-6-9">Measuring specialization in species interaction networks</a>. <em>BMC Ecology</em>, <strong>6</strong>, 9.
</div>
<div id="ref-bosch2009" class="csl-entry" role="listitem">
Bosch, J., Martín González, A.M., Rodrigo, A. &amp; Navarro, D. (2009). <a href="https://doi.org/10.1111/j.1461-0248.2009.01296.x">Plant<span></span>pollinator networks: adding the pollinator<span>’</span>s perspective</a>. <em>Ecology Letters</em>, <strong>12</strong>, 409–419.
</div>
<div id="ref-bosmans2018" class="csl-entry" role="listitem">
Bosmans, L., Pozo, M.I., Verreth, C., Crauwels, S., Wilberts, L., Sobhy, I.S., Wäckers, F., Jacquemyn, H. &amp; Lievens, B. (2018). <a href="https://doi.org/10.1371/journal.pone.0204612">Habitat-specific variation in gut microbial communities and pathogen prevalence in bumblebee queens (Bombus terrestris)</a> (J.C. Nieh, Ed.). <em>PLOS ONE</em>, <strong>13</strong>, e0204612.
</div>
<div id="ref-callahan2016" class="csl-entry" role="listitem">
Callahan, B.J., McMurdie, P.J., Rosen, M.J., Han, A.W., Johnson, A.J.A. &amp; Holmes, S.P. (2016). <a href="https://doi.org/10.1038/nmeth.3869">DADA2: High resolution sample inference from illumina amplicon data</a>. <em>Nature methods</em>, <strong>13</strong>, 581–583.
</div>
<div id="ref-cheng2016" class="csl-entry" role="listitem">
Cheng, T., Xu, C., Lei, L., Li, C., Zhang, Y. &amp; Zhou, S. (2016). <a href="https://doi.org/10.1111/1755-0998.12438">Barcoding the kingdom Plantae: new PCR primers for <span> <em>ITS</em> </span> regions of plants with improved universality and specificity</a>. <em>Molecular Ecology Resources</em>, <strong>16</strong>, 138–149.
</div>
<div id="ref-chen2010" class="csl-entry" role="listitem">
Chen, S., Yao, H., Han, J., Liu, C., Song, J., Shi, L., Zhu, Y., Ma, X., Gao, T., Pang, X., Luo, K., Li, Y., Li, X., Jia, X., Lin, Y. &amp; Leon, C. (2010). <a href="https://doi.org/10.1371/journal.pone.0008613">Validation of the ITS2 Region as a Novel DNA Barcode for Identifying Medicinal Plant Species</a>. <em>PLOS ONE</em>, <strong>5</strong>, e8613.
</div>
<div id="ref-cullen2021" class="csl-entry" role="listitem">
Cullen, N., Fetters, A. &amp; Ashman, T.-L. (2021). <a href="https://doi.org/10.1016/j.cois.2020.11.002">Integrating microbes into pollination</a>. <em>Current Opinion in Insect Science</em>, <strong>44</strong>, 48–54.
</div>
<div id="ref-davis2018" class="csl-entry" role="listitem">
Davis, N.M., Proctor, D.M., Holmes, S.P., Relman, D.A. &amp; Callahan, B.J. (2018). <a href="https://doi.org/10.1186/s40168-018-0605-2">Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data</a>. <em>Microbiome</em>, <strong>6</strong>, 226.
</div>
<div id="ref-devriese2024" class="csl-entry" role="listitem">
Devriese, A., Peeters, G., Brys, R. &amp; Jacquemyn, H. (2024). <a href="https://doi.org/10.1002/aps3.11601">The impact of extraction method and pollen concentration on community composition for pollen metabarcoding</a>. <em>Applications in Plant Sciences</em>, <strong>12</strong>, e11601.
</div>
<div id="ref-dharampal2019" class="csl-entry" role="listitem">
Dharampal, P.S., Carlson, C., Currie, C.R. &amp; Steffan, S.A. (2019). <a href="https://doi.org/10.1098/rspb.2018.2894">Pollen-borne microbes shape bee fitness</a>. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <strong>286</strong>, 20182894.
</div>
<div id="ref-dipasquale2016" class="csl-entry" role="listitem">
Di Pasquale, G., Alaux, C., Conte, Y.L., Odoux, J.-F., Pioz, M., Vaissière, B.E., Belzunces, L.P. &amp; Decourtye, A. (2016). <a href="https://doi.org/10.1371/journal.pone.0162818">Variations in the Availability of Pollen Resources Affect Honey Bee Health</a>. <em>PLOS ONE</em>, <strong>11</strong>, e0162818.
</div>
<div id="ref-diputaciónforaldebizkaia2023" class="csl-entry" role="listitem">
Diputación Foral de Bizkaia. (2023). <a href="https://www.bizkaia.eus/es/tema-detalle/-/edukia/dt/2663">Puntos de Observación</a>.
</div>
<div id="ref-donald2022" class="csl-entry" role="listitem">
Donald, M.L., Galbraith, J.A., Erastova, D.A., Podolyan, A., Miller, T.E.X. &amp; Dhami, M.K. (2022). <a href="https://doi.org/10.1111/1462-2920.16159">Nectar resources affect bird-dispersed microbial metacommunities in suburban and rural gardens</a>. <em>Environmental Microbiology</em>, <strong>24</strong>, 5654–5665.
</div>
<div id="ref-donald2023" class="csl-entry" role="listitem">
Donald, M.L., San Juan, P.A. &amp; Dhami, M.K. (2023). <a href="https://doi.org/10.1080/03014223.2023.2245763">Microbial signatures of vertebrate visitation in floral nectar: a case study with two endemic Aotearoa New Zealand plant species</a>. <em>New Zealand Journal of Zoology</em>, 1–11.
</div>
<div id="ref-bipartite" class="csl-entry" role="listitem">
Dormann, C.F., Fruend, J., Bluethgen, N. &amp; Gruber, B. (2009). Indices, graphs and null models: Analyzing bipartite ecological networks. <strong>2</strong>, 7–24.
</div>
<div id="ref-evans2020" class="csl-entry" role="listitem">
Evans, D. &amp; Kitson, J. (2020). <a href="https://doi.org/10.1016/j.cois.2020.01.005">Molecular ecology as a tool for understanding pollination and other plant<span></span>insect interactions</a>. <em>Current Opinion in Insect Science</em>, <strong>38</strong>, 26–33.
</div>
<div id="ref-gous2019" class="csl-entry" role="listitem">
Gous, A., Swanevelder, D.Z.H., Eardley, C.D. &amp; Willows-Munro, S. (2019). <a href="https://doi.org/10.1111/eva.12707">Plant<span></span>pollinator interactions over time: Pollen metabarcoding from bees in a historic collection</a>. <em>Evolutionary Applications</em>, <strong>12</strong>, 187–197.
</div>
<div id="ref-haag2023" class="csl-entry" role="listitem">
Haag, K.L., Caesar, L., Silveira Regueira-Neto, M. da, Sousa, D.R. de, Montenegro Marcelino, V., Queiroz Balbino, V. de &amp; Torres Carvalho, A. (2023). <a href="https://doi.org/10.1007/s00248-022-02027-3">Temporal Changes in Gut Microbiota Composition and Pollen Diet Associated with Colony Weakness of a Stingless Bee</a>. <em>Microbial Ecology</em>, <strong>85</strong>, 1514–1526.
</div>
<div id="ref-hammer2021" class="csl-entry" role="listitem">
Hammer, T.J., Le, E., Martin, A.N. &amp; Moran, N.A. (2021). <a href="https://doi.org/10.1007/s00040-021-00837-1">The gut microbiota of bumblebees</a>. <em>Insectes Sociaux</em>, <strong>68</strong>, 287–301.
</div>
<div id="ref-jones2018" class="csl-entry" role="listitem">
Jones, J.C., Fruciano, C., Hildebrand, F., Al Toufalilia, H., Balfour, N.J., Bork, P., Engel, P., Ratnieks, F.L. &amp; Hughes, W.O. (2018). <a href="https://doi.org/10.1002/ece3.3597">Gut microbiota composition is associated with environmental landscape in honey bees</a>. <em>Ecology and Evolution</em>, <strong>8</strong>, 441–451.
</div>
<div id="ref-katumo2022" class="csl-entry" role="listitem">
Katumo, D.M., Liang, H., Ochola, A.C., Lv, M., Wang, Q.-F. &amp; Yang, C.-F. (2022). <a href="https://doi.org/10.1016/j.pld.2022.01.005">Pollinator diversity benefits natural and agricultural ecosystems, environmental health, and human welfare</a>. <em>Plant Diversity</em>, <strong>44</strong>, 429–435.
</div>
<div id="ref-klein2006" class="csl-entry" role="listitem">
Klein, A.-M., Vaissière, B.E., Cane, J.H., Steffan-Dewenter, I., Cunningham, S.A., Kremen, C. &amp; Tscharntke, T. (2006). <a href="https://doi.org/10.1098/rspb.2006.3721">Importance of pollinators in changing landscapes for world crops</a>. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <strong>274</strong>, 303–313.
</div>
<div id="ref-leach2018" class="csl-entry" role="listitem">
Leach, M.E. &amp; Drummond, F. (2018). <a href="https://doi.org/10.1155/2018/9607246">A Review of Native Wild Bee Nutritional Health</a>. <em>International Journal of Ecology</em>, <strong>2018</strong>, 9607246.
</div>
<div id="ref-lecocq2015" class="csl-entry" role="listitem">
Lecocq, T., Brasero, N., Martinet, B., Valterovà, I. &amp; Rasmont, P. (2015). <a href="https://doi.org/10.1111/syen.12137">Highly polytypic taxon complex: interspecific and intraspecific integrative taxonomic assessment of the widespread pollinator ombus pascuorum Scopoli 1763 (Hymenoptera: Apidae)</a>. <em>Systematic Entomology</em>, <strong>40</strong>, 881–890.
</div>
<div id="ref-leonhardt2012" class="csl-entry" role="listitem">
Leonhardt, S.D. &amp; Blüthgen, N. (2012). <a href="https://doi.org/10.1007/s13592-011-0112-y">The same, but different: pollen foraging in honeybee and bumblebee colonies</a>. <em>Apidologie</em>, <strong>43</strong>, 449–464.
</div>
<div id="ref-leontidou2021" class="csl-entry" role="listitem">
Leontidou, K., Vokou, D., Sandionigi, A., Bruno, A., Lazarina, M., De Groeve, J., Li, M., Varotto, C., Girardi, M., Casiraghi, M. &amp; Cristofori, A. (2021). <a href="https://doi.org/10.1038/s41598-021-97619-3">Plant biodiversity assessment through pollen DNA metabarcoding in Natura 2000 habitats (Italian Alps)</a>. <em>Scientific Reports</em>, <strong>11</strong>, 18226.
</div>
<div id="ref-leponiemi2023" class="csl-entry" role="listitem">
Leponiemi, M., Freitak, D., Moreno-Torres, M., Pferschy-Wenzig, E.-M., Becker-Scarpitta, A., Tiusanen, M., Vesterinen, E.J. &amp; Wirta, H. (2023). <a href="https://doi.org/10.1038/s41598-023-42102-4">Honeybees<span>’</span> foraging choices for nectar and pollen revealed by DNA metabarcoding</a>. <em>Scientific Reports</em>, <strong>13</strong>, 14753.
</div>
<div id="ref-lignon2024" class="csl-entry" role="listitem">
Lignon, V.A., Mas, F., Jones, E.E., Kaiser, C. &amp; Dhami, M.K. (2024). <a href="https://doi.org/10.1080/03014223.2024.2353285">The floral interface: a playground for interactions between insect pollinators, microbes, and plants</a>. <em>New Zealand Journal of Zoology</em>, 1–20.
</div>
<div id="ref-li2025" class="csl-entry" role="listitem">
Li, Y., Liu, C., Wang, Y., Li, M., Zou, S., Hu, X., Chen, Z., Li, M., Ma, C., Obi, C.J., Zhou, X., Zou, Y. &amp; Tang, M. (2025). <a href="https://doi.org/10.1111/1744-7917.70051">Urban wild bee well-being revealed by gut metagenome data: A mason bee model</a>. <em>Insect Science</em>, <strong>n/a</strong>.
</div>
<div id="ref-loidi2011" class="csl-entry" role="listitem">
Loidi, J., Campos, J.A., Buirrun, I. &amp; García-Mijangos, I. (2011). <em>La vegetación de la Comunidad Autónoma del País Vasco: leyenda del mapa de series de vegetación a escala 1:50.000</em>. Universidad del País Vasco = Euskal Herriko Unibertsitatea, Leioa, Vizcaya.
</div>
<div id="ref-lowe2022" class="csl-entry" role="listitem">
Lowe, A., Jones, L., Witter, L., Creer, S. &amp; Vere, N. de. (2022). <a href="https://doi.org/10.3390/d14040236">Using DNA Metabarcoding to Identify Floral Visitation by Pollinators</a>. <em>Diversity</em>, <strong>14</strong>, 236.
</div>
<div id="ref-maebe2021" class="csl-entry" role="listitem">
Maebe, K., Hart, A.F., Marshall, L., Vandamme, P., Vereecken, N.J., Michez, D. &amp; Smagghe, G. (2021). <a href="https://doi.org/10.1111/gcb.15751">Bumblebee resilience to climate change, through plastic and adaptive responses</a>. <em>Global Change Biology</em>, <strong>27</strong>, 4223–4237.
</div>
<div id="ref-magrach2023" class="csl-entry" role="listitem">
Magrach, A., Artamendi, M., Lapido, P.D., Parejo, C. &amp; Rubio, E. (2023). <a href="https://doi.org/10.1002/ecs2.4521">Indirect interactions between pollinators drive interaction rewiring through space</a>. <em>Ecosphere</em>, <strong>14</strong>, e4521.
</div>
<div id="ref-mantegna2024" class="csl-entry" role="listitem">
Mantegna, C. (2024). <a href="https://doi.org/10.1038/d41586-024-01178-2">Ecologists: don<span>’</span>t lose touch with the joy of fieldwork</a>. <em>Nature</em>, <strong>628</strong>, 692–692.
</div>
<div id="ref-mayr2021" class="csl-entry" role="listitem">
Mayr, A.V., Keller, A., Peters, M.K., Grimmer, G., Krischke, B., Geyer, M., Schmitt, T. &amp; Steffan-Dewenter, I. (2021). <a href="https://doi.org/10.1002/ece3.7605">Cryptic species and hidden ecological interactions of halictine bees along an elevational gradient</a>. <em>Ecology and Evolution</em>, <strong>11</strong>, 7700–7712.
</div>
<div id="ref-milla2022" class="csl-entry" role="listitem">
Milla, L., Schmidt-Lebuhn, A., Bovill, J. &amp; Encinas-Viso, F. (2022). <a href="https://doi.org/10.1002/2688-8319.12120">Monitoring of honey bee floral resources with pollen DNA metabarcoding as a complementary tool to vegetation surveys</a>. <em>Ecological Solutions and Evidence</em>, <strong>3</strong>, e12120.
</div>
<div id="ref-nanglu2023" class="csl-entry" role="listitem">
Nanglu, K., de&nbsp;Carle, D., Cullen, T.M., Anderson, E.B., Arif, S., Castañeda, R.A., Chang, L.M., Iwama, R.E., Fellin, E., Manglicmot, R.C., Massey, M.D. &amp; Astudillo-Clavijo, V. (2023). <a href="https://doi.org/10.1002/ece3.10621">The nature of science: The fundamental role of natural history in ecology, evolution, conservation, and education</a>. <em>Ecology and Evolution</em>, <strong>13</strong>, e10621.
</div>
<div id="ref-negri2015" class="csl-entry" role="listitem">
Negri, I., Mavris, C., Prisco, G.D., Caprio, E. &amp; Pellecchia, M. (2015). <a href="https://doi.org/10.1371/journal.pone.0132491">Honey Bees (Apis mellifera, L.) as Active Samplers of Airborne Particulate Matter</a>. <em>PLOS ONE</em>, <strong>10</strong>, e0132491.
</div>
<div id="ref-vegan" class="csl-entry" role="listitem">
Oksanen, J., Simpson, G.L., Blanchet, F.G., Kindt, R., Legendre, P., Minchin, P.R., O’Hara, R.B., Solymos, P., Stevens, M.H.H., Szoecs, E., Wagner, H., Barbour, M., Bedward, M., Bolker, B., Borcard, D., Carvalho, G., Chirico, M., De Caceres, M., Durand, S., Evangelista, H.B.A., FitzJohn, R., Friendly, M., Furneaux, B., Hannigan, G., Hill, M.O., Lahti, L., McGlinn, D., Ouellette, M.-H., Ribeiro Cunha, E., Smith, T., Stier, A., Ter Braak, C.J.F. &amp; Weedon, J. (2024). <a href="https://CRAN.R-project.org/package=vegan">Vegan: Community ecology package</a>.
</div>
<div id="ref-ollerton2017" class="csl-entry" role="listitem">
Ollerton, J. (2017). <a href="https://doi.org/10.1146/annurev-ecolsys-110316-022919">Pollinator Diversity: Distribution, Ecological Function, and Conservation</a>. <em>Annual Review of Ecology, Evolution, and Systematics</em>, <strong>48</strong>, 353–376.
</div>
<div id="ref-R-climaemet" class="csl-entry" role="listitem">
Pizarro, M., Hernangómez, D. &amp; Fernández-Avilés, G. (2021). <em><a href="https://doi.org/10.32614/CRAN.package.climaemet"><span class="nocase">climaemet</span>: Climate <span>AEMET</span> tools</a></em>.
</div>
<div id="ref-pojar1973" class="csl-entry" role="listitem">
Pojar, J. (1973). <a href="https://doi.org/10.2307/2424049">Pollination of typically anemophilous salt marsh plants by bumble bees, bombus terricola occidentalis grne</a>. <em>The American Midland Naturalist</em>, <strong>89</strong>, 448–451.
</div>
<div id="ref-pornon2017" class="csl-entry" role="listitem">
Pornon, A., Andalo, C., Burrus, M. &amp; Escaravage, N. (2017). <a href="https://doi.org/10.1038/s41598-017-16785-5">DNA metabarcoding data unveils invisible pollination networks</a>. <em>Scientific Reports</em>, <strong>7</strong>, 16828.
</div>
<div id="ref-richardson2015" class="csl-entry" role="listitem">
Richardson, R.T., Lin, C.-H., Quijia, J.O., Riusech, N.S., Goodell, K. &amp; Johnson, R.M. (2015). <a href="https://doi.org/10.3732/apps.1500043">Rank-based characterization of pollen assemblages collected by honey bees using a multi-locus metabarcoding approach</a>. <em>Applications in Plant Sciences</em>, <strong>3</strong>, 1500043.
</div>
<div id="ref-ruedenauer2016" class="csl-entry" role="listitem">
Ruedenauer, F.A., Spaethe, J. &amp; Leonhardt, S.D. (2016). <a href="https://doi.org/10.1007/s00265-016-2129-8">Hungry for quality<span></span>individual bumblebees forage flexibly to collect high-quality pollen</a>. <em>Behavioral Ecology and Sociobiology</em>, <strong>70</strong>, 1209–1217.
</div>
<div id="ref-saunders2018" class="csl-entry" role="listitem">
Saunders, M.E. (2018). <a href="https://doi.org/10.1111/icad.12243">Insect pollinators collect pollen from wind-pollinated plants: implications for pollination ecology and sustainable agriculture</a>. <em>Insect Conservation and Diversity</em>, <strong>11</strong>, 13–31.
</div>
<div id="ref-selva2024" class="csl-entry" role="listitem">
Selva, S., Moretti, M., Ruedenauer, F., Keller, A., Fournier, B., Leonhardt, S.D., Eggenberger, H.A. &amp; Abella, J.C. (2024). <a href="https://ecoevorxiv.org/repository/view/7812/">Urban bumblebees diversify their foraging strategy to maintain nutrient intake</a>.
</div>
<div id="ref-sirois-delisle2018" class="csl-entry" role="listitem">
Sirois-Delisle, C. &amp; Kerr, J.T. (2018). <a href="https://doi.org/10.1038/s41598-018-32665-y">Climate change-driven range losses among bumblebee species are poised to accelerate</a>. <em>Scientific Reports</em>, <strong>8</strong>, 14464.
</div>
<div id="ref-smart2017" class="csl-entry" role="listitem">
Smart, M.D., Cornman, R.S., Iwanowicz, D.D., McDermott-Kubeczko, M., Pettis, J.S., Spivak, M.S. &amp; Otto, C.R.V. (2017). <a href="https://doi.org/10.1093/ee/nvw159">A comparison of honey bee-collected pollen from working agricultural lands using light microscopy and ITS metabarcoding</a>. <em>Environmental Entomology</em>, <strong>46</strong>, 38–49.
</div>
<div id="ref-steffan2019" class="csl-entry" role="listitem">
Steffan, S.A., Dharampal, P.S., Danforth, B.N., Gaines-Day, H.R., Takizawa, Y. &amp; Chikaraishi, Y. (2019). <a href="https://doi.org/10.1086/704281">Omnivory in bees: Elevated trophic positions among all major bee families</a>. <em>The American Naturalist</em>, <strong>194</strong>, 414–421.
</div>
<div id="ref-tanaka2020" class="csl-entry" role="listitem">
Tanaka, K., Nozaki, A., Nakadai, H., Shiwa, Y. &amp; Shimizu-Kadota, M. (2020). <a href="https://doi.org/10.1186/s13104-020-05361-2">Using pollen DNA metabarcoding to profile nectar sources of urban beekeeping in K<span>ō</span>t<span>ō</span>-ku, Tokyo</a>. <em>BMC Research Notes</em>, <strong>13</strong>, 515.
</div>
<div id="ref-terrell1984" class="csl-entry" role="listitem">
Terrell, E.E. &amp; Batra, S.W.T. (1984). <a href="https://www.jstor.org/stable/4033059">Insects collect pollen of eastern wildrice, zizania aquatica (poaceae)</a>. <em>Castanea</em>, <strong>49</strong>, 31–34.
</div>
<div id="ref-timberlake2024b" class="csl-entry" role="listitem">
Timberlake, T.P., Tew, N.E. &amp; Memmott, J. (2024a). <a href="https://doi.org/10.1098/rspb.2024.1523">Gardens reduce seasonal hunger gaps for farmland pollinators</a>. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <strong>291</strong>.
</div>
<div id="ref-timberlake2024" class="csl-entry" role="listitem">
Timberlake, T.P., Vere, N. de, Jones, L.E., Vaughan, I.P., Baude, M. &amp; Memmott, J. (2024b). <a href="https://doi.org/10.1002/2688-8319.12360">Ten-a-day: Bumblebee pollen loads reveal high consistency in foraging breadth among species, sites and seasons</a>. <em>Ecological Solutions and Evidence</em>, <strong>5</strong>, e12360.
</div>
<div id="ref-vanbergen2013" class="csl-entry" role="listitem">
Vanbergen, A.J. &amp; Insect Pollinators Initiative, the. (2013). <a href="https://doi.org/10.1890/120126">Threats to an ecosystem service: pressures on pollinators</a>. <em>Frontiers in Ecology and the Environment</em>, <strong>11</strong>, 251–259.
</div>
<div id="ref-vaudo2015" class="csl-entry" role="listitem">
Vaudo, A.D. (2015). <a href="https://doi.org/10.1016/j.cois.2015.05.008">Bee nutrition and floral resource restoration</a>. <em>Current Opinion in Insect Science</em>, <strong>10</strong>, 133–141.
</div>
<div id="ref-devega2021" class="csl-entry" role="listitem">
Vega, C. de, Álvarez-Pérez, S., Albaladejo, R.G., Steenhuisen, S.-L., Lachance, M.-A., Johnson, S.D. &amp; Herrera, C.M. (2021). <a href="https://doi.org/10.1111/1365-2745.13726">The role of plant<span></span>pollinator interactions in structuring nectar microbial communities</a>. <em>Journal of Ecology</em>, <strong>109</strong>, 3379–3395.
</div>
<div id="ref-velthuis2006" class="csl-entry" role="listitem">
Velthuis, H.H.W. &amp; Doorn, A. van. (2006). <a href="https://doi.org/10.1051/apido:2006019">A century of advances in bumblebee domestication and the economic and environmental aspects of its commercialization for pollination</a>. <em>Apidologie</em>, <strong>37</strong>, 421–451.
</div>
<div id="ref-devere2017" class="csl-entry" role="listitem">
Vere, N. de, Jones, L.E., Gilmore, T., Moscrop, J., Lowe, A., Smith, D., Hegarty, M.J., Creer, S. &amp; Ford, C.R. (2017). <a href="https://doi.org/10.1038/srep42838">Using DNA metabarcoding to investigate honey bee foraging reveals limited flower use despite high floral availability</a>. <em>Scientific Reports</em>, <strong>7</strong>, 42838.
</div>
<div id="ref-white1990" class="csl-entry" role="listitem">
White, T.J., Bruns, T.D., Lee, S.B. &amp; Taylor, J.W. (1990). <a href="https://www.researchgate.net/publication/223397588_White_T_J_T_D_Bruns_S_B_Lee_and_J_W_Taylor_Amplification_and_direct_sequencing_of_fungal_ribosomal_RNA_Genes_for_phylogenetics">Amplification and direct sequencing of fungal ribosomal RNA Genes for phylogenetics</a>. pp. 315–322. Academic Press.
</div>
<div id="ref-woodcock2019" class="csl-entry" role="listitem">
Woodcock, B.A., Garratt, M.P.D., Powney, G.D., Shaw, R.F., Osborne, J.L., Soroka, J., Lindström, S.a.M., Stanley, D., Ouvrard, P., Edwards, M.E., Jauker, F., McCracken, M.E., Zou, Y., Potts, S.G., Rundlöf, M., Noriega, J.A., Greenop, A., Smith, H.G., Bommarco, R., Werf, W. van der, Stout, J.C., Steffan-Dewenter, I., Morandin, L., Bullock, J.M. &amp; Pywell, R.F. (2019). <a href="https://doi.org/10.1038/s41467-019-09393-6">Meta-analysis reveals that pollinator functional diversity and abundance enhance crop pollination and yield</a>. <em>Nature Communications</em>, <strong>10</strong>, 1481.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>