<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Interaction methodologies</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="interaction_methodologies_files/libs/clipboard/clipboard.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/quarto.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/popper.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/anchor.min.js"></script>
<link href="interaction_methodologies_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="interaction_methodologies_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-1bc8a17f135ab3d594c857e9f48e611b.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Interaction methodologies</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><strong>Discussion</strong></p>
<p>Bumblebees (<em>Bombus spp.</em>) are excellent model pollinators for numerous reasons. On top of the importance of bees in pollen transport relative to other pollinators <span class="citation" data-cites="alarcón2010">(<a href="#ref-alarcón2010" role="doc-biblioref">Alarcón 2010</a>)</span>, bumblebees specifically offer the advantage of abundance and ease of observation in the wild, as well as relevance due to their common use in agricultural settings <span class="citation" data-cites="velthuis2006">(<a href="#ref-velthuis2006" role="doc-biblioref">Velthuis &amp; Doorn 2006</a>)</span> and climate dependence <span class="citation" data-cites="sirois-delisle2018">(<a href="#ref-sirois-delisle2018" role="doc-biblioref">Sirois-Delisle &amp; Kerr 2018</a>)</span> <span class="citation" data-cites="maebe2021">(<a href="#ref-maebe2021" role="doc-biblioref">Maebe <em>et al.</em> 2021</a>)</span>. Given the quantity of existing research on bumblebees and their shared ecology with other pollinators, their environmental and nutritional needs are already broadly defined. Bees generally need a combination of pollen and nectar to meet their nutritional needs <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Pollen is the main source of protein and fats in the bee diet [<span class="citation" data-cites="vaudo2015">Vaudo (<a href="#ref-vaudo2015" role="doc-biblioref">2015</a>)</span>]<span class="citation" data-cites="steffan2019">(<a href="#ref-steffan2019" role="doc-biblioref">Steffan <em>et al.</em> 2019</a>)</span><span class="citation" data-cites="dharampal2019">(<a href="#ref-dharampal2019" role="doc-biblioref">Dharampal <em>et al.</em> 2019</a>)</span>, and diversity of pollen sources promotes healthy individual physiology <span class="citation" data-cites="dipasquale2016">(<a href="#ref-dipasquale2016" role="doc-biblioref">Di Pasquale <em>et al.</em> 2016</a>)</span>. For this reason it is generally understood that plant diversity supports bee populations, a notion that is supported by the observed effect of landscape composition on the bee microbiome and its implications for individual health [<span class="citation" data-cites="bosmans2018">Bosmans <em>et al.</em> (<a href="#ref-bosmans2018" role="doc-biblioref">2018</a>)</span>]<span class="citation" data-cites="jones2018">(<a href="#ref-jones2018" role="doc-biblioref">Jones <em>et al.</em> 2018</a>)</span>. While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interaction networks, including the implications of different interaction tpyes for plants and pollinators, interaction patterns over time and space, and the functions of interaction participants. Our analysis uses the bumblebee, <em>Bombus pascuorum</em> as a model species for investigating how plant-pollinator interaction observation methodologies can provide insight on interaction specifics.</p>
<p>We evaluated the independent and complementary abilities of three pollinator-perspective based methodologies for observation of <em>B. pascuorum</em> plant interactions, contextualized by surveys of floral resource availability. We compared each methodology in terms of the diversity of interactions and our relative sampling effort in obtaining these results. Both metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity. Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Comparing the two metabarcoding methodologies, gut content metabarcoding outperformed corbicular pollen metabarcoding in terms of taxonomic diversity captured for each sampling period, and over all. The number of gut samples in 2023 was equal to the number of individuals captured (n = 126), while not all captured individuals carried sufficient pollen for genomic DNA extraction (n = 25). Given this, gut content metabarcoding was the more efficient methodology. The temporal distribution of taxonomic diversity observed by gut contentsmetabarcoding was also the key dataset in observing seasonal forage patterns. We observed more forage taxa present in the gut contents than were flowering within our transects in the early and late parts of the sampling season (Figure 2), on either end of the mid-summer floral peak. This suggested that the composition of the plant community present in the forage network was likely distinct from that of the floral community present within our transects.</p>
<p>We also compared interaction network topology as described by each methodology, analyzing the difference in the plant taxa included in the interaction network of <em>B. pascuorum</em> as detected by the each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Between the three interaction methodologies, no statistical differences were found, although the data dispersal differences were significant between the interaction observations and metabarcoding methodologies. <strong>This was likely a result of..</strong>. Both metabarcoding methodologies overlapped minimally with the flower count community and showed a statistical difference to the results of this methodology. This supported the notion that the interaction network included parts of the plant community beyond the floral taxa observed within our transects. This was partially to be expected, given that the metabarcoding methodologies and flower counts had a core difference in purpose. The latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.</p>
<p>Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span>, <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay <em>et al.</em> 2022</a>)</span>,<span class="citation" data-cites="smart2017">(<a href="#ref-smart2017" role="doc-biblioref">Smart <em>et al.</em> 2017</a>)</span>, metabarcoding increased observed interaction diversity by more than six-fold compared to interaction transect results. Metabarcoding also detected 93 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This, along with the temporal data, suggests that a large number of taxa within the <em>B. pascuorum</em> interaction network lie either beyond the spatial extent of our transects or outside of plant community traditionally sampled in plant-pollinator network surveys. Our reference database for metabarcode sequences included 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences <span class="citation" data-cites="bell2021">(<a href="#ref-bell2021" role="doc-biblioref">Bell 2021</a>)</span>, allowing metabarcoding to identify taxa from functional groups beyond the floral community sampled in our transects. Through metabarcoding we observed interactions with a variety of taxa outside of the entomophilous meadow and shrublant plant community. Among these were included trees and shrubs (n = 9), grasses (n =15), and other herbaceous plants (n = 7), indicating a different foraging pattern than was expected for <em>B. pascuorum.</em> The diversity of plant types observed within our metabarcoding data indicated that <em>B. pascuorum</em> forages a greater diversity and spatial range than previously expected.</p>
<p>Our observations of interactions with the anemophilous community are supported by similar previously documented interactions <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla <em>et al.</em> 2022</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere <em>et al.</em> 2017</a>)</span> <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span> [<span class="citation" data-cites="terrell1984">Terrell &amp; Batra (<a href="#ref-terrell1984" role="doc-biblioref">1984</a>)</span>]<span class="citation" data-cites="pojar1973">(<a href="#ref-pojar1973" role="doc-biblioref">Pojar 1973</a>)</span>, and have especially intriguing implications in the context of gut content metabarcoding observations. Previous studies using external pollen metabarcoding have removed wind-pollinated taxa from their analyses (e.g.<span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka <em>et al.</em> 2020</a>)</span>, <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span>). This is understandable, given the argument that wind-borne pollen in samples may represent false positive interactions <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon <em>et al.</em> 2017</a>)</span>, <span class="citation" data-cites="negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri <em>et al.</em> 2015</a>)</span>. Our gut content results, however, suggest that the practice removing these taxa as contaminants can be a large oversight, especially when using external pollen loads as standalone proxies for forage networks. Our observation of DNA from anemophilous taxa within gut samples suggest that interactions with these taxa are more than coincidental interactions with wind-borne pollen. Bumblebees are selective foragers that can assess the nutritional content of pollen, and make forage choices based on these assessments <span class="citation" data-cites="leonhardt2012">(<a href="#ref-leonhardt2012" role="doc-biblioref">Leonhardt &amp; Blüthgen 2012</a>)</span>. Multiple studies have concluded that bumblebees use this selective ability to forage for pollen quality, rather than quantity <span class="citation" data-cites="ruedenauer2016">(<a href="#ref-ruedenauer2016" role="doc-biblioref">Ruedenauer <em>et al.</em> 2016</a>)</span>,<span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span>, which has been used as an explanation for the adaptability of Bumblebee diets in response to environmental variability <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva <em>et al.</em> 2024</a>)</span>. Our detection of DNA from anemophilous pollen sources indicate intentional forage interactions with these taxa as a means of meeting the nutritional needs for adult bumblebees.</p>
<p>Existing hypotheses for pollinator forage adaptations in response to environmental changes have suggested that bees expand forage diversity beyond the flowering community and across habitats in order to survive annual “hunger gaps” <span class="citation" data-cites="timberlake2024b">(<a href="#ref-timberlake2024b" role="doc-biblioref">Timberlake <em>et al.</em> 2024a</a>)</span> <span class="citation" data-cites="becher2024">(<a href="#ref-becher2024" role="doc-biblioref">Becher <em>et al.</em> 2024</a>)</span>, when blooming floral species are limited <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake <em>et al.</em> 2024b</a>)</span>. Our observation of high forage diversity in gut contents before and after the floral peak, distinct interaction and flowering taxa network topologies, and consumption of taxa across functional groups, all together support these hypotheses. While the community beyond the physical area of our transects likely played a large role in these observations, the detection of anemophilous taxa in gut contents during the periods where forage diversity was higher than flowering diversity provide evidence for a community driven component as well. These observations show how the broader taxonomic detection capacity of metabarcoding allows for detection of interactions that otherwise would go unobserved by flower visitation surveys. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.</p>
<p>Our comparative analysis understates the resolution of our metabarcoding data. We chose to aggregate our metabarcoding data by sampling day while comparing community composition across methodologies in order to balance sampling effort. By doing so, we disregarded the opportunity to study interactions at the individual level offered by metabarcoding <span class="citation" data-cites="gous2019">(<a href="#ref-gous2019" role="doc-biblioref">Gous <em>et al.</em> 2019</a>)</span> <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">Bell <em>et al.</em> 2016</a>)</span>. To demonstrate the value of this resolution, we compared the interaction data from individual specimens with both between pollen and gut content samples. The low proportion of shared taxa between both sample sources at the individual level indicated a difference in the communities detected within both sample sources that was previously unobserved when methodologies were compared at the sampling day level. To clarify the different indications of these results, we also tested the difference in community composition between both methodologies at the individual level, this time observing a significant difference between the two methodologies (Table _ ).</p>
<p>The differences and commonalities in the communities detected by each methodology may reflect the result of selective interactions made by <em>B. pascuorum</em>. Bumblebees may forage pollen for immediate consumption, or transport pollen on the corbiculae to the colony nest for larval consumption <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Focusing on similarities between the two individual level datasets, the repeated detection of certain taxa by both methodologies increased confidence in these identifications, again demonstrating the complementary potential of these methodologies. For example, <em>Vicia</em> was observed by both methodologies across all specimens with both sample types in the first three periods. This adds to our previous knowledge from interaction transects of the strong relationship between <em>Vicia pyrenaica</em> and <em>B. pascuorum</em> (Artamendi, unpublished data), demonstrating the value of field surveys in validating laboratory based methodologies.</p>
<p>Our results suggest complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based <span class="citation" data-cites="mantegna2024">(<a href="#ref-mantegna2024" role="doc-biblioref">Mantegna 2024</a>)</span> <span class="citation" data-cites="nanglu2023">(<a href="#ref-nanglu2023" role="doc-biblioref">Nanglu <em>et al.</em> 2023</a>)</span>, and as molecular tools mature in their reliability <span class="citation" data-cites="richardson2015">(<a href="#ref-richardson2015" role="doc-biblioref">Richardson <em>et al.</em> 2015</a>)</span> <strong>NOT the best source for this</strong>. Given interaction transects’ disadvantage in sampling efficiency, incorporating this methodology as a validation of others may be the best way to integrate this methodology into future studies. Field observations can fill gaps left by metabarcoding methodologies, such as confirmation of pollination efficacy in interactions, or interaction frequency. As the external pollen load is a direct observation of the pollen that can be transferred between flowers, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. Pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval growth, given that external pollen will be returned to the nest to feed developing larvae <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. In a similar manner, gut content metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity <span class="citation" data-cites="li2025">(<a href="#ref-li2025" role="doc-biblioref">Li <em>et al.</em> 2025</a>)</span>. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen <em>et al.</em> 2021</a>)</span>, including parasite and disease transfer <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon <em>et al.</em> 2024</a>)</span>. Although they are not equal, our research highlights overall that each methodology offers advantages and disadvantages in terms of sensitivity, sampling effort, and perspective.</p>
<p>Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. <span class="citation" data-cites="magrach2023">(<a href="#ref-magrach2023" role="doc-biblioref">Magrach <em>et al.</em> 2023</a>)</span> <span class="citation" data-cites="devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese <em>et al.</em> 2024</a>)</span>). We present gut content metabarcoding as a promising new methodology for observing plant-pollinator interactions, but we advocate for pairing this methodology with others. Apart from pairing with other plant-pollinator interaction methodologies, the ability to pair gut content metabarcoding with other microbiome studies is highly intriguing for revealing linked interactions within the plant-pollinator-microbe network. The microbial exchange involved in plant pollinator interactions is an important component of pollinator interactions that is yet to be fully explored <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon <em>et al.</em> 2024</a>)</span> <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen <em>et al.</em> 2021</a>)</span>. Further application of gut content metabarcoding to plant-pollinator interaction studies will hopefully uncover new cases of synergy with other analyses, as well as further methods of interpreting the results of this methodology. For example, the ability to quantify interaction frequency using metabarcoding at the individual level would be a large advance in the applicability of metabarcoing for gut content or pollen samples. Given our observations, we conclude that the keys to making such methodological advances are likely to lie in finding the connections with similar data sources that can fill the gaps in the information provided by our current methodologies.</p>


<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-alarcón2010" class="csl-entry" role="listitem">
Alarcón, R. (2010). <a href="https://doi.org/10.1111/j.1600-0706.2009.17694.x">Congruence between visitation and pollen-transport networks in a California plant<span></span>pollinator community</a>. <em>Oikos</em>, <strong>119</strong>, 35–44.
</div>
<div id="ref-baksay2022" class="csl-entry" role="listitem">
Baksay, S., Andalo, C., Galop, D., Burrus, M., Escaravage, N. &amp; Pornon, A. (2022). <a href="https://doi.org/10.3389/fevo.2022.735588">Using metabarcoding to investigate the strength of plant-pollinator interactions from surveys of visits to DNA sequences</a>. <em>Frontiers in Ecology and Evolution</em>, <strong>10</strong>.
</div>
<div id="ref-becher2024" class="csl-entry" role="listitem">
Becher, M.A., Twiston-Davies, G., Osborne, J.L. &amp; Lander, T.A. (2024). <a href="https://doi.org/10.1111/icad.12736">Resource gaps pose the greatest threat for bumblebees during the colony establishment phase</a>. <em>Insect Conservation and Diversity</em>, <strong>17</strong>, 676–689.
</div>
<div id="ref-bell2021" class="csl-entry" role="listitem">
Bell, K. (2021). <a href="https://doi.org/10.6084/M9.FIGSHARE.14936004.V1">ITS2 july 2021</a>.
</div>
<div id="ref-bell2016" class="csl-entry" role="listitem">
Bell, K.L., Vere, N. de, Keller, A., Richardson, R.T., Gous, A., Burgess, K.S. &amp; Brosi, B.J. (2016). <a href="https://doi.org/10.1139/gen-2015-0200">Pollen DNA barcoding: Current applications and future prospects</a>. <em>Genome</em>, <strong>59</strong>, 629–640.
</div>
<div id="ref-bosmans2018" class="csl-entry" role="listitem">
Bosmans, L., Pozo, M.I., Verreth, C., Crauwels, S., Wilberts, L., Sobhy, I.S., Wäckers, F., Jacquemyn, H. &amp; Lievens, B. (2018). <a href="https://doi.org/10.1371/journal.pone.0204612">Habitat-specific variation in gut microbial communities and pathogen prevalence in bumblebee queens (Bombus terrestris)</a> (J.C. Nieh, Ed.). <em>PLOS ONE</em>, <strong>13</strong>, e0204612.
</div>
<div id="ref-cullen2021" class="csl-entry" role="listitem">
Cullen, N., Fetters, A. &amp; Ashman, T.-L. (2021). <a href="https://doi.org/10.1016/j.cois.2020.11.002">Integrating microbes into pollination</a>. <em>Current Opinion in Insect Science</em>, <strong>44</strong>, 48–54.
</div>
<div id="ref-devriese2024" class="csl-entry" role="listitem">
Devriese, A., Peeters, G., Brys, R. &amp; Jacquemyn, H. (2024). <a href="https://doi.org/10.1002/aps3.11601">The impact of extraction method and pollen concentration on community composition for pollen metabarcoding</a>. <em>Applications in Plant Sciences</em>, <strong>12</strong>, e11601.
</div>
<div id="ref-dharampal2019" class="csl-entry" role="listitem">
Dharampal, P.S., Carlson, C., Currie, C.R. &amp; Steffan, S.A. (2019). <a href="https://doi.org/10.1098/rspb.2018.2894">Pollen-borne microbes shape bee fitness</a>. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <strong>286</strong>, 20182894.
</div>
<div id="ref-dipasquale2016" class="csl-entry" role="listitem">
Di Pasquale, G., Alaux, C., Conte, Y.L., Odoux, J.-F., Pioz, M., Vaissière, B.E., Belzunces, L.P. &amp; Decourtye, A. (2016). <a href="https://doi.org/10.1371/journal.pone.0162818">Variations in the Availability of Pollen Resources Affect Honey Bee Health</a>. <em>PLOS ONE</em>, <strong>11</strong>, e0162818.
</div>
<div id="ref-gous2019" class="csl-entry" role="listitem">
Gous, A., Swanevelder, D.Z.H., Eardley, C.D. &amp; Willows-Munro, S. (2019). <a href="https://doi.org/10.1111/eva.12707">Plant<span></span>pollinator interactions over time: Pollen metabarcoding from bees in a historic collection</a>. <em>Evolutionary Applications</em>, <strong>12</strong>, 187–197.
</div>
<div id="ref-jones2018" class="csl-entry" role="listitem">
Jones, J.C., Fruciano, C., Hildebrand, F., Al Toufalilia, H., Balfour, N.J., Bork, P., Engel, P., Ratnieks, F.L. &amp; Hughes, W.O. (2018). <a href="https://doi.org/10.1002/ece3.3597">Gut microbiota composition is associated with environmental landscape in honey bees</a>. <em>Ecology and Evolution</em>, <strong>8</strong>, 441–451.
</div>
<div id="ref-leonhardt2012" class="csl-entry" role="listitem">
Leonhardt, S.D. &amp; Blüthgen, N. (2012). <a href="https://doi.org/10.1007/s13592-011-0112-y">The same, but different: pollen foraging in honeybee and bumblebee colonies</a>. <em>Apidologie</em>, <strong>43</strong>, 449–464.
</div>
<div id="ref-lignon2024" class="csl-entry" role="listitem">
Lignon, V.A., Mas, F., Jones, E.E., Kaiser, C. &amp; Dhami, M.K. (2024). <a href="https://doi.org/10.1080/03014223.2024.2353285">The floral interface: a playground for interactions between insect pollinators, microbes, and plants</a>. <em>New Zealand Journal of Zoology</em>, 1–20.
</div>
<div id="ref-li2025" class="csl-entry" role="listitem">
Li, Y., Liu, C., Wang, Y., Li, M., Zou, S., Hu, X., Chen, Z., Li, M., Ma, C., Obi, C.J., Zhou, X., Zou, Y. &amp; Tang, M. (2025). <a href="https://doi.org/10.1111/1744-7917.70051">Urban wild bee well-being revealed by gut metagenome data: A mason bee model</a>. <em>Insect Science</em>, <strong>n/a</strong>.
</div>
<div id="ref-maebe2021" class="csl-entry" role="listitem">
Maebe, K., Hart, A.F., Marshall, L., Vandamme, P., Vereecken, N.J., Michez, D. &amp; Smagghe, G. (2021). <a href="https://doi.org/10.1111/gcb.15751">Bumblebee resilience to climate change, through plastic and adaptive responses</a>. <em>Global Change Biology</em>, <strong>27</strong>, 4223–4237.
</div>
<div id="ref-magrach2023" class="csl-entry" role="listitem">
Magrach, A., Artamendi, M., Lapido, P.D., Parejo, C. &amp; Rubio, E. (2023). <a href="https://doi.org/10.1002/ecs2.4521">Indirect interactions between pollinators drive interaction rewiring through space</a>. <em>Ecosphere</em>, <strong>14</strong>, e4521.
</div>
<div id="ref-mantegna2024" class="csl-entry" role="listitem">
Mantegna, C. (2024). <a href="https://doi.org/10.1038/d41586-024-01178-2">Ecologists: don<span>’</span>t lose touch with the joy of fieldwork</a>. <em>Nature</em>, <strong>628</strong>, 692–692.
</div>
<div id="ref-milla2022" class="csl-entry" role="listitem">
Milla, L., Schmidt-Lebuhn, A., Bovill, J. &amp; Encinas-Viso, F. (2022). <a href="https://doi.org/10.1002/2688-8319.12120">Monitoring of honey bee floral resources with pollen DNA metabarcoding as a complementary tool to vegetation surveys</a>. <em>Ecological Solutions and Evidence</em>, <strong>3</strong>, e12120.
</div>
<div id="ref-nanglu2023" class="csl-entry" role="listitem">
Nanglu, K., de&nbsp;Carle, D., Cullen, T.M., Anderson, E.B., Arif, S., Castañeda, R.A., Chang, L.M., Iwama, R.E., Fellin, E., Manglicmot, R.C., Massey, M.D. &amp; Astudillo-Clavijo, V. (2023). <a href="https://doi.org/10.1002/ece3.10621">The nature of science: The fundamental role of natural history in ecology, evolution, conservation, and education</a>. <em>Ecology and Evolution</em>, <strong>13</strong>, e10621.
</div>
<div id="ref-negri2015" class="csl-entry" role="listitem">
Negri, I., Mavris, C., Prisco, G.D., Caprio, E. &amp; Pellecchia, M. (2015). <a href="https://doi.org/10.1371/journal.pone.0132491">Honey Bees (Apis mellifera, L.) as Active Samplers of Airborne Particulate Matter</a>. <em>PLOS ONE</em>, <strong>10</strong>, e0132491.
</div>
<div id="ref-pojar1973" class="csl-entry" role="listitem">
Pojar, J. (1973). <a href="https://doi.org/10.2307/2424049">Pollination of typically anemophilous salt marsh plants by bumble bees, bombus terricola occidentalis grne</a>. <em>The American Midland Naturalist</em>, <strong>89</strong>, 448–451.
</div>
<div id="ref-pornon2017" class="csl-entry" role="listitem">
Pornon, A., Andalo, C., Burrus, M. &amp; Escaravage, N. (2017). <a href="https://doi.org/10.1038/s41598-017-16785-5">DNA metabarcoding data unveils invisible pollination networks</a>. <em>Scientific Reports</em>, <strong>7</strong>, 16828.
</div>
<div id="ref-richardson2015" class="csl-entry" role="listitem">
Richardson, R.T., Lin, C.-H., Quijia, J.O., Riusech, N.S., Goodell, K. &amp; Johnson, R.M. (2015). <a href="https://doi.org/10.3732/apps.1500043">Rank-based characterization of pollen assemblages collected by honey bees using a multi-locus metabarcoding approach</a>. <em>Applications in Plant Sciences</em>, <strong>3</strong>, 1500043.
</div>
<div id="ref-ruedenauer2016" class="csl-entry" role="listitem">
Ruedenauer, F.A., Spaethe, J. &amp; Leonhardt, S.D. (2016). <a href="https://doi.org/10.1007/s00265-016-2129-8">Hungry for quality<span></span>individual bumblebees forage flexibly to collect high-quality pollen</a>. <em>Behavioral Ecology and Sociobiology</em>, <strong>70</strong>, 1209–1217.
</div>
<div id="ref-selva2024" class="csl-entry" role="listitem">
Selva, S., Moretti, M., Ruedenauer, F., Keller, A., Fournier, B., Leonhardt, S.D., Eggenberger, H.A. &amp; Abella, J.C. (2024). <a href="https://ecoevorxiv.org/repository/view/7812/">Urban bumblebees diversify their foraging strategy to maintain nutrient intake</a>.
</div>
<div id="ref-sirois-delisle2018" class="csl-entry" role="listitem">
Sirois-Delisle, C. &amp; Kerr, J.T. (2018). <a href="https://doi.org/10.1038/s41598-018-32665-y">Climate change-driven range losses among bumblebee species are poised to accelerate</a>. <em>Scientific Reports</em>, <strong>8</strong>, 14464.
</div>
<div id="ref-smart2017" class="csl-entry" role="listitem">
Smart, M.D., Cornman, R.S., Iwanowicz, D.D., McDermott-Kubeczko, M., Pettis, J.S., Spivak, M.S. &amp; Otto, C.R.V. (2017). <a href="https://doi.org/10.1093/ee/nvw159">A comparison of honey bee-collected pollen from working agricultural lands using light microscopy and ITS metabarcoding</a>. <em>Environmental Entomology</em>, <strong>46</strong>, 38–49.
</div>
<div id="ref-steffan2019" class="csl-entry" role="listitem">
Steffan, S.A., Dharampal, P.S., Danforth, B.N., Gaines-Day, H.R., Takizawa, Y. &amp; Chikaraishi, Y. (2019). <a href="https://doi.org/10.1086/704281">Omnivory in bees: Elevated trophic positions among all major bee families</a>. <em>The American Naturalist</em>, <strong>194</strong>, 414–421.
</div>
<div id="ref-tanaka2020" class="csl-entry" role="listitem">
Tanaka, K., Nozaki, A., Nakadai, H., Shiwa, Y. &amp; Shimizu-Kadota, M. (2020). <a href="https://doi.org/10.1186/s13104-020-05361-2">Using pollen DNA metabarcoding to profile nectar sources of urban beekeeping in K<span>ō</span>t<span>ō</span>-ku, Tokyo</a>. <em>BMC Research Notes</em>, <strong>13</strong>, 515.
</div>
<div id="ref-terrell1984" class="csl-entry" role="listitem">
Terrell, E.E. &amp; Batra, S.W.T. (1984). <a href="https://www.jstor.org/stable/4033059">Insects collect pollen of eastern wildrice, zizania aquatica (poaceae)</a>. <em>Castanea</em>, <strong>49</strong>, 31–34.
</div>
<div id="ref-timberlake2024b" class="csl-entry" role="listitem">
Timberlake, T.P., Tew, N.E. &amp; Memmott, J. (2024a). <a href="https://doi.org/10.1098/rspb.2024.1523">Gardens reduce seasonal hunger gaps for farmland pollinators</a>. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <strong>291</strong>.
</div>
<div id="ref-timberlake2024" class="csl-entry" role="listitem">
Timberlake, T.P., Vere, N. de, Jones, L.E., Vaughan, I.P., Baude, M. &amp; Memmott, J. (2024b). <a href="https://doi.org/10.1002/2688-8319.12360">Ten-a-day: Bumblebee pollen loads reveal high consistency in foraging breadth among species, sites and seasons</a>. <em>Ecological Solutions and Evidence</em>, <strong>5</strong>, e12360.
</div>
<div id="ref-vaudo2015" class="csl-entry" role="listitem">
Vaudo, A.D. (2015). <a href="https://doi.org/10.1016/j.cois.2015.05.008">Bee nutrition and floral resource restoration</a>. <em>Current Opinion in Insect Science</em>, <strong>10</strong>, 133–141.
</div>
<div id="ref-velthuis2006" class="csl-entry" role="listitem">
Velthuis, H.H.W. &amp; Doorn, A. van. (2006). <a href="https://doi.org/10.1051/apido:2006019">A century of advances in bumblebee domestication and the economic and environmental aspects of its commercialization for pollination</a>. <em>Apidologie</em>, <strong>37</strong>, 421–451.
</div>
<div id="ref-devere2017" class="csl-entry" role="listitem">
Vere, N. de, Jones, L.E., Gilmore, T., Moscrop, J., Lowe, A., Smith, D., Hegarty, M.J., Creer, S. &amp; Ford, C.R. (2017). <a href="https://doi.org/10.1038/srep42838">Using DNA metabarcoding to investigate honey bee foraging reveals limited flower use despite high floral availability</a>. <em>Scientific Reports</em>, <strong>7</strong>, 42838.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>