---
title: "interaction_methodologies"
format: pdf
editor: visual
bibliography: references.bib
---

Bumblebees (Bombus spp.) are excellent model pollinators for numerous reasons. On top of the relative importance of bees in pollen transport relative to other pollinators [@alarcón2010], bumblebees specifically offer the advantage of abundance and ease of observation in the wild, common use in agricultural settings [@velthuis2006] and climate dependence [@sirois-delisle2018] [@maebe2021]. Given the quantity of existing research on bumblebees, their environmental and nutritional needs are broadly defined. Bees generally need a combination of pollen and nectar to meet their macronutritional needs [@vaudo2015]. Pollen is the main source of protein and fats in the bee diet, and the microbial content of pollen is an important source of nutrition during the larval period [@steffan2019], [@dharampal2019]. A diversity of pollen sources from floral species promotes healthy individual physiology [@dipasquale2016] and for this reason it is generally understood that plant diversity supports bee populations. In addition to the affecting biodiversity available to bees within a landscape, landscape use has an effect on the pollinator internal microbiome and individual health [@jones2018]. While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interactions, including which are most important to pollinators, and in what spatiotemporal contexts. 

We evaluated the independent and complementary abilities of three pollinator-perspective based methodologies for observation of *B. pascuorum* and plant interactions, contextualized by surveys of floral resource availability. We compared each methodology in terms of their abilities to capture diversity in interaction taxa and their data outputs in relation to sampling effort. In this comparison, metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity (Figure 1). This advantage was maintained across the sampling season, as interaction networks generally expanded (Figure 2). Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Interaction transect taxa inherently represented a subset of the taxa observed by flower counts, with cumulative interaction observations including only 23% of the taxa identified across the season by flower counts. Metabarcoding methodologies, however, were not limited to detection of interactions included within the spatial constraint, observation periods, nor the floral diversity of the transects. Both metabarcoding methodologies observed greater total interaction taxa than the floral diversity surveys. During three consecutive sampling periods from May to mid-July, however, more taxa were present within the floral diversity of the transects than metabarcoding detected in *B. pascuorum* interactions (Figure 2). Finally, comparing the two metabarcoding methodologies, gut metabarcoding outperformed corbicular pollen metabarcoding cumulatively and in each sampling period in terms of taxonomic diversity captured. Comparing the sampling effort of both methodologies, gut metabarcoding also presented an easier methodology, given that the number of samples was equal to the number of individuals captured, while not all captured individuals carried sufficient pollen for genomic DNA extraction. These results suggest that in a raw evaluation of capacity and demanded effort, gut metabarcoding provides the best methodology for detecting interaction networks. However, the seasonal complexities observed between floral resource availability and interaction observations hint that the communities included in these networks are also an important factor in choosing methodologies for future studies.

We also compared community composition as detected by each methodology, analyzing the difference in the plant taxa included in the interaction network of *B. pascuorum* as detected by the each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Community composition analysis found no significant differences between the two metabarcoding methodologies, but both overlapped minimally with the flower count community and showed a statistical difference to the results of this methodology. This reflected an underlying dynamic of the *B. pascuorum* interaction network structure, first signaled by the differences in taxonomic diversity observed by these methodologies across periods (Figure 2). The core difference between the metabarcoding methodologies and flower counts was that the latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.

Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions [@milla2022], [@baksay2022],[@smart2017], metabarcoding increased observed interaction diversity by more than six-fold. Metabarcoding detected 94 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This suggests that a large number of taxa within the *B. pascuorum* interaction network lie either beyond the spatial extent of our transects or outside of entomophilous community traditionally sampled in plant-pollinator network surveys. Our reference database for metabarcode sequences included all 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences [@bell2021], allowing metabarcoding to identify taxa from beyond the entomophilous community. Our metabarcoding data revealed a variety of anemophilous taxa within the *B. pascuorum* interaction network, including trees and shrubs (n = 9), grasses (n =15), and other herbaceous plants (n = 7), indicating a greater diversity and spatial range of bumblebee foraging than previously expected. Indeed, our transects did not extend beyond grass and shrub meadows, although other habitat types were adjacent or partially integrated into to the transect areas, including forests and wetlands.

Our observations of interactions with the anemophilous community support other previously documented interaction observations [@selva2024] [@milla2022] [@timberlake2024] [@devere2017] [@tanaka2020] [@terrell1984][@pojar1973], and have especially intriguing implications in the context of gut metabarcoding. Previous studies using external pollen metabarcoding have removed anemophilous taxa, specifically wind-pollinated taxa from their analyses (e.g.[@tanaka2020], [@pornon2017]). It is understandable that in an external pollen metabarcoding context with the objective of characterizing pollination interactions, that these sources of pollen could be overwhelmingly represented in results. Our results suggest that in studies intending to use external pollen loads as proxies for forage networks, choosing to omit these taxa could be a large oversight. Gut metabarcoding data are direct observations of consumed plant material, and our finding of pollen or plant material consumed from anemophilous species within or nearby our transects could support existing hypotheses for pollinator forage adaptations in response to environmental changes. For example, expanding forage diversity beyond the floral community and across habitats is likely to explain how Bumblebees survive annual “hunger gaps” [@timberlake2024b] [@becher2024] , when blooming floral species are limited [@timberlake2024]. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.

In comparing community composition across methodologies, we chose to aggregate our metabarcoding data by sampling day. By doing this, we ensured that sampling effort was balanced across methodologies. However, this understated the resolution of the metabarcoding data. One of the great advantages of applying metabarcoding to pollinator samples is the ability to study interactions at the individual level [@gous2019][@bell2016].

As a result, the communities detected by metabarcoding were able to include those of the transect methodologies, as well as parts of the Gorbeia plant community beyond the transects, speaking to the complementary nature of the data provided by each of the methodologies together.

Our results suggest complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based, and as molecular tools mature in their reliability (maybe could cite [@richardson2015] as a paper suggesting need for growth of molecular techniques, but it's old). Interaction transects have the previously observed disadvantage in sampling efficiency, suggesting that a reduced approach, focused on validation of other methodologies, may be the best way to integrate this methodology into future studies. As the external pollen load is a direct observation of the pollen that can be transferred between floweres, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. In the case of bumblebees, which are selective foragers with an ability to discern nutritional value in pollen [@leonhardt2012], pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval growth [@vaudo2015]. In a similar manner, gut metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity [@li2025]. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions [@cullen2021], including parasite and disease transfer [@lignon2024]. Given the different information offered by each methodology,

Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. [@magrach2023], [@devriese2024]). While our results suggest that a single methodology approach may have limitations

As standalone methodologies, each offers advantages and disadvantages in terms of informative capacities, sensitivity, and sampling effort. Interaction transects, fof
