<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.32">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Coupling Diet and Pollen Metabarcoding with Field Surveys to Reveal Plant - pollinator Interactions</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="interaction_methodologies_files/libs/clipboard/clipboard.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="interaction_methodologies_files/libs/quarto-html/popper.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/anchor.min.js"></script>
<link href="interaction_methodologies_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/quarto-html/quarto-syntax-highlighting-37eea08aefeeee20ff55810ff984fec1.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="interaction_methodologies_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-bb462d781dde1847d9e3ccf7736099dd.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent quarto-light">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Coupling Diet and Pollen Metabarcoding with Field Surveys to Reveal Plant - pollinator Interactions</h1>
<p class="subtitle lead">Christian Gostout <sup>1</sup>, Luis J. Chueca <sup>1</sup>, Jon Poza, Xabier Salgado-Irazabal<sup>1</sup>, Estefanía Tobajas<sup>1</sup>, Jennifer Rose<sup>1</sup>, Brais Hermosilla<sup>1</sup>, Marion Donald<sup>2</sup>, Manpreet Dhami<sup>2</sup>, Ainhoa Magrach<sup>1,3</sup> <sup>1</sup>BC3 Basque Centre for Climate Change, Leioa, Spain, <sup>2</sup>Bioeconomy Science Institute, Lincoln, New Zealand, <sup>3</sup>IKERBASQUE, Basque Foundation for Science, Bilbao, Spain</p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><strong>Key-words:</strong> interaction, metabarcoding, network, pollinator</p>
<p><strong>Abstract</strong></p>
<p>Our understanding of plant-pollinator interaction networks hinges on the methods used to describe their nodes and links. Currently, most networks are built from field observations that may overlook many consumer–resource links. Further, interaction networks lack descriptive links that characterize interaction types and outcomes. Towards a more complete approach for building interaction networks, we compare plant interactions from the wild pollinator species, Bombus pascuorum, observed by three methodologies with different implications for interaction outcomes. We compare floral visitation interactions obtained from field observations to plant consumption interactions revealed by metabarcoding of gut contents and pollen transport interactions detected by corbicular pollen loads. Our approach adds functional context to plant–pollinator network links and reveals new interactions. We show that both metabarcoding approaches increase sampling efficiency and reveal links left unobserved by field observations of visitation, highlighting plant taxa that are not pollinator-dependent, yet constitute important dietary resources. Paired with floral diversity surveys, gut content results also reveal seasonal patterns in the spatial extent and functional diversity included in forage, which other methodologies fail to demonstrate. Metabarcoding data analyzed at the individual specimen level further reveal heterogeneity in plant resource use between pollen transport and consumption. Metabarcoding methodologies capture greater spatial, temporal, and taxonomic ranges, while field observations provide validating datasets with taxonomic precision.Our results show that integrating visitation, transport, and consumption data changes network topology and the roles of plant nodes, offering a more nuanced and complete map of interactions with clearer priorities for management. We advocate for defining links explicitly by their functions and combining methods to account for hidden structure in ecological networks.</p>
<p><strong>Introduction</strong></p>
<p>Pollination is a critical ecosystem service that is currently threatened by different global changes, including habitat loss, intensifying agriculture, pathogens, and invasive species <span class="citation" data-cites="klein2006">(<a href="#ref-klein2006" role="doc-biblioref">Klein et al., 2006</a>)</span>. Pollinators critically support the reproduction of 94% of wild flowering plants and 75% of crop species <span class="citation" data-cites="vanbergen2013">(<a href="#ref-vanbergen2013" role="doc-biblioref">Vanbergen &amp; Insect Pollinators Initiative, 2013</a>)</span>, contributing to 35% of global food production <span class="citation" data-cites="klein2006">(<a href="#ref-klein2006" role="doc-biblioref">Klein et al., 2006</a>)</span>. However, our understanding of this topic is incomplete given that, historically, the methodological approach to studying plant-pollinator interactions has been biased towards the plant side of interactions <span class="citation" data-cites="evans2020">Bosch et al. (<a href="#ref-bosch2009" role="doc-biblioref">2009</a>)</span>. As a consequence, the well-established relationship between pollinator diversity and the productivity of plant communities <span class="citation" data-cites="artamendi2025">Katumo et al. (<a href="#ref-katumo2022" role="doc-biblioref">2022</a>)</span> lacks an equally developed mirrored perspective, describing the floral diversity that supports pollinator populations.&nbsp;</p>
<p>Network theory provides a useful framework to summarize patterns of plant–pollinator interaction <span class="citation" data-cites="burkle2011">(<a href="#ref-burkle2011" role="doc-biblioref">Burkle &amp; Alarcón, 2011</a>)</span>, but the strong influence of the interaction types that define links on the scope of networks has yet to be accounted for in most studies. Existing methodologies for reconstructing interaction networks tend to emphasize structural patterns, while overlooking the functional outcomes of interactions that are critical for understanding how plant communities support pollinators <span class="citation" data-cites="quintero2022">(<a href="#ref-quintero2022" role="doc-biblioref">Quintero et al., 2022</a>)</span>. In eusocial bees, for example, plant interactions may have several outcomes. Bees consume plant material, including pollen, nectar, or even plant tissue <span class="citation" data-cites="vaudo2015">Pashalidou et al. (<a href="#ref-pashalidou2020" role="doc-biblioref">2020</a>)</span>. They also collect pollen on their corbicula for transport to the nest for feeding drones and larvae <span class="citation" data-cites="leach2018">Vaudo (<a href="#ref-vaudo2015" role="doc-biblioref">2015</a>)</span>. Finally, visitation of the reproductive parts of flowers can have various outcomes for both the plant and pollinator, including pollination and pathogen transfer (Lignon et al.&nbsp;2024). Interaction networks generally represent only one of these outcomes, although each is important to understanding how plant taxa support pollinators.</p>
<p>The importance of different outcomes in plant-pollinator interactions becomes clear when considering the biodiversity necessary to support pollinators across life stages. Because the resources needed for foraging adult pollinator nutrition can be different from those needed at the larval stage, or by other colony members <span class="citation" data-cites="leach2018">Vaudo (<a href="#ref-vaudo2015" role="doc-biblioref">2015</a>)</span>, transported pollen may not completely represent the interactions necessary to sustain adult pollinator diets. This is especially true for bumblebees (<em>Bombus spp.</em>), which are able to evaluate pollen resource quality to make discerning forage choices <span class="citation" data-cites="leonhardt2012">Timberlake, Tew, et al. (<a href="#ref-timberlake2024a" role="doc-biblioref">2024a</a>)</span>. Bumblebees make trial-and-error floral visits in order to find adequate forage <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>)</span>, which may result in pollen transport without consumption. Conversely, consumption, or simply visitation, may occur without resulting in transport <span class="citation" data-cites="popic2012">(<a href="#ref-popic2012" role="doc-biblioref">Popic et al., 2012</a>)</span>. Accounting for different interaction outcomes, such as visitation, transport, and consumption, is a critical next step in representing the network of plant diversity used by pollinators.</p>
<p>Shifting network studies to incorporate the pollinator perspective and leveraging the contributions of different methodologies can produce a more complete image of interaction networks. Research based on microscopy and molecular analyses of pollen load samples sourced from insect specimens can identify greater plant species diversity within interaction networks compared to studies based solely on field observations of floral visitation <span class="citation" data-cites="baksay2022">Bosch et al. (<a href="#ref-bosch2009" role="doc-biblioref">2009</a>)</span>. Additionally, studies adopting a pollinator-centered view have revealed greater detail in forage preference trends, such as how pollinators use forage quality or quantity-based strategies <span class="citation" data-cites="timberlake2024a">Selva et al. (<a href="#ref-selva2024" role="doc-biblioref">2024</a>)</span>, seasonal changes <span class="citation" data-cites="leponiemi2023">(<a href="#ref-leponiemi2023" role="doc-biblioref">Leponiemi et al., 2023</a>)</span>, life cycle timing, and metabolic specialization <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>.</p>
<p>Genetic tools can detect plant-pollinator interactions that may be unobserved in pollen microscopy and traditional field surveys <span class="citation" data-cites="lowe2022">K. L. Bell et al. (<a href="#ref-bell2016" role="doc-biblioref">2016</a>)</span>, and target specific interaction types. Amplicon sequence metabarcoding of pollen samples complements the visitation interactions observed by field studies <span class="citation" data-cites="arstingstall2021">K. L. Bell et al. (<a href="#ref-bell2017" role="doc-biblioref">2017</a>)</span>, increasing species detection by 9 - 144% <span class="citation" data-cites="milla2022">Smart et al. (<a href="#ref-smart2017" role="doc-biblioref">2017</a>)</span> and network sampling completeness up to 30%, while reducing exaggeration of specialization <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>)</span> and revealing interactions beyond the traditionally surveyed floral community <span class="citation" data-cites="milla2022">Vere et al. (<a href="#ref-devere2017" role="doc-biblioref">2017</a>)</span>. Advances in the reliability and accessibility of amplicon sequencing have made these approaches more feasible for studying plant-pollinator interactions. Field surveys of visitation can now be effectively complemented by genetic tools <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>)</span> targeting specific interaction types, enhancing our understanding of interaction diversity.</p>
<p>Most studies applying metabarcoding to pollinator-sourced samples for constructing interaction networks analyze the external pollen loads of bees or pollen stored in nest reserves of honey and beebread <span class="citation" data-cites="selva2024">Leontidou et al. (<a href="#ref-leontidou2021" role="doc-biblioref">2021</a>)</span>, despite limitations of these sampling targets. Pollen in these samples can come from the environment, even including accumulation of windborne material <span class="citation" data-cites="negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri et al., 2015</a>)</span>. To account for this, past studies have ignored detections of wind pollinated taxa&nbsp;<span class="citation" data-cites="tanaka2020">Pornon et al. (<a href="#ref-pornon2017" role="doc-biblioref">2017</a>)</span>, although this may introduce bias to results, given that many plant taxa have partial identities as wind or insect pollinated taxa <span class="citation" data-cites="saunders2018">(<a href="#ref-saunders2018" role="doc-biblioref">Saunders, 2018</a>)</span> A more fundamental issue with externally carried pollen and nest reserves is present in their restricted ability to represent interaction types. Studies of external pollen carried by eusocial bees, for example, have generally sequenced the DNA of pollen from the corbicula (e.g. <span class="citation" data-cites="shi2025">Shi et al. (<a href="#ref-shi2025" role="doc-biblioref">2025</a>)</span>). Corbicular pollen provides an easily obtained sample, containing a mixture of pollen collected for transport to the nest for brood feeding <span class="citation" data-cites="leach2018">Vaudo (<a href="#ref-vaudo2015" role="doc-biblioref">2015</a>)</span>, which only directly observes interactions where pollen is transported <span class="citation" data-cites="arstingstall2021">(<a href="#ref-arstingstall2021" role="doc-biblioref">Arstingstall et al., 2021</a>)</span>. Given the role of this pollen in bees’ life cycles, it is easy to overstep the interpretative capacity of these sample types when characterizing forage networks to describe diet, or successful pollination interactions.</p>
<p>Pollinator intestinal tracts (hereafter: guts) represent an additional source for observing interactions, specifically those related to consumption of pollen and other plant material <span class="citation" data-cites="li2025">Mayr et al. (<a href="#ref-mayr2021" role="doc-biblioref">2021</a>)</span>. Plant DNA detected in gut contents can reveal interactions with consumption as the exclusive outcome, which, aside from flower visits, can include nectar robbing <span class="citation" data-cites="popic2012">(<a href="#ref-popic2012" role="doc-biblioref">Popic et al., 2012</a>)</span> and plant damage <span class="citation" data-cites="pashalidou2020">(<a href="#ref-pashalidou2020" role="doc-biblioref">Pashalidou et al., 2020</a>)</span>. The gut content approach can also account for environmental contamination in external pollen and nest stores by highlighting oversights resulting from the exclusion of interactions with the anemophilous and partially-anemophilous plant taxa in external pollen studies. There is an accumulating body of evidence supporting the idea that pollinators must regularly search across functional groups of the plant community to meet their nutritional needs <span class="citation" data-cites="ibiyemi2025">Pojar (<a href="#ref-pojar1973" role="doc-biblioref">1973</a>)</span>, although little attention has been given to these observations as a potentially important part of plant-pollinator networks <span class="citation" data-cites="saunders2018">(<a href="#ref-saunders2018" role="doc-biblioref">Saunders, 2018</a>)</span>. This understudied component of pollinator forage together with the surprising lack of genetic analyses of pollinator gut contents, represents a clear knowledge gap and an opportunity to uncover finer detail in pollinator interaction networks.</p>
<p>Our objective is to determine whether a combined methodological approach can provide further insights into pollinator forage ecology and plant-pollinator interaction networks by expanding interaction detections and providing context to network links. We assess how metabarcoding of pollinator gut contents can complement or challenge the characterization of plant-pollinator interaction networks described by more common methodologies, including field surveys of plant pollinator interactions and external pollen load metabarcoding. To this end, we compare interaction networks constructed from each of these methodologies for a single model pollinator, <em>Bombus pascuorum</em>, an easily identified bumblebee common to most of Europe <span class="citation" data-cites="lecocq2015">(<a href="#ref-lecocq2015" role="doc-biblioref">Lecocq et al., 2015</a>)</span>. Our focus on a single pollinator species holds pollinator identity constant and attributes differences in network structure to methodology, rather than to variation among pollinator species. We hypothesize that the consumption interactions detected in gut metabarcoding will include a network of plant taxa distinct from those detected by other methodologies. Although we expect overlap between networks constructed by different methodologies, we expect to observe previously overlooked interaction network structure, including new links and significance of network links. Ideally, the resulting combination of observations will generate a network that will elevate our capacity to detect meaningful plant-pollinator interactions, and learn more about interaction types and implications for pollinator health.</p>
<p><strong>Methods</strong></p>
<p>Our sample collection was conducted in Gorbeia Natural Park, a protected area in Spain. Within Gorbeia, we selected 16 sampling sites located within the mixed zones of meadows and shrublands found at higher elevations within the park.&nbsp; We conducted fieldwork from early April to the end of July 2023 covering the main flowering period and peak annual pollinator activity in Gorbeia. On each sampling day during this timeframe, we visited field sites in pairs. Sampling days were organized into six periods, in which we sampled each site pair once per period. We conducted three types of surveys during daily peaks of pollinator activity, including floral diversity surveys (“flower counts”), interaction transect surveys, and Bombus pascuorum specimen collection for amplicon sequencing analyses.</p>
<p><em>Interaction transects and floral resource availability surveys</em>&nbsp;</p>
<p>We used the one 250 m transect at each site for both interaction transect and flower count surveys, recording observations within ~2 m of the transect line. Interaction surveys were conducted three times per day, each lasting 1 h. All insects observed contacting the reproductive parts of herbaceous flowers within the transect were recorded; for this study, we retained only Bombus pascuorum interaction data. Surveys were spaced by ~2 hours (~11:00, ~13:00, ~15:00), and transects were walked at a constant pace to cover the full length within an hour. For each site and sampling period, one flower count was conducted by recording all of the flowering herbaceous species within the transects.</p>
<p><em>Bombus pascuorum specimens</em></p>
<p>For every period visit at each site, we collected up to five <em>B. pascuorum</em> specimens for molecular analyses (N = 126). We brought specimens back from the field and froze them at -20°C until processed. In the lab, we extracted the entire gut and honey stomach of <em>B. pascuorum</em> individuals. Additionally, if present, we collected pollen pellets from the corbicula of specimens into sterile 1.5 mL centrifuge tubes. Pollen samples were stored individually by specimen sample at -20°C.</p>
<p>Gut Content DNA extraction</p>
<p>Genomic DNA was extracted from <em>B. pascuorum</em> guts using the NucleoSpin® 96 Soil kit (Macherey-Nagel, Düren, Germany) and amplified in duplicate using the DFD forward and ASDFAS reverse primers. To avoid site and period bias, all samples were randomized using a randomizer program before the DNA extraction. We followed the kit manufacturer protocol, only adjusting centrifuge times to account for the lower maximum velocity of the large centrifuge used to process large sample numbers simultaneously (See Supporting Information). To confirm successful DNA extraction, Nanodrop tests were performed on random samples.</p>
<p><em>DNA extraction from corbicular pollen pellets</em></p>
<p>DNA was extracted from pollen pellets (N = 25) using the Machery-Nagel NucleoSpin 8 Food kit, including additional initial steps recommended by the kit’s supplementary protocol for pollen DNA extraction (See Supporting Information). Qubit fluorometry tests using random samples confirmed successful DNA extractions.</p>
<p><em>Amplicon Sequencing</em>&nbsp;</p>
<p>Our metabarcoding sequence libraries were built by amplifying and sequencing the internal transcribed spacer (ITS2) region of the ribosomal DNA in our extract samples. For all samples, we used existing primers for amplification of the ITS2 region (See Supporting Information). Libraries were sequenced on an Illumina platform to generate paired-end raw reads. We used demultiplexed raw sequence data, with primer and adapter sequences removed, in further bioinformatic analyses.</p>
<p><em>Bioinformatics: taxonomic assignment and contaminant analysis</em></p>
<p>Raw Illumina sequences were processed using the DADA2 bioinformatics pipeline <span class="citation" data-cites="callahan2016">(<a href="#ref-callahan2016" role="doc-biblioref">Callahan et al., 2016</a>)</span>. Taxonomy was added to the ASVs using an existing reference sequence database <span class="citation" data-cites="bell2021">(<a href="#ref-bell2021" role="doc-biblioref">K. Bell, 2021</a>)</span>, which provided reference sequences at the species level for all but 21 of the species present in the study area, all of which were identifiable to the genus level in the database. We removed likely contaminants and misidentified ASVs from our bioinformatics results using a three-step screening process. First, ASVs were analyzed for contaminants using the decontam package in R <span class="citation" data-cites="davis2018">(<a href="#ref-davis2018" role="doc-biblioref">Davis et al., 2018</a>)</span>. Second, we conducted a BLAST search using ITS2 Database <span class="citation" data-cites="ankenbrand2015">(<a href="#ref-ankenbrand2015" role="doc-biblioref">Ankenbrand et al., 2015</a>)</span> to verify taxa that were identified by only one ASV within our results. Finally, the remaining list of taxa was assessed by a local botanist for remaining errors.</p>
<p><em>Statistical analysis</em></p>
<p>We analyzed the results of each methodology together using statistical tools for comparing interaction plant communities across methodology, time, and individual specimens. As an initial broad test of whether the methodologies detected interactions with different plant communities, we used binary presence-absence matrices to compare the communities detected by each methodology on each sampling day. Data were aggregated by sampling day for all sets of observations. Community composition was contrasted using the Raup-Crick dissimilarity index in a PERMANOVA test within the R package, vegan <span class="citation" data-cites="vegan">(<a href="#ref-vegan" role="doc-biblioref">Oksanen et al., 2024</a>)</span>with methodology as the independent variable. Further pairwise comparisons of these data were made by subsetting the dissimilarity matrix used in the first test by each unique methodology pair and using multiple PERMANOVAs to test the pairs. We also used vegan to observe beta dispersal of our data as a further means of understanding PERMANOVA results.</p>
<p>Among our <em>B. pascuorum</em> specimens, 25 provided both pollen and gut samples. Using the data from this subset of samples, we compared the plant communities detected by the two metabarcoding methodologies at the individual sample level without aggregation. As before, Raup-Crick dissimilarity matrices were calculated using binary detection data from pollen and gut detections. PERMANOVA compared both methodologies’ detected communities in strata defined by specimens of sample origin.</p>
<p><em>B. pascuorum - plant interaction Network Metrics</em>&nbsp;</p>
<p>We used interaction frequencies from the three methodologies to build <em>B. pascuorum</em>–plant interaction networks and calculate species-level metrics for plant importance and specialization. Plant importance was the proportion of all <em>B. pascuorum</em> interactions involving a given plant genus. For metabarcoding and pollen-load data, interactions were counted as the number of individual bee samples in which a plant taxon was detected; for observational data, interactions corresponded to recorded visits. Species-level specialization (<em>d’</em>) was calculated following <span class="citation" data-cites="blüthgen2006">Blüthgen et al. (<a href="#ref-blüthgen2006" role="doc-biblioref">2006</a>)</span>, as implemented in the R package bipartite <span class="citation" data-cites="bipartite">(<a href="#ref-bipartite" role="doc-biblioref">Dormann et al., 2009</a>)</span>.</p>
<p>We created a composite interaction network for <em>B. pascuorum</em>, incorporating the data of each methodology and the interaction outcome types as network metadata. Network nodes included <em>B. pascuorum</em> and the list of plant genera detected across the three interaction datasets. Single plant genera were assigned between one and three links corresponding to interaction type, depending on their detection across methodologies.</p>
<hr>
<p><strong>Results</strong></p>
<p><em>Interaction networks compared to floral diversity</em>&nbsp;</p>
<p>Comparing flower count surveys to the plant networks detected by the three interaction methodologies, we observed that B. pascuorum did not interact with the entirety of the available floral diversity within our transects (Figure 1). Flower counts identified a total of 117 flowering plant genera across the sampling season. Among these, 39 were not included within the interaction networks of any of the other methodologies. By inherent design, the 27 genera identified by interaction transects were a subset of the diversity observed by flower counts, amounting to 23% of the floral taxa identified across the season. Gut content metabarcoding detected 58% of the taxa observed by flower counts, and corbicular pollen metabarcoding detected 53%. Unlike the results of interaction transects, the taxa detected by metabarcoding methodologies were not restricted to those included within the potential interaction network indicated by flower counts.</p>
<p><em>Comparison of interaction methodology sensitivity</em></p>
<p>Both metabarcoding methodologies detected multiple unique taxa, while interaction transects detected fewer total taxa, with zero unique interactions (Figure 1). Gut content metabarcoding detected interactions with 131 total plant genera, including 33 taxa uniquely identified by this methodology. The total diversity identified by this methodology included all of the genera identified by interaction transects. Corbicular pollen metabarcoding detected 123 plant genera, with 31 uniquely detected genera. Again, these genera included all of those identified by interaction transects. The two metabarcoding methodologies shared 83 common plant genera, representing 67% of the total corbicular pollen diversity and 63% of the gut content diversity.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="../results/venn.figure.png" class="img-fluid figure-img" width="1500"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 1:</strong> Total diversity and overlap of plant genera observed by four observation methodologies: transect surveys of floral diversity (“flower counts”) and B. pascuorum - flower interactions, and metabarcoding of plant DNA in corbicular pollen and gut contents of B. pascuorum. For the three interaction observation methodologies, the total number of taxa represents the degree of B. pascuorum in the interaction network constructed by the corresponding methodology.</em></p>
<p>Taxonomic diversity observed by sampling period revealed different temporal patterns in flowering taxa and interaction diversity over the sampling season (Figure 2). While there was a net increase in available floral taxa and interactions from the first to last period, peak floral diversity was observed in period four, while peak interactions were observed in period six. Metabarcoding methodologies consistently detected more taxa than interaction transects, with gut content metabarcoding detecting more taxa than the other interaction methodologies in all periods. In periods one, two, and six, before and after the peak observed floral diversity, gut content metabarcoding detected more taxa than were observed in flower counts. During periods three through five, floral diversity was greater than gut content diversity.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="../results/interaction.diversity.periods.png" class="img-fluid figure-img" width="1528"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 2:</strong> Taxonomic diversity in Bombus pascuorum interaction networks over six sampling periods (April - August, 2023) observed through floral visitation surveys and ITS2 metabarcoding of DNA extracted from bumblebee gut contents and corbicular pollen loads. The number of plant genera indicated is a cumulative raw value for each methodology and period, with no standardization for sampling effort. Interaction diversity for transects is represented by the total number of taxa observed over each transect and sampling day, for each period. For metabarcoding methodologies, interaction diversity is the total number of plant genera observed across all samples collected during the given period. Available flowering taxa diversity within transects is displayed as a contextualizing data set to show the seasonal floral peak.</em></p>
<p><em>Functional diversity observations</em></p>
<p>While the taxa included in the interaction networks of flower counts and interaction transects only included taxa from the entomophilous community, both metabarcoding methodologies detected taxa from the anemophilous community as well. Anemophilous plant taxa represented 20% (N = 36) of the total identified plant taxa between the two methodologies (N = 172), including 19 genera from Poaceae, eight tree/woody plant genera, and nine other herbaceous genera (Table S2).&nbsp;During the first, second, and sixth periods, when gut content metabarcoding detected more taxa than were observed in the entomophilous community of the transects. Of the taxa observed by gut metabarcoding during these periods, an average of 14% were anemophilous or partially anemophilous (Figure S1).</p>
<p><em>Plant community composition across methodologies</em></p>
<p>Plant communities detected by each methodology were compared for statistical evaluation of the differences between results. Binary presence-absence matrices were used to compare the communities detected by each methodology, aggregated by sampling day. Community composition was contrasted using the Raup-Crick dissimilarity index within a PERMANOVA test, including methodology as the independent variable (Table 1). A PERMANOVA test <span class="citation" data-cites="vegan">(<a href="#ref-vegan" role="doc-biblioref">Oksanen et al., 2024</a>)</span> of the differences in taxonomic composition across methodologies&nbsp;indicated that methodology has a significant effect (P &lt; 0.001) on the detected plant community involved in pollinator interactions, accounting for 28% of the variation in detected interaction networks. In pairwise comparisons (Table 2), the flower count communities differed significantly (P = 0.001, Holm-Bonferroni adjusted) from those of all other methodologies. Between pairs of other methodologies, no significant differences were observed. Differences in methodologies were also assessed in an ordination using non-metric Multidimensional Scaling (nMDS), yielding a stress value of 0.17 (Figure 3). Average distances to the group centroid indicated by beta dispersal showed high dispersal of interaction transect data (distance to mean = 0.62) compared to the more centered metabarcoding and flower count results (distance to mean ≤0.10). An ANOVA test of mean dispersal by methodology indicated a significant difference (P &lt; 0.001) in dispersal attributed to methodology.&nbsp;</p>
<p><em><strong>Table 1.</strong> Results of testing the effect of methodology on the community composition of plant taxa detected by four observation methodologies: transect surveys of floral diversity and B. pascuorum - flower interactions, and metabarcoding of plant DNA in corbicular pollen and gut contents of B. pascuorum. Methodology’s effect was tested using a Permutational Multivariate Analysis of Variance (PERMANOVA, Oksanen et al.&nbsp;2022), using the Raup-Crick dissimilarity index with 9999 permutations. The test indicated that up to 28% of the variation in community composition may be explained by the influence of methodology.</em></p>
<div class="cell">
<div class="cell-output-display">
<table class="lightable-minimal caption-top table table-sm table-striped small" data-quarto-postprocess="true">
<caption>PERMANOVA analysis of interaction methodology's effect on observed plant community</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th"></th>
<th style="text-align: right;" data-quarto-table-cell-role="th">Df</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">SumOfSqs</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">R2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">F</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">Pr(&gt;F)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Model</td>
<td style="text-align: right;">2</td>
<td style="text-align: right;">0.217003</td>
<td style="text-align: right;">0.0134103</td>
<td style="text-align: right;">0.6524416</td>
<td style="text-align: right;">0.6477</td>
</tr>
<tr class="even">
<td style="text-align: left;">Residual</td>
<td style="text-align: right;">96</td>
<td style="text-align: right;">15.964866</td>
<td style="text-align: right;">0.9865897</td>
<td style="text-align: right;">NA</td>
<td style="text-align: right;">NA</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Total</td>
<td style="text-align: right;">98</td>
<td style="text-align: right;">16.181869</td>
<td style="text-align: right;">1.0000000</td>
<td style="text-align: right;">NA</td>
<td style="text-align: right;">NA</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><em><strong>Table 2.</strong> Results of pairwise tests comparing the community composition of plant taxa detected by four observation methodologies: transect surveys of floral diversity and B. pascuorum - flower interactions, and metabarcoding of plant DNA in corbicular pollen and gut contents of B. pascuorum. Methodologies were compared by repeating Permutational Multivariate Analysis of Variance (PERMANOVA, Oksanen et al.&nbsp;2022) for the results of each methodology pair. PERMANOVA tests used the Raup-Crick dissimilarity index with 9999 permutations, and adjusted p-values using the Holm–Bonferroni method. Pairwise tests indicated a significant difference between surveys of floral diversity (“flower count”) and the three other interaction methodologies. No differences were observed between the three interaction observation methodologies.</em></p>
<div class="cell">
<div class="cell-output-display">
<table class="lightable-minimal caption-top table table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th">Methodology 1</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Methodology 2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">DF1</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">DF2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">R²</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">F</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">p</th>
<th style="text-align: left;" data-quarto-table-cell-role="th">Adjusted p</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">flower count</td>
<td style="text-align: left;">gut metabarcoding</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.534</td>
<td style="text-align: right;">161.69</td>
<td style="text-align: left;">&lt;0.001</td>
<td style="text-align: left;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">flower count</td>
<td style="text-align: left;">pollen metabarcoding</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.376</td>
<td style="text-align: right;">64.95</td>
<td style="text-align: left;">&lt;0.001</td>
<td style="text-align: left;">&lt;0.001</td>
</tr>
<tr class="odd">
<td style="text-align: left;">flower count</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.230</td>
<td style="text-align: right;">37.64</td>
<td style="text-align: left;">&lt;0.001</td>
<td style="text-align: left;">&lt;0.001</td>
</tr>
<tr class="even">
<td style="text-align: left;">gut metabarcoding</td>
<td style="text-align: left;">pollen metabarcoding</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.130</td>
<td style="text-align: right;">9.38</td>
<td style="text-align: left;">0.038</td>
<td style="text-align: left;">0.114</td>
</tr>
<tr class="odd">
<td style="text-align: left;">gut metabarcoding</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.010</td>
<td style="text-align: right;">0.80</td>
<td style="text-align: left;">0.55</td>
<td style="text-align: left;">1</td>
</tr>
<tr class="even">
<td style="text-align: left;">pollen metabarcoding</td>
<td style="text-align: left;">interaction</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">1</td>
<td style="text-align: right;">-0.024</td>
<td style="text-align: right;">-1.13</td>
<td style="text-align: left;">0.997</td>
<td style="text-align: left;">1</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 3:</strong> Non-metric dimensional scaling (nMDS) of interaction plant communities for Bombus pascuorum as detected by three interaction observation methodologies and a floral diversity survey. Interaction methodologies included floral visitation surveys and ITS2 metabarcoding of DNA extracted from bumblebee gut contents and corbicular pollen loads. Observations from each methodology are aggregated by sampling day, reduced to binary presence/absence data, and compared in ordination using the Raup-Crick dissimilarity index (ordination stress = 0.17). PERMANOVA comparisons of the communities represented by NMDS show no significant differences between the interaction methodologies. Floral diversity survey results were significantly different (P &lt; 0.001) from each of the interaction methodologies. As visible in the nMDS polygons, interaction transect data were more dispersed&nbsp; (distance to mean = 0.62) than the other methodologies&nbsp; (distance to mean ≤0.10). Significant differences in data dispersal attributed to methodology were indicated by an ANOVA test.</em></p>
<p><em>Specimen level metabarcoding results</em></p>
<p>Taxonomic detections from the two metabarcoding methodologies were compared by individual specimens. For specimens providing both types of samples, fewer taxa were detected in gut content samples (mean = 12 genera, sd = 9) than in pollen samples (mean = 18 genera, sd = 7). Comparing the number of taxa identified by each specimen for both sample sources, 20% of the detected taxa (mean = 6 genera, sd = 3) were shared, on average. A PERMANOVA test of the differences in taxonomic composition between the two methodologies using sample specimen as a blocking factor showed a significant effect (P &lt; 0.01) of metabarcoding sample type on the resulting plant community (Table S3). This effect accounted for 17% of the variation in detected interaction networks between the two methodologies at the individual level.&nbsp;</p>
<p><em>Species Level Interaction Network Metrics</em></p>
<p>We calculated the values of network metrics at the species level for <em>B. pascuorum</em> and the plant taxa within its interaction networks as constructed based on different the three interaction methodologies. <em>B. pascuorum</em> as an interaction network node had the highest degree of 131 within the gut content metabarcoding network, followed by 123 in the corbicular pollen metabarcoding network. <em>B. pascuorum</em> had the lowest degree in the interaction transect network, with a value of 27. <em>B. pascuorum</em> interaction specialization, calculated as <em>d’</em> <span class="citation" data-cites="blüthgen2006">(<a href="#ref-blüthgen2006" role="doc-biblioref">Blüthgen et al., 2006</a>)</span><em>,</em> decreased across the sampling season when calculated based on interaction transect results and pollen metabarcoding results, while specialization maintained more or less stable for gut content metabarcoding results (Figure 4). Interaction transect data indicated complete specialization in the first period. Importance of the plant taxa within the <em>B. pascuorum</em> interaction networks from different methodologies identified <em>Lotus</em> as the most central plant genus in each network, although network importance structure differed between transect and metabarcoding methodologies (Figure 5). The two metabarcoding networks showed more balanced importance across interaction taxa, while the top plant taxa of interaction transect network occupied a greater share of the total observed interactions.</p>
<div class="cell">
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="interaction_methodologies_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><em><strong>Figure 4</strong>: Specialization of plant interactions for B. pascuorum as indicated by networks constructed from three interaction observation methodologies. Specialization was calculated as d’ using the methodology of Blüthgen et al.&nbsp;(2006), with d’ = 1 representing perfect specialist behavior. Specialization of B. pascuorum for each period was calculated relative to interaction data from the other periods.</em></p>
<p><em><strong>Figure 5.</strong> Plant genus “importance” within B. pascuorum interaction networks constructed from three interaction observation methodologies (a) interaction transects, (b) gut content metabarcoding, and (c) corbicular pollen metabarcoding. Importance was calculated by methodology for each plant genus as the proportion of total plant interactions observed by the given methodology represented by interactions with the specific plant genus. Importance is represented in a treemap, with block size proportional to importance, and color scaled to minimum and maximum values observed by each methodology.</em></p>
<p><strong>Discussion</strong></p>
<p>We chose a bumblebee as our model wild pollinator for numerous reasons. On top of the importance of bees in pollen transport relative to other pollinators <span class="citation" data-cites="alarcón2010">(<a href="#ref-alarcón2010" role="doc-biblioref">Alarcón, 2010</a>)</span>, bumblebees specifically offer the advantage of abundance and ease of observation in the wild, as well as relevance due to their common use in agricultural settings <span class="citation" data-cites="velthuis2006">(<a href="#ref-velthuis2006" role="doc-biblioref">Velthuis &amp; Doorn, 2006</a>)</span> and climate dependence <span class="citation" data-cites="sirois-delisle2018">(<a href="#ref-sirois-delisle2018" role="doc-biblioref">Sirois-Delisle &amp; Kerr, 2018</a>)</span> <span class="citation" data-cites="maebe2021">(<a href="#ref-maebe2021" role="doc-biblioref">Maebe et al., 2021</a>)</span>. Given the quantity of existing research on bumblebees and their shared ecology with other pollinators, their environmental and nutritional needs are already broadly defined. Bees generally need a combination of pollen and nectar to meet their nutritional needs <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>. Pollen is the main source of protein and fats in the bee diet <span class="citation" data-cites="vaudo2015">Vaudo (<a href="#ref-vaudo2015" role="doc-biblioref">2015</a>)</span> [<span class="citation" data-cites="steffan2019">Steffan et al. (<a href="#ref-steffan2019" role="doc-biblioref">2019</a>)</span>]<span class="citation" data-cites="dharampal2019">(<a href="#ref-dharampal2019" role="doc-biblioref">Dharampal et al., 2019</a>)</span>, and a diversity of pollen sources in the diet promotes healthy individual physiology <span class="citation" data-cites="dipasquale2016">(<a href="#ref-dipasquale2016" role="doc-biblioref">Di Pasquale et al., 2016</a>)</span>. For this reason it can be generally stated that plant diversity supports bee populations, a notion that is supported by the observed effect of landscape composition on the bee microbiome and its implications for individual health <span class="citation" data-cites="bosmans2018 jones2018">(<a href="#ref-bosmans2018" role="doc-biblioref">Bosmans et al., 2018</a>; <a href="#ref-jones2018" role="doc-biblioref">Jones et al., 2018</a>)</span>. While these relationships indicate that a greater diversity of potential interactions in plant-bee networks promotes bee health, it is also important to understand the implications of different interaction network links for plants and pollinators, network changes over time and space, and the functional roles of network nodes. Our analysis uses the bumblebee, <em>Bombus pascuorum</em> as a model species for investigating how plant-pollinator interaction observation methodologies can provide insight on interaction specifics.</p>
<p>Similar to the results of Quintero et al.&nbsp;(2022)’s evaluation of seed dispersal interaction observations, we found unique and complementary abilities for the three pollinator-perspective based methodologies for observation of <em>B. pascuorum</em> plant interaction networks. We compared each methodology in terms of the diversity of detected interaction nodes (<em>B. pascuorum</em> degree), community composition, and relative sampling effort in network construction. Both metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity (Figure 1). Considering this, and the time dedicated to data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Interaction transects did outperform metabarcoding methodologies in providing taxonomic resolution, as we were able to detect interactions at the species-species level, whereas metabarcoding was only able to provide species-genus level interactions. The interaction networks constructed by each methodology also showed different topology and patterns of specialization for <em>B. pascuorum</em>, exaggerating interaction specialization and the importance of the plant taxa with the highest interaction frequencies within the network (Figure 5). Specialization based on interaction data showed perfect specialist behavior (<em>d’</em> = 1) for the first sampling period, followed by high specialization across the following periods (Figure 4). While a strong link between <em>Bombus pascuorum</em> and certain early season taxa, such as <em>Vicia pyrenaica,</em> is previoulsy documented (Artamendi et al.&nbsp;in prep), perfect specialist behavior was not a plausible result, nor were the magnitude of the <em>d’</em> values likely given the results of the other two methodologies, which estimated far lower specialization and reconstructed more evenly spread network models.</p>
<p>Comparing the two metabarcoding methodologies, gut content metabarcoding outperformed corbicular pollen metabarcoding in terms of taxonomic diversity captured for each sampling period, and over all. The number of gut samples was equal to the number of individuals captured (n = 126), while not all captured individuals carried sufficient pollen for genomic DNA extraction (n = 25). Given this, gut content metabarcoding was the more efficient methodology. On the individual sample level, however, a higher mean number of taxa per sample was detected in pollen (mean =18) samples than in gut samples (mean =12). Therefore, in study designs where number of pollen and gut samples is easier to balance, pollen may be a more efficient sample source for maximizing taxonomic detections. The temporal distribution of taxonomic diversity and the specialization index for <em>B. pascuorum</em> observed by gut content metabarcoding were also the key results in revealing seasonal forage patterns. We observed more forage taxa present in the gut contents than were flowering within our transects in the early and late parts of the sampling season (Figure 2), on either end of the mid-summer floral peak. Additionally we also observed a maintained level of interaction specialization across all of the periods for gut content metabarcoding, while other methodologies indicated a decrease in specialization across the season. These combined results suggested that the composition of the plant community present in the consumption targeted forage network was likely distinct from that of the floral community present within our other surveys.</p>
<p>We observed a complementary dynamic between methodologies through overlap in network community composition. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Between the three interaction methodologies, no statistical differences were found (Table 1), although the data dispersal differences were significant between the interaction observations and metabarcoding methodologies. We interpreted dispersal differences as a result of the differences in spatial and taxonomic coverage offered by each methodology. The habitat and plant community types between each transect vary, while metabarcoding is able to capture all interactions coming from the broader landscape and plant community, resulting in more consistent results between sites and sampling days. Both metabarcoding methodologies overlapped minimally with the flower count community (Figure 3) and showed a statistical difference to the results of this methodology. This suggested that the interaction network included parts of the plant community beyond the floral taxa observed within our transects. This was partially to be expected, given that the metabarcoding methodologies and flower counts had a core difference in purpose. The latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.</p>
<p>Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>)</span> <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay et al., 2022</a>)</span> <span class="citation" data-cites="smart2017">(<a href="#ref-smart2017" role="doc-biblioref">Smart et al., 2017</a>)</span>, metabarcoding increased observed interaction diversity by more than six-fold compared to interaction transect results. Metabarcoding also detected a combined 93 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This, along with the temporal data, suggested that a large number of taxa within the <em>B. pascuorum</em> interaction network came from beyond the spatial extent of our transects and/or outside of the plant community traditionally sampled in plant-pollinator network surveys. Our reference database of sequences included 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences <span class="citation" data-cites="bell2021">(<a href="#ref-bell2021" role="doc-biblioref">K. Bell, 2021</a>)</span>, allowing metabarcoding to identify taxa from functional groups beyond the floral community sampled in our transects (Figure S2, Table S2). Through metabarcoding, we observed interactions with a variety of taxa outside of the entomophilous meadow and shrubland plant community, including trees and shrubs, grasses, and other herbaceous plants. The diversity of plant groups observed within our metabarcoding data indicated that <em>B. pascuorum</em> forages a greater diversity and spatial range than previously expected in order to maintain a stable diversity of consumed pollen, as indicated by the temporal distributions of gut metabarcoding network data.</p>
<p>Our observations of interactions with the anemophilous community are supported by similar previously documented interactions <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>)</span> <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla et al., 2022</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere et al., 2017</a>)</span> <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al., 2020</a>)</span> [<span class="citation" data-cites="terrell1984">Terrell &amp; Batra (<a href="#ref-terrell1984" role="doc-biblioref">1984</a>)</span>]<span class="citation" data-cites="pojar1973">(<a href="#ref-pojar1973" role="doc-biblioref">Pojar, 1973</a>)</span>, and have especially intriguing implications for bumblebee forage behavior. Previous studies using external pollen metabarcoding have removed wind-pollinated taxa from their analyses (e.g.<span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al., 2020</a>)</span> <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon et al., 2017</a>)</span>). This is understandable, given the argument that wind-borne pollen in samples may represent false positive interactions <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon et al., 2017</a>)</span> <span class="citation" data-cites="negri2015">(<a href="#ref-negri2015" role="doc-biblioref">Negri et al., 2015</a>)</span>. Our gut content results, however, suggest that the practice of removing these taxa as contaminants is a large oversight, especially so if using external pollen loads as standalone proxies for forage networks. Our observation of DNA from anemophilous taxa within gut samples suggest that interactions with these taxa may be more than coincidental interactions with wind-borne pollen. Indeed, beyond consumption for adult nutrition, there are previous indications that pollen from flowering trees supports colony establishment success and low larval mortality <span class="citation" data-cites="wood2022">(<a href="#ref-wood2022" role="doc-biblioref">Wood et al., 2022</a>)</span>. Our results support the hypothesis that bumblebees forage selectively for consumption and transport of high quality pollen <span class="citation" data-cites="ruedenauer2016">(<a href="#ref-ruedenauer2016" role="doc-biblioref">Ruedenauer et al., 2016</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>)</span>, adapting their forage to take advantage of the best available resources as they change with environmental variability <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al., 2024</a>)</span>. Our detection of DNA from anemophilous pollen sources across methodologies indicates intentional forage interactions with these taxa as a means of meeting the nutritional needs for bumblebees at various lifecycle stages.</p>
<p>Existing hypotheses for pollinator forage adaptations in response to environmental changes have suggested that bees expand forage diversity beyond the flowering community and across habitats in order to survive annual “hunger gaps” <span class="citation" data-cites="timberlake2024b">(<a href="#ref-timberlake2024b" role="doc-biblioref">Timberlake, Tew, et al., 2024b</a>)</span> <span class="citation" data-cites="becher2024">(<a href="#ref-becher2024" role="doc-biblioref">Becher et al., 2024</a>)</span>, when blooming floral species are limited <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake, Vere, et al., 2024</a>)</span> <span class="citation" data-cites="wood2022">(<a href="#ref-wood2022" role="doc-biblioref">Wood et al., 2022</a>)</span>. Our observation of high forage diversity in gut contents before and after the floral peak, distinct interaction and flowering taxa network topologies, and consumption of taxa across functional groups, all together support these hypotheses. While the community beyond the physical area of our transects likely played a large role in these observations, the detection of anemophilous taxa in gut contents during the periods where forage diversity was higher than flowering diversity provide evidence for a community driven component as well. These observations show how the broader taxonomic detection capacity of metabarcoding allows for detection of interactions that otherwise would go unobserved by flower visitation surveys. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.</p>
<p>Our comparative analysis understates the resolution of our metabarcoding data. We chose to aggregate our metabarcoding data by sampling day while comparing community composition across methodologies in order to balance sampling effort. By doing so, we disregarded the opportunity to study interactions at the individual level offered by metabarcoding <span class="citation" data-cites="gous2019">(<a href="#ref-gous2019" role="doc-biblioref">Gous et al., 2019</a>)</span> <span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">K. L. Bell et al., 2016</a>)</span>. To demonstrate the value of this resolution, we compared the interaction data from individual specimens with both pollen and gut content samples. The low proportion of shared taxa between both sample sources at the individual level indicated a difference in the communities detected within both sample sources that was previously unobserved when methodologies were compared at the sampling day level. To clarify the different indications of these results, we also tested the difference in community composition between both methodologies at the individual level, this time observing a significant difference between the two methodologies.</p>
<p>The differences and commonalities in the communities detected by each methodology may reflect the result of selective interactions made by <em>B. pascuorum</em>. Many of our gut content samples were sourced from queens, which forage exclusively for the inmediate nutrition needed for ovary development <span class="citation" data-cites="leach2018">(<a href="#ref-leach2018" role="doc-biblioref">Leach &amp; Drummond, 2018</a>)</span> <span class="citation" data-cites="vogt1994">(<a href="#ref-vogt1994" role="doc-biblioref">Vogt et al., 1994</a>)</span>. Our pollen samples, on the other hand, would have come from workers foraging both for individual nutrition and for pollen to transport to the nest for larval consumption <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo, 2015</a>)</span>. <strong>In these samples it is possible that there existed more overlap between interaction types.</strong> Focusing on similarities between the two individual level datasets, the repeated detection of certain taxa by both methodologies increased confidence in these identifications, again demonstrating the complementary potential of these methodologies. For example, <em>Vicia</em> was observed by both methodologies across all specimens with both sample types in the first three periods. This adds to our previous knowledge from interaction transects of the strong relationship between <em>Vicia pyrenaica</em> and <em>B. pascuorum</em> (Artamendi et al., unpublished data), emphasizing the value of field surveys in validating laboratory based methodologies.</p>
<p>We observed complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based <span class="citation" data-cites="mantegna2024">(<a href="#ref-mantegna2024" role="doc-biblioref">Mantegna, 2024</a>)</span> <span class="citation" data-cites="nanglu2023">(<a href="#ref-nanglu2023" role="doc-biblioref">Nanglu et al., 2023</a>)</span>, and as molecular tools continue to overcome their limitations <span class="citation" data-cites="yuan2025">(<a href="#ref-yuan2025" role="doc-biblioref">Yuan et al., 2025</a>)</span>. Given the lower sampling efficiency of interaction transects, incorporating them as a validation of others may be the best way to integrate this methodology into future studies. Field observations can fill gaps left by metabarcoding methodologies, such as confirmation of pollination efficacy in interactions, or interaction frequency. Additionally, they provide species level taxonomic resolution which can fill in lower taxonomic levels unreached by bioinformatic processing. As the external pollen load is a direct observation of the pollen that can be transferred between flowers, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. Pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval development. In a similar manner, gut content metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity <span class="citation" data-cites="li2025">(<a href="#ref-li2025" role="doc-biblioref">Li et al., 2025</a>)</span>. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen et al., 2021</a>)</span> <span class="citation" data-cites="keller2021">(<a href="#ref-keller2021" role="doc-biblioref">Keller et al., 2021</a>)</span> , including parasite and disease transfer <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon et al., 2024</a>)</span>. Although they are not equal, our research highlights overall that each methodology offers advantages and disadvantages in terms of sensitivity, sampling effort, and perspective.</p>
<p>Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. <span class="citation" data-cites="magrach2023">(<a href="#ref-magrach2023" role="doc-biblioref">Magrach et al., 2023</a>)</span> <span class="citation" data-cites="devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese et al., 2024</a>)</span>). We present gut content metabarcoding as a promising new methodology for describing plant-pollinator interaction networks, but above all, we advocate for using combinations of methodologies to construct interaction networks with descriptive links. Apart from pairing with other plant-pollinator interaction methodologies, the ability to pair gut content metabarcoding with other microbiome studies is highly intriguing for revealing linked interactions within the plant-pollinator-microbe network. The microbial exchange involved in plant pollinator interactions is an important component of pollinator interactions that is yet to be fully explored <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon et al., 2024</a>)</span> <span class="citation" data-cites="keller2021">(<a href="#ref-keller2021" role="doc-biblioref">Keller et al., 2021</a>)</span> <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen et al., 2021</a>)</span> <span class="citation" data-cites="vannette2020">(<a href="#ref-vannette2020" role="doc-biblioref">Vannette, 2020</a>)</span> . Further application of gut content metabarcoding to plant-pollinator interaction studies will hopefully uncover new cases of synergy with other analyses, as well as further methods of interpreting the results of this methodology. For example, the ability to quantify interaction frequency using metabarcoding at the individual level would be a large advance in the applicability of metabarcoding for gut content or pollen samples. Given our observations, we conclude that the keys to making such methodological advances are likely to lie in finding the connections with similar data sources that can fill the gaps in the information provided by our current methodologies.</p>


<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-alarcón2010" class="csl-entry" role="listitem">
Alarcón, R. (2010). Congruence between visitation and pollen-transport networks in a California plant<span></span>pollinator community. <em>Oikos</em>, <em>119</em>(1), 35–44. <a href="https://doi.org/10.1111/j.1600-0706.2009.17694.x">https://doi.org/10.1111/j.1600-0706.2009.17694.x</a>
</div>
<div id="ref-ankenbrand2015" class="csl-entry" role="listitem">
Ankenbrand, M. J., Keller, A., Wolf, M., Schultz, J., &amp; Förster, F. (2015). ITS2 Database V: Twice as Much: Table 1. <em>Molecular Biology and Evolution</em>, <em>32</em>(11), 3030–3032. <a href="https://doi.org/10.1093/molbev/msv174">https://doi.org/10.1093/molbev/msv174</a>
</div>
<div id="ref-arstingstall2021" class="csl-entry" role="listitem">
Arstingstall, K. A., DeBano, S. J., Li, X., Wooster, D. E., Rowland, M. M., Burrows, S., &amp; Frost, K. (2021). Capabilities and limitations of using DNA metabarcoding to study plant<span></span>pollinator interactions. <em>Molecular Ecology</em>, <em>30</em>(20), 5266–5297. <a href="https://doi.org/10.1111/mec.16112">https://doi.org/10.1111/mec.16112</a>
</div>
<div id="ref-artamendi2025" class="csl-entry" role="listitem">
Artamendi, M., Martin, P. A., Bartomeus, I., &amp; Magrach, A. (2025). Loss of pollinator diversity consistently reduces reproductive success for wild and cultivated plants. <em>Nature Ecology &amp; Evolution</em>, <em>9</em>(2), 296–313. <a href="https://doi.org/10.1038/s41559-024-02595-2">https://doi.org/10.1038/s41559-024-02595-2</a>
</div>
<div id="ref-baksay2022" class="csl-entry" role="listitem">
Baksay, S., Andalo, C., Galop, D., Burrus, M., Escaravage, N., &amp; Pornon, A. (2022). Using metabarcoding to investigate the strength of plant-pollinator interactions from surveys of visits to DNA sequences. <em>Frontiers in Ecology and Evolution</em>, <em>10</em>. <a href="https://doi.org/10.3389/fevo.2022.735588">https://doi.org/10.3389/fevo.2022.735588</a>
</div>
<div id="ref-becher2024" class="csl-entry" role="listitem">
Becher, M. A., Twiston-Davies, G., Osborne, J. L., &amp; Lander, T. A. (2024). Resource gaps pose the greatest threat for bumblebees during the colony establishment phase. <em>Insect Conservation and Diversity</em>, <em>17</em>(4), 676–689. <a href="https://doi.org/10.1111/icad.12736">https://doi.org/10.1111/icad.12736</a>
</div>
<div id="ref-bell2021" class="csl-entry" role="listitem">
Bell, K. (2021). <em>ITS2 july 2021</em>. figshare. <a href="https://doi.org/10.6084/M9.FIGSHARE.14936004.V1">https://doi.org/10.6084/M9.FIGSHARE.14936004.V1</a>
</div>
<div id="ref-bell2017" class="csl-entry" role="listitem">
Bell, K. L., Fowler, J., Burgess, K. S., Dobbs, E. K., Gruenewald, D., Lawley, B., Morozumi, C., &amp; Brosi, B. J. (2017). Applying pollen DNA metabarcoding to the study of plant<span></span>pollinator interactions. <em>Applications in Plant Sciences</em>, <em>5</em>(6), 1600124. <a href="https://doi.org/10.3732/apps.1600124">https://doi.org/10.3732/apps.1600124</a>
</div>
<div id="ref-bell2016" class="csl-entry" role="listitem">
Bell, K. L., Vere, N. de, Keller, A., Richardson, R. T., Gous, A., Burgess, K. S., &amp; Brosi, B. J. (2016). Pollen DNA barcoding: Current applications and future prospects. <em>Genome</em>, <em>59</em>(9), 629–640. <a href="https://doi.org/10.1139/gen-2015-0200">https://doi.org/10.1139/gen-2015-0200</a>
</div>
<div id="ref-blüthgen2006" class="csl-entry" role="listitem">
Blüthgen, N., Menzel, F., &amp; Blüthgen, N. (2006). Measuring specialization in species interaction networks. <em>BMC Ecology</em>, <em>6</em>(1), 9. <a href="https://doi.org/10.1186/1472-6785-6-9">https://doi.org/10.1186/1472-6785-6-9</a>
</div>
<div id="ref-bosch2009" class="csl-entry" role="listitem">
Bosch, J., Martín González, A. M., Rodrigo, A., &amp; Navarro, D. (2009). Plant<span></span>pollinator networks: adding the pollinator<span>’</span>s perspective. <em>Ecology Letters</em>, <em>12</em>(5), 409–419. <a href="https://doi.org/10.1111/j.1461-0248.2009.01296.x">https://doi.org/10.1111/j.1461-0248.2009.01296.x</a>
</div>
<div id="ref-bosmans2018" class="csl-entry" role="listitem">
Bosmans, L., Pozo, M. I., Verreth, C., Crauwels, S., Wilberts, L., Sobhy, I. S., Wäckers, F., Jacquemyn, H., &amp; Lievens, B. (2018). Habitat-specific variation in gut microbial communities and pathogen prevalence in bumblebee queens (Bombus terrestris). <em>PLOS ONE</em>, <em>13</em>(10), e0204612. <a href="https://doi.org/10.1371/journal.pone.0204612">https://doi.org/10.1371/journal.pone.0204612</a>
</div>
<div id="ref-burkle2011" class="csl-entry" role="listitem">
Burkle, L. A., &amp; Alarcón, R. (2011). The future of plant<span></span>pollinator diversity: Understanding interaction networks across time, space, and global change. <em>American Journal of Botany</em>, <em>98</em>(3), 528–538. <a href="https://doi.org/10.3732/ajb.1000391">https://doi.org/10.3732/ajb.1000391</a>
</div>
<div id="ref-callahan2016" class="csl-entry" role="listitem">
Callahan, B. J., McMurdie, P. J., Rosen, M. J., Han, A. W., Johnson, A. J. A., &amp; Holmes, S. P. (2016). DADA2: High resolution sample inference from illumina amplicon data. <em>Nature Methods</em>, <em>13</em>(7), 581–583. <a href="https://doi.org/10.1038/nmeth.3869">https://doi.org/10.1038/nmeth.3869</a>
</div>
<div id="ref-cullen2021" class="csl-entry" role="listitem">
Cullen, N., Fetters, A., &amp; Ashman, T.-L. (2021). Integrating microbes into pollination. <em>Current Opinion in Insect Science</em>, <em>44</em>, 48–54. <a href="https://doi.org/10.1016/j.cois.2020.11.002">https://doi.org/10.1016/j.cois.2020.11.002</a>
</div>
<div id="ref-davis2018" class="csl-entry" role="listitem">
Davis, N. M., Proctor, D. M., Holmes, S. P., Relman, D. A., &amp; Callahan, B. J. (2018). Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data. <em>Microbiome</em>, <em>6</em>(1), 226. <a href="https://doi.org/10.1186/s40168-018-0605-2">https://doi.org/10.1186/s40168-018-0605-2</a>
</div>
<div id="ref-devriese2024" class="csl-entry" role="listitem">
Devriese, A., Peeters, G., Brys, R., &amp; Jacquemyn, H. (2024). The impact of extraction method and pollen concentration on community composition for pollen metabarcoding. <em>Applications in Plant Sciences</em>, <em>12</em>(5), e11601. <a href="https://doi.org/10.1002/aps3.11601">https://doi.org/10.1002/aps3.11601</a>
</div>
<div id="ref-dharampal2019" class="csl-entry" role="listitem">
Dharampal, P. S., Carlson, C., Currie, C. R., &amp; Steffan, S. A. (2019). Pollen-borne microbes shape bee fitness. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>286</em>(1904), 20182894. <a href="https://doi.org/10.1098/rspb.2018.2894">https://doi.org/10.1098/rspb.2018.2894</a>
</div>
<div id="ref-dipasquale2016" class="csl-entry" role="listitem">
Di Pasquale, G., Alaux, C., Conte, Y. L., Odoux, J.-F., Pioz, M., Vaissière, B. E., Belzunces, L. P., &amp; Decourtye, A. (2016). Variations in the Availability of Pollen Resources Affect Honey Bee Health. <em>PLOS ONE</em>, <em>11</em>(9), e0162818. <a href="https://doi.org/10.1371/journal.pone.0162818">https://doi.org/10.1371/journal.pone.0162818</a>
</div>
<div id="ref-bipartite" class="csl-entry" role="listitem">
Dormann, C. F., Fruend, J., Bluethgen, N., &amp; Gruber, B. (2009). <em>Indices, graphs and null models: Analyzing bipartite ecological networks.</em> <em>2</em>, 7–24.
</div>
<div id="ref-evans2020" class="csl-entry" role="listitem">
Evans, D., &amp; Kitson, J. (2020). Molecular ecology as a tool for understanding pollination and other plant<span></span>insect interactions. <em>Current Opinion in Insect Science</em>, <em>38</em>, 26–33. <a href="https://doi.org/10.1016/j.cois.2020.01.005">https://doi.org/10.1016/j.cois.2020.01.005</a>
</div>
<div id="ref-gous2019" class="csl-entry" role="listitem">
Gous, A., Swanevelder, D. Z. H., Eardley, C. D., &amp; Willows-Munro, S. (2019). Plant<span></span>pollinator interactions over time: Pollen metabarcoding from bees in a historic collection. <em>Evolutionary Applications</em>, <em>12</em>(2), 187–197. <a href="https://doi.org/10.1111/eva.12707">https://doi.org/10.1111/eva.12707</a>
</div>
<div id="ref-haag2023" class="csl-entry" role="listitem">
Haag, K. L., Caesar, L., Silveira Regueira-Neto, M. da, Sousa, D. R. de, Montenegro Marcelino, V., Queiroz Balbino, V. de, &amp; Torres Carvalho, A. (2023). Temporal Changes in Gut Microbiota Composition and Pollen Diet Associated with Colony Weakness of a Stingless Bee. <em>Microbial Ecology</em>, <em>85</em>(4), 1514–1526. <a href="https://doi.org/10.1007/s00248-022-02027-3">https://doi.org/10.1007/s00248-022-02027-3</a>
</div>
<div id="ref-ibiyemi2025" class="csl-entry" role="listitem">
Ibiyemi, D., Harris-Shultz, K., Jespersen, D., &amp; Joseph, S. V. (2025). Understanding the Foraging Behavior of Sweat Bees, Bumble Bees, and Honey Bees on Centipedegrass for Conservation Strategies. <em>Journal of Insect Behavior</em>, <em>38</em>(2), 29. <a href="https://doi.org/10.1007/s10905-025-09893-y">https://doi.org/10.1007/s10905-025-09893-y</a>
</div>
<div id="ref-jones2018" class="csl-entry" role="listitem">
Jones, J. C., Fruciano, C., Hildebrand, F., Al Toufalilia, H., Balfour, N. J., Bork, P., Engel, P., Ratnieks, F. L., &amp; Hughes, W. O. (2018). Gut microbiota composition is associated with environmental landscape in honey bees. <em>Ecology and Evolution</em>, <em>8</em>(1), 441–451. <a href="https://doi.org/10.1002/ece3.3597">https://doi.org/10.1002/ece3.3597</a>
</div>
<div id="ref-katumo2022" class="csl-entry" role="listitem">
Katumo, D. M., Liang, H., Ochola, A. C., Lv, M., Wang, Q.-F., &amp; Yang, C.-F. (2022). Pollinator diversity benefits natural and agricultural ecosystems, environmental health, and human welfare. <em>Plant Diversity</em>, <em>44</em>(5), 429–435. <a href="https://doi.org/10.1016/j.pld.2022.01.005">https://doi.org/10.1016/j.pld.2022.01.005</a>
</div>
<div id="ref-keller2021" class="csl-entry" role="listitem">
Keller, A., McFrederick, Q., &amp; Leonhardt, S. (2021). (More than) hitchhikers through the network: The shared microbiome of bees and flowers. <em>Current Opinion in Insect Science</em>, <em>44</em>, 8–15. <a href="https://doi.org/10.1016/j.cois.2020.09.007">https://doi.org/10.1016/j.cois.2020.09.007</a>
</div>
<div id="ref-klein2006" class="csl-entry" role="listitem">
Klein, A.-M., Vaissière, B. E., Cane, J. H., Steffan-Dewenter, I., Cunningham, S. A., Kremen, C., &amp; Tscharntke, T. (2006). Importance of pollinators in changing landscapes for world crops. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>274</em>(1608), 303–313. <a href="https://doi.org/10.1098/rspb.2006.3721">https://doi.org/10.1098/rspb.2006.3721</a>
</div>
<div id="ref-leach2018" class="csl-entry" role="listitem">
Leach, M. E., &amp; Drummond, F. (2018). A Review of Native Wild Bee Nutritional Health. <em>International Journal of Ecology</em>, <em>2018</em>(1), 9607246. <a href="https://doi.org/10.1155/2018/9607246">https://doi.org/10.1155/2018/9607246</a>
</div>
<div id="ref-lecocq2015" class="csl-entry" role="listitem">
Lecocq, T., Brasero, N., Martinet, B., Valterovà, I., &amp; Rasmont, P. (2015). Highly polytypic taxon complex: interspecific and intraspecific integrative taxonomic assessment of the widespread pollinator ombus pascuorum Scopoli 1763 (Hymenoptera: Apidae). <em>Systematic Entomology</em>, <em>40</em>(4), 881–890. <a href="https://doi.org/10.1111/syen.12137">https://doi.org/10.1111/syen.12137</a>
</div>
<div id="ref-leonhardt2012" class="csl-entry" role="listitem">
Leonhardt, S. D., &amp; Blüthgen, N. (2012). The same, but different: pollen foraging in honeybee and bumblebee colonies. <em>Apidologie</em>, <em>43</em>(4), 449–464. <a href="https://doi.org/10.1007/s13592-011-0112-y">https://doi.org/10.1007/s13592-011-0112-y</a>
</div>
<div id="ref-leontidou2021" class="csl-entry" role="listitem">
Leontidou, K., Vokou, D., Sandionigi, A., Bruno, A., Lazarina, M., De Groeve, J., Li, M., Varotto, C., Girardi, M., Casiraghi, M., &amp; Cristofori, A. (2021). Plant biodiversity assessment through pollen DNA metabarcoding in Natura 2000 habitats (Italian Alps). <em>Scientific Reports</em>, <em>11</em>(1), 18226. <a href="https://doi.org/10.1038/s41598-021-97619-3">https://doi.org/10.1038/s41598-021-97619-3</a>
</div>
<div id="ref-leponiemi2023" class="csl-entry" role="listitem">
Leponiemi, M., Freitak, D., Moreno-Torres, M., Pferschy-Wenzig, E.-M., Becker-Scarpitta, A., Tiusanen, M., Vesterinen, E. J., &amp; Wirta, H. (2023). Honeybees<span>’</span> foraging choices for nectar and pollen revealed by DNA metabarcoding. <em>Scientific Reports</em>, <em>13</em>(1), 14753. <a href="https://doi.org/10.1038/s41598-023-42102-4">https://doi.org/10.1038/s41598-023-42102-4</a>
</div>
<div id="ref-li2025" class="csl-entry" role="listitem">
Li, Y., Liu, C., Wang, Y., Li, M., Zou, S., Hu, X., Chen, Z., Li, M., Ma, C., Obi, C. J., Zhou, X., Zou, Y., &amp; Tang, M. (2025). Urban wild bee well-being revealed by gut metagenome data: A mason bee model. <em>Insect Science</em>, <em>n/a</em>(n/a). <a href="https://doi.org/10.1111/1744-7917.70051">https://doi.org/10.1111/1744-7917.70051</a>
</div>
<div id="ref-lignon2024" class="csl-entry" role="listitem">
Lignon, V. A., Mas, F., Jones, E. E., Kaiser, C., &amp; Dhami, M. K. (2024). The floral interface: a playground for interactions between insect pollinators, microbes, and plants. <em>New Zealand Journal of Zoology</em>, 1–20. <a href="https://doi.org/10.1080/03014223.2024.2353285">https://doi.org/10.1080/03014223.2024.2353285</a>
</div>
<div id="ref-lowe2022" class="csl-entry" role="listitem">
Lowe, A., Jones, L., Witter, L., Creer, S., &amp; Vere, N. de. (2022). Using DNA Metabarcoding to Identify Floral Visitation by Pollinators. <em>Diversity</em>, <em>14</em>(4), 236. <a href="https://doi.org/10.3390/d14040236">https://doi.org/10.3390/d14040236</a>
</div>
<div id="ref-maebe2021" class="csl-entry" role="listitem">
Maebe, K., Hart, A. F., Marshall, L., Vandamme, P., Vereecken, N. J., Michez, D., &amp; Smagghe, G. (2021). Bumblebee resilience to climate change, through plastic and adaptive responses. <em>Global Change Biology</em>, <em>27</em>(18), 4223–4237. <a href="https://doi.org/10.1111/gcb.15751">https://doi.org/10.1111/gcb.15751</a>
</div>
<div id="ref-magrach2023" class="csl-entry" role="listitem">
Magrach, A., Artamendi, M., Lapido, P. D., Parejo, C., &amp; Rubio, E. (2023). Indirect interactions between pollinators drive interaction rewiring through space. <em>Ecosphere</em>, <em>14</em>(6), e4521. <a href="https://doi.org/10.1002/ecs2.4521">https://doi.org/10.1002/ecs2.4521</a>
</div>
<div id="ref-mantegna2024" class="csl-entry" role="listitem">
Mantegna, C. (2024). Ecologists: don<span>’</span>t lose touch with the joy of fieldwork. <em>Nature</em>, <em>628</em>(8009), 692–692. <a href="https://doi.org/10.1038/d41586-024-01178-2">https://doi.org/10.1038/d41586-024-01178-2</a>
</div>
<div id="ref-mayr2021" class="csl-entry" role="listitem">
Mayr, A. V., Keller, A., Peters, M. K., Grimmer, G., Krischke, B., Geyer, M., Schmitt, T., &amp; Steffan-Dewenter, I. (2021). Cryptic species and hidden ecological interactions of halictine bees along an elevational gradient. <em>Ecology and Evolution</em>, <em>11</em>(12), 7700–7712. <a href="https://doi.org/10.1002/ece3.7605">https://doi.org/10.1002/ece3.7605</a>
</div>
<div id="ref-milla2022" class="csl-entry" role="listitem">
Milla, L., Schmidt-Lebuhn, A., Bovill, J., &amp; Encinas-Viso, F. (2022). Monitoring of honey bee floral resources with pollen DNA metabarcoding as a complementary tool to vegetation surveys. <em>Ecological Solutions and Evidence</em>, <em>3</em>(1), e12120. <a href="https://doi.org/10.1002/2688-8319.12120">https://doi.org/10.1002/2688-8319.12120</a>
</div>
<div id="ref-nanglu2023" class="csl-entry" role="listitem">
Nanglu, K., de&nbsp;Carle, D., Cullen, T. M., Anderson, E. B., Arif, S., Castañeda, R. A., Chang, L. M., Iwama, R. E., Fellin, E., Manglicmot, R. C., Massey, M. D., &amp; Astudillo-Clavijo, V. (2023). The nature of science: The fundamental role of natural history in ecology, evolution, conservation, and education. <em>Ecology and Evolution</em>, <em>13</em>(10), e10621. <a href="https://doi.org/10.1002/ece3.10621">https://doi.org/10.1002/ece3.10621</a>
</div>
<div id="ref-negri2015" class="csl-entry" role="listitem">
Negri, I., Mavris, C., Prisco, G. D., Caprio, E., &amp; Pellecchia, M. (2015). Honey Bees (Apis mellifera, L.) as Active Samplers of Airborne Particulate Matter. <em>PLOS ONE</em>, <em>10</em>(7), e0132491. <a href="https://doi.org/10.1371/journal.pone.0132491">https://doi.org/10.1371/journal.pone.0132491</a>
</div>
<div id="ref-vegan" class="csl-entry" role="listitem">
Oksanen, J., Simpson, G. L., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R., O’Hara, R. B., Solymos, P., Stevens, M. H. H., Szoecs, E., Wagner, H., Barbour, M., Bedward, M., Bolker, B., Borcard, D., Carvalho, G., Chirico, M., De Caceres, M., Durand, S., … Weedon, J. (2024). <em>Vegan: Community ecology package</em>. <a href="https://CRAN.R-project.org/package=vegan">https://CRAN.R-project.org/package=vegan</a>
</div>
<div id="ref-pashalidou2020" class="csl-entry" role="listitem">
Pashalidou, F. G., Lambert, H., Peybernes, T., Mescher, M. C., &amp; De Moraes, C. M. (2020). Bumble bees damage plant leaves and accelerate flower production when pollen is scarce. <em>Science</em>, <em>368</em>(6493), 881–884. <a href="https://doi.org/10.1126/science.aay0496">https://doi.org/10.1126/science.aay0496</a>
</div>
<div id="ref-pojar1973" class="csl-entry" role="listitem">
Pojar, J. (1973). Pollination of typically anemophilous salt marsh plants by bumble bees, bombus terricola occidentalis grne. <em>The American Midland Naturalist</em>, <em>89</em>(2), 448–451. <a href="https://doi.org/10.2307/2424049">https://doi.org/10.2307/2424049</a>
</div>
<div id="ref-popic2012" class="csl-entry" role="listitem">
Popic, T. J., Wardle, G. M., &amp; Davila, Y. C. (2012). Flower<span>-</span>visitor networks only partially predict the function of pollen transport by bees. <em>Austral Ecology</em>, <em>38</em>(1), 76–86. <a href="https://doi.org/10.1111/j.1442-9993.2012.02377.x">https://doi.org/10.1111/j.1442-9993.2012.02377.x</a>
</div>
<div id="ref-pornon2017" class="csl-entry" role="listitem">
Pornon, A., Andalo, C., Burrus, M., &amp; Escaravage, N. (2017). DNA metabarcoding data unveils invisible pollination networks. <em>Scientific Reports</em>, <em>7</em>(1), 16828. <a href="https://doi.org/10.1038/s41598-017-16785-5">https://doi.org/10.1038/s41598-017-16785-5</a>
</div>
<div id="ref-quintero2022" class="csl-entry" role="listitem">
Quintero, E., Isla, J., &amp; Jordano, P. (2022). Methodological overview and data-merging approaches in the study of plant<span></span>frugivore interactions. <em>Oikos</em>, <em>2022</em>(2). <a href="https://doi.org/10.1111/oik.08379">https://doi.org/10.1111/oik.08379</a>
</div>
<div id="ref-ruedenauer2016" class="csl-entry" role="listitem">
Ruedenauer, F. A., Spaethe, J., &amp; Leonhardt, S. D. (2016). Hungry for quality<span></span>individual bumblebees forage flexibly to collect high-quality pollen. <em>Behavioral Ecology and Sociobiology</em>, <em>70</em>(8), 1209–1217. <a href="https://doi.org/10.1007/s00265-016-2129-8">https://doi.org/10.1007/s00265-016-2129-8</a>
</div>
<div id="ref-saunders2018" class="csl-entry" role="listitem">
Saunders, M. E. (2018). Insect pollinators collect pollen from wind-pollinated plants: implications for pollination ecology and sustainable agriculture. <em>Insect Conservation and Diversity</em>, <em>11</em>(1), 13–31. <a href="https://doi.org/10.1111/icad.12243">https://doi.org/10.1111/icad.12243</a>
</div>
<div id="ref-selva2024" class="csl-entry" role="listitem">
Selva, S., Moretti, M., Ruedenauer, F., Keller, A., Fournier, B., Leonhardt, S. D., Eggenberger, H. A., &amp; Abella, J. C. (2024). <em>Urban bumblebees diversify their foraging strategy to maintain nutrient intake</em>. <a href="https://ecoevorxiv.org/repository/view/7812/">https://ecoevorxiv.org/repository/view/7812/</a>
</div>
<div id="ref-shi2025" class="csl-entry" role="listitem">
Shi, H., Ratering, S., Schneider, B., &amp; Schnell, S. (2025). Microbiome of honey bee corbicular pollen: Factors influencing its structure and potential for studying pathogen transmission. <em>Science of The Total Environment</em>, <em>958</em>, 178107. <a href="https://doi.org/10.1016/j.scitotenv.2024.178107">https://doi.org/10.1016/j.scitotenv.2024.178107</a>
</div>
<div id="ref-sirois-delisle2018" class="csl-entry" role="listitem">
Sirois-Delisle, C., &amp; Kerr, J. T. (2018). Climate change-driven range losses among bumblebee species are poised to accelerate. <em>Scientific Reports</em>, <em>8</em>(1), 14464. <a href="https://doi.org/10.1038/s41598-018-32665-y">https://doi.org/10.1038/s41598-018-32665-y</a>
</div>
<div id="ref-smart2017" class="csl-entry" role="listitem">
Smart, M. D., Cornman, R. S., Iwanowicz, D. D., McDermott-Kubeczko, M., Pettis, J. S., Spivak, M. S., &amp; Otto, C. R. V. (2017). A comparison of honey bee-collected pollen from working agricultural lands using light microscopy and ITS metabarcoding. <em>Environmental Entomology</em>, <em>46</em>(1), 38–49. <a href="https://doi.org/10.1093/ee/nvw159">https://doi.org/10.1093/ee/nvw159</a>
</div>
<div id="ref-steffan2019" class="csl-entry" role="listitem">
Steffan, S. A., Dharampal, P. S., Danforth, B. N., Gaines-Day, H. R., Takizawa, Y., &amp; Chikaraishi, Y. (2019). Omnivory in bees: Elevated trophic positions among all major bee families. <em>The American Naturalist</em>, <em>194</em>(3), 414–421. <a href="https://doi.org/10.1086/704281">https://doi.org/10.1086/704281</a>
</div>
<div id="ref-tanaka2020" class="csl-entry" role="listitem">
Tanaka, K., Nozaki, A., Nakadai, H., Shiwa, Y., &amp; Shimizu-Kadota, M. (2020). Using pollen DNA metabarcoding to profile nectar sources of urban beekeeping in K<span>ō</span>t<span>ō</span>-ku, Tokyo. <em>BMC Research Notes</em>, <em>13</em>(1), 515. <a href="https://doi.org/10.1186/s13104-020-05361-2">https://doi.org/10.1186/s13104-020-05361-2</a>
</div>
<div id="ref-terrell1984" class="csl-entry" role="listitem">
Terrell, E. E., &amp; Batra, S. W. T. (1984). Insects collect pollen of eastern wildrice, zizania aquatica (poaceae). <em>Castanea</em>, <em>49</em>(1), 31–34. <a href="https://www.jstor.org/stable/4033059">https://www.jstor.org/stable/4033059</a>
</div>
<div id="ref-timberlake2024a" class="csl-entry" role="listitem">
Timberlake, T. P., Tew, N. E., &amp; Memmott, J. (2024a). Gardens reduce seasonal hunger gaps for farmland pollinators. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>291</em>(2033). <a href="https://doi.org/10.1098/rspb.2024.1523">https://doi.org/10.1098/rspb.2024.1523</a>
</div>
<div id="ref-timberlake2024b" class="csl-entry" role="listitem">
Timberlake, T. P., Tew, N. E., &amp; Memmott, J. (2024b). Gardens reduce seasonal hunger gaps for farmland pollinators. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>291</em>(2033). <a href="https://doi.org/10.1098/rspb.2024.1523">https://doi.org/10.1098/rspb.2024.1523</a>
</div>
<div id="ref-timberlake2024" class="csl-entry" role="listitem">
Timberlake, T. P., Vere, N. de, Jones, L. E., Vaughan, I. P., Baude, M., &amp; Memmott, J. (2024). Ten-a-day: Bumblebee pollen loads reveal high consistency in foraging breadth among species, sites and seasons. <em>Ecological Solutions and Evidence</em>, <em>5</em>(3), e12360. <a href="https://doi.org/10.1002/2688-8319.12360">https://doi.org/10.1002/2688-8319.12360</a>
</div>
<div id="ref-vanbergen2013" class="csl-entry" role="listitem">
Vanbergen, A. J., &amp; Insect Pollinators Initiative, the. (2013). Threats to an ecosystem service: pressures on pollinators. <em>Frontiers in Ecology and the Environment</em>, <em>11</em>(5), 251–259. <a href="https://doi.org/10.1890/120126">https://doi.org/10.1890/120126</a>
</div>
<div id="ref-vannette2020" class="csl-entry" role="listitem">
Vannette, R. L. (2020). The Floral Microbiome: Plant, Pollinator, and Microbial Perspectives. <em>Annual Review of Ecology, Evolution, and Systematics</em>, <em>51</em>(Volume 51, 2020), 363–386. <a href="https://doi.org/10.1146/annurev-ecolsys-011720-013401">https://doi.org/10.1146/annurev-ecolsys-011720-013401</a>
</div>
<div id="ref-vaudo2015" class="csl-entry" role="listitem">
Vaudo, A. D. (2015). Bee nutrition and floral resource restoration. <em>Current Opinion in Insect Science</em>, <em>10</em>, 133–141. <a href="https://doi.org/10.1016/j.cois.2015.05.008">https://doi.org/10.1016/j.cois.2015.05.008</a>
</div>
<div id="ref-velthuis2006" class="csl-entry" role="listitem">
Velthuis, H. H. W., &amp; Doorn, A. van. (2006). A century of advances in bumblebee domestication and the economic and environmental aspects of its commercialization for pollination. <em>Apidologie</em>, <em>37</em>(4), 421–451. <a href="https://doi.org/10.1051/apido:2006019">https://doi.org/10.1051/apido:2006019</a>
</div>
<div id="ref-devere2017" class="csl-entry" role="listitem">
Vere, N. de, Jones, L. E., Gilmore, T., Moscrop, J., Lowe, A., Smith, D., Hegarty, M. J., Creer, S., &amp; Ford, C. R. (2017). Using DNA metabarcoding to investigate honey bee foraging reveals limited flower use despite high floral availability. <em>Scientific Reports</em>, <em>7</em>(1), 42838. <a href="https://doi.org/10.1038/srep42838">https://doi.org/10.1038/srep42838</a>
</div>
<div id="ref-vogt1994" class="csl-entry" role="listitem">
Vogt, F. D., Heinrich, B., Dabolt, T. O., &amp; McBath, H. L. (1994). Ovary development and colony founding in subarctic and temperate-zone bumblebee queens. <em>Canadian Journal of Zoology</em>, <em>72</em>(9), 1551–1556. <a href="https://doi.org/10.1139/z94-206">https://doi.org/10.1139/z94-206</a>
</div>
<div id="ref-wood2022" class="csl-entry" role="listitem">
Wood, T. J., Vanderplanck, M., Vastrade, M., Vaudo, A. D., &amp; Michez, D. (2022). Trees for bees: could woody plant pollen be used as a consistent resource in bee-focused agri-environment schemes? <em>Entomologia Generalis</em>, <em>42</em>(3), 361. <a href="https://doi.org/10.1127/entomologia/2021/1241">https://doi.org/10.1127/entomologia/2021/1241</a>
</div>
<div id="ref-woodcock2019" class="csl-entry" role="listitem">
Woodcock, B. A., Garratt, M. P. D., Powney, G. D., Shaw, R. F., Osborne, J. L., Soroka, J., Lindström, S. a. M., Stanley, D., Ouvrard, P., Edwards, M. E., Jauker, F., McCracken, M. E., Zou, Y., Potts, S. G., Rundlöf, M., Noriega, J. A., Greenop, A., Smith, H. G., Bommarco, R., … Pywell, R. F. (2019). Meta-analysis reveals that pollinator functional diversity and abundance enhance crop pollination and yield. <em>Nature Communications</em>, <em>10</em>(1), 1481. <a href="https://doi.org/10.1038/s41467-019-09393-6">https://doi.org/10.1038/s41467-019-09393-6</a>
</div>
<div id="ref-yuan2025" class="csl-entry" role="listitem">
Yuan, H., Han, J., Yang, M., Chen, S., &amp; Pang, X. (2025). DNA metabarcoding: Current applications and challenges. <em>Journal of Agricultural and Food Chemistry</em>, <em>73</em>(33), 20616–20632. <a href="https://doi.org/10.1021/acs.jafc.5c06002">https://doi.org/10.1021/acs.jafc.5c06002</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




</body></html>