---
title: "interaction_methodologies"
format: pdf
editor: visual
bibliography: references.bib
---

Bumblebees (Bombus spp.) are excellent model pollinators for numerous reasons. On top of the relative importance of bees in pollen transport relative to other pollinators (Alarcón 2010), bumblebees specifically offer the advantage of abundance and ease of observation in the wild, common use in agricultural settings (Velthius and van Doorn 2006), and climate dependence (Sirois-Delisle and Kerr 2018, Maebe et al. 2021). Given the quantity of existing research on bumblebees, their environmental and nutritional needs are broadly defined. Bees generally need a combination of pollen and nectar to meet their macronutritional needs (Vaudo et al. 2015). Pollen is the main source of protein and fats in the bee diet, and the microbial content of pollen is an important source of nutrition during the larval period (Steffan et al. 2019, Dharampal et al. 2019). A diversity of pollen sources from floral species promotes healthy individual physiology (Di Pasquale et al. 2016), and for this reason it is generally understood that plant diversity supports bee populations. In addition to the biodiversity available to bees within a landscape, landscape use has an effect on the pollinator internal microbiome and individual health (Jones et al. 2018). While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interactions, including which are most important to pollinators, and in what spatiotemporal contexts. 

Our study presents an evaluation of the independent and complementary abilities of three pollinator-perspective based methodologies for observation of *B. pascuorum* and plant interactions, contextualized by a survey of floral resource availability. We compared each methodology in terms of their abilities to capture diversity in interaction taxa and their data outputs in relation to sampling effort. In this comparison metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity (Figure 1) . Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. This advantage was maintained across the sampling season, as interaction networks generally expanded. Interaction transect taxa inherently represented a subset of the taxa observed by flower counts, with cumulative interaction observations only including 23% of the taxa identified across the season by flower counts. Metabarcoding methodologies, however, were not limited to detection of interactions included within the spatial constraint, observation periods, nor the floral diversity of the transects. Both metabarcoding methodologies observed greater total interaction taxa than the floral diversity surveys. During three consecutive sampling periods from May to mid-July, however, more floral diversity was available within the transects than metabarcoding detected in *B. pascuorum* interactions (Figure 2). Finally, comparing the two metabarcoding methodologies, gut metabarcoding outperformed corbicular pollen metabarcoding cumulatively and in each sampling period in terms of taxonomic diversity captured. Comparing the sampling effort of both methodologies, gut metabarcoding also presented an easier methodology, given that the number of samples was equal to the number of individuals captured, while not all captured individuals carried sufficient pollen for genomic DNA extraction. These results suggest that in a raw evaluation of capacity and demanded effort, gut metabarcoding provides the best methodology for detecting interaction networks. However, the seasonal complexities observed between floral resource availability and interaction observations hint that the communities included in these networks are also an important factor in choosing methodologies for future studies.

We also compared community composition as detected by each methodology, analyzing the difference in the plant taxa included in the interaction network of *B. pascuorum* as detected by the each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Community composition analysis found no significant differences between the two metabarcoding methodologies, but both overlapped minimally and showed a statistical difference with the flower count community. This reflected an underlying dynamic of the *B. pascuorum* interaction network structure, first signaled by the differences in taxonomic diversity observed by these methodologies across periods (Figure 2). The core difference between the metabarcoding methodologies and flower counts is that the latter is ultimatley not a methodology for observing interactions, but rather a survey of the some of taxa that could be included in interactions. As with interaction transects, flower counts are also limited in space, time, and taxonomy, while metabarcoding methodologies are unrestricted these boundaries. As a result, the communities detected by metabarcoding were able to include those of the transect methodologies, as well as parts of the Gorbeia plant community beyond the transects, speaking to the complementary nature of the data provided by each of the methodologies together.

Our results suggest a complementary nature between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based, and as molecular tools mature in their reliability (Sources for this argument?).

Pollen metabarcoding is relevant to knowing which species are benefitting from the pollination service offered by pollinators, and in the case of bees, which plant species provide nutrition for developing larvae (Vaudo et al. 2015).

Gut metabarcoding provides an important perspective on the nutritional needs of adult pollinators, showing which taxa are foraged actively for fuel (Li et al. 2025). Also identifies hot taxa for microbiota acquisition, including parasites, given that plants are points of microbial exchange (Lignon et al. 2024, Cullen et al. 2021)

Metabarcoding can detect interactions beyond our transects (spatial constraints of our study) and beyond the floral community (methodological constraints of our study)

Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. [@magrach2023], [@devriese2024]).

As standalone methodologies, each offers advantages and disadvantages in terms of informative capacities, sensitivity, and sampling effort. Interaction transects, fof
