---
title: "Interaction methodologies"
format: html
editor: visual
bibliography: references.bib
csl: methods-in-ecology-and-evolution.csl
---

**Discussion**

Bumblebees (*Bombus spp.*) are excellent model pollinators for numerous reasons. On top of the importance of bees in pollen transport relative to other pollinators [@alarcón2010], bumblebees specifically offer the advantage of abundance and ease of observation in the wild, common use in agricultural settings [@velthuis2006] and climate dependence [@sirois-delisle2018] [@maebe2021]. Given the quantity of existing research on bumblebees and their shared ecology with other pollinators, their environmental and nutritional needs are already broadly defined. Bees generally need a combination of pollen and nectar to meet their nutritional needs [@vaudo2015]. Pollen is the main source of protein and fats in the bee diet [@vaudo2015][@steffan2019][@dharampal2019], and diversity of pollen sources promotes healthy individual physiology [@dipasquale2016]. For this reason it is generally understood that plant diversity supports bee populations, a notion that is supported by the observed effect of landscape composition on the bee microbiome and its implications for individual health [@bosmans2018][@jones2018]. While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interaction networks, including the implications of different interaction tpyes for plants and pollinators, interaction patterns over time and space, and the functions of interaction participants. Our analysis uses the bumblebee, *Bombus pascuorum* as a model species for investigating how plant-pollinator interaction observation methodologies can provide insight on interaction specifics.

We evaluated the independent and complementary abilities of three pollinator-perspective based methodologies for observation of *B. pascuorum* plant interactions, contextualized by surveys of floral resource availability. We compared each methodology in terms of the diversity of interactions and our relative sampling effort in obtaining these results. Both metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity. Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Comparing the two metabarcoding methodologies, gut content metabarcoding outperformed corbicular pollen metabarcoding in terms of taxonomic diversity captured for each sampling period and over all. The number of gut samples in 2023 was equal to the number of individuals captured (n = 126), while not all captured individuals carried sufficient pollen for genomic DNA extraction (n = 25). Given this, gut content metabarcoding was the more efficient methodology. These results suggested that in a raw evaluation of sensitivity and demanded effort, gut content metabarcoding provided the best methodology for detecting interaction networks. Despite this basic comparison, the seasonal complexities observed between floral resource availability and interaction observations indicated that the structure of the communities included in these networks was also an important factor in comparing methodologies.

We also compared community composition as detected by each methodology, analyzing the difference in the plant taxa included in the interaction network of *B. pascuorum* as detected by the each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Community composition analysis found no significant differences between the two metabarcoding methodologies, but both overlapped minimally with the flower count community and showed a statistical difference to the results of this methodology. This reflected an underlying dynamic of the *B. pascuorum* interaction network structure, first signaled by the differences in taxonomic diversity observed by these methodologies across periods (Figure 2). The core difference between the metabarcoding methodologies and flower counts was that the latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.

Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions [@milla2022], [@baksay2022],[@smart2017], metabarcoding increased observed interaction diversity by more than six-fold compared to interaction transect results. Metabarcoding also detected 93 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This suggests that a large number of taxa within the *B. pascuorum* interaction network lie either beyond the spatial extent of our transects or outside of entomophilous community traditionally sampled in plant-pollinator network surveys. Our reference database for metabarcode sequences included 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences [@bell2021], allowing metabarcoding to identify taxa from beyond the community sampled in our transects. Through metabarcoding we observed interactions with a variety of taxa outside of the entomophilous meadow and shrublant plant community. Among these were included trees and shrubs (n = 9), grasses (n =15), and other herbaceous plants (n = 7), indicating a different foraging pattern than was expected for *B. pascuorum.*The diversity of plant types observed within our metabarcoding data indicated that *B. pascuorum* forages a greater diversity and spatial range than previously expected.

Our observations of interactions with the anemophilous community are supported by similar previously documented interactions [@selva2024] [@milla2022] [@timberlake2024] [@devere2017] [@tanaka2020] [@terrell1984][@pojar1973], and have especially intriguing implications in the context of gut content metabarcoding observations. Previous studies using external pollen metabarcoding have removed anemophilous taxa, specifically wind-pollinated taxa from their analyses (e.g.[@tanaka2020], [@pornon2017]). This is understandable, given the arguement that windborne pollen may represent false positive interactions [@pornon2017], [@negri2015]. Our gut content results, however, suggest that studies using this workaround, especially those claiming to use external pollen loads as standalone proxies for forage networks, are vulnerable to a large oversight. Our observation of DNA from anemophilous taxa within gut samples suggest that interactions with these taxa are more than coincidental interactions with windborne pollen. Bumblebees are selective foragers that can assess the nutritional content of pollen, and make forage choices based on these assessments [@leonhardt2012]. Multiple studies have concluded that bumblebees use this selective ability to forage for pollen quality, rather than quantity [@ruedenauer2016],[@timberlake2024], which has been used as an explanation for the adaptability of Bumblebee diets in response to environmental variability [@selva2024].

Our detection of DNA from anemophilous pollen sources indicate intentional forage interactions with these taxa as a means of meeting the nutritional needs for adult bumblebees. **Our observation of taxonomic diversity detected by methodologies over periods may also support this idea, given that we observed more taxa in the gut contents than were present in our transects for the two periods before, and period after the floral peak of the season. Indeed, some of these taxa were the from the anemophilous group (confirm).**These observations could support existing hypotheses for pollinator forage adaptations in response to environmental changes. For example, expanding forage diversity beyond the floral community and across habitats is likely to explain how Bumblebees survive annual “hunger gaps” [@timberlake2024b] [@becher2024], when blooming floral species are limited [@timberlake2024]. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.

In comparing community composition across methodologies, we chose to aggregate our metabarcoding data by sampling day. By doing this, we ensured that sampling effort was balanced across methodologies. However, this understated the resolution of the metabarcoding data. One of the great advantages of applying metabarcoding to pollinator samples is the ability to study interactions at the individual level [@gous2019] [@bell2016]. We compared the interaction data from individual specimens with both between pollen and gut samples. Again, more taxa were detected in gut samples (mean = 20 genera, sd = 10) than in pollen samples (mean = 18 genera, sd = 7). With individual level data, we were able to compare the number of taxa identified for each specimen shared by both sample sources, which was on average only 15% of the detected taxa. While the plant communities detected by both methodologies were not statistically different when aggregated by sampling day, it still appears that both methodologies detect a range of unique taxa. The taxa unique to each methodology may reflect the result of selective interactions made by *B. pascuorum,* choosing to consume pollen from certain flora immediately, or to carry pollen to the colony on the corbiculae for larval consumption [@vaudo2015]. The repeated detection of certain taxa by both methodologies increased confidence in these identifications, again demonstrating the complementary potential of these methodologies. For example, *Vicia* was observed by both methodologies across all specimens with both sample types in the first three periods. This adds to our previous knowledge from interaction transects of the strong relationship between *Vicia pyrenaica* and *B. pascuorum* (Artamendi, unpublished data), demonstrating the value of field surveys in validating laboratory based methodologies.

Our results suggest complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based, and as molecular tools mature in their reliability (maybe could cite [@richardson2015] as a paper suggesting need for growth of molecular techniques, but it's old). Given interaction transects' disadvantage in sampling efficiency, incorporating this methodology as a validation of others may be the best way to integrate this methodology into future studies. Field observations can fill gaps left by metabarcoding methodologies, such as confirmation of pollination efficacy in interactions, or interaction frequency. As the external pollen load is a direct observation of the pollen that can be transferred between flowers, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. Pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval growth, given that external pollen will be returned to the nest to feed developing larvae [@vaudo2015]. In a similar manner, gut content metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity [@li2025]. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions [@cullen2021], including parasite and disease transfer [@lignon2024]. Although they are not equal, our research highlights overall that each methodology offers advantages and disadvantages in terms of sensitivity, sampling effort, and perspective.

Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. [@magrach2023] [@devriese2024]). We present gut content metabarcoding as a promising new methodology for observing plant-pollinator interactions, but we advocate for pairing this methodology with others. Apart from making this pairing with other plant-pollinator interaction methodologies, the ability to pair gut content metabarcoding with other microbiome studies is highly intriguing for revealing linked interactions within the plant-pollinator-microbe network. Apart from the nutritional importance of the microbial content consumed with pollen [@steffan2019][@dharampal2019], the microbial exchange involved in plant pollinator interactions is an important component of pollinator interactions that is yet to be fully explored [@lignon2024] [@cullen2021]. Further application of gut content metabarcoding to plant-pollinator interaction studies will hopefully uncover cases of complementary analysis with other techniques, as well as further methods of interpreting the results of this methodology. For example, the ability to quantify interaction frequency using metabarcoding at the individual level would be a large advance in the applicability of metabarcoing for gut content or pollen samples. Given our observations, we conclude that the keys to making such methodological advances are likely to lie in finding the connections with similar data sources that can fill the gaps in the information provided by our current methodologies.
