---
title: "Interaction methodologies"
format: html
editor: visual
bibliography: references.bib
csl: methods-in-ecology-and-evolution.csl
---

**Introduction**

The global importance and threatened loss of pollination as a critical ecosystem service is clear within the existing research on pollinator communities [@klein2006] [@vanbergen2013]. Our understanding of this topic is incomplete, however, given that historically the methodological approach to studying plant-pollinator interactions has been biased towards the plant side of the interaction [@evans2020, @bosch2009]. Pollination maintains plant community biodiversity, which is essential to ecosystem functioning [@ollerton2017] across global ecosystems and agroecosystems. Pollinators critically support the reproduction of 94% of wild flowering plants and 75% of crop species [@vanbergen2013], contributing to 35% of global food production [@klein2006]. The well established relationship between pollinator diversity and the productivity of plant communities [@artamendi2025] lacks an equally developed mirrored perspective, describing the floral diversity that supports pollinators. Our understanding of plant-pollinator interaction networks is dependent on the perspective from which nodes and links are observed, and the functional outcomes of observed interactions. The current set of methodologies for reconstructing interaction networks overlooks consumer–resource links and interaction outcomes that are critical for understanding how interaction networks support pollinators. Shifting network studies to incorporate the pollinator perspective, and leveraging the contributions of different methodologies is essential to creating a more complete image of interaction networks.

Approaching the study of plant-pollinator interactions from the pollinator perspective provides valuable insights on pollinators' needs. Research based on microscopy and molecular analyses of pollen load samples from different pollinator specimens has shown that this perspective allows for identification of greater plant species diversity within interaction networks compared to studies based solely on field observations [@bosch2009]. Additionally, studies based in the pollinator perspective have revealed greater detail in forage preference trends, such as how different bee species may preferentially forage using either quality or quantity based strategies [@timberlake2024] based on ability to detect pollen quality [@leonhardt2012], life cycle timing, or metabolic specialization [@vaudo2015]. Advances in reliability and accessibility of molecular techniques have made these approaches more feasible for studying plant-pollinator interactions. Field surveys [@milla2022]and pollen grain analyses [@baksay2022, @baksay2020, @bell2016, @smart2017] can now be effectively complemented by genetic tools, enhancing our understanding of these complex interactions.

Genetic tools have a great capacity to detect plant-pollinator interactions that may be unobserved by pollen microscopy and traditional field surveys [@lowe2022, @arstingstall2021, @pornon2017, @bell2016]. Tools such as metabarcoding of pollen samples, whether carried externally by pollinators [@timberlake2024, @selva2024, @devriese2024], or deposited in beehive stores of honey or beebread [@leponiemi2023, @devere2017, @anderson2013] complement the pollination interactions observed by field studies [@arstingstall2021, @bell2017]. The parallel application of pollen metabarcoding techniques to traditional pollination network studies has demonstrated that genetic surveys expand capacity of network studies in species detection by 9 - 144% [@milla2022, @baksay2022, @smart2017], network sampling completeness up to 30% [@arstingstall2021], and revealed interactions beyond the traditionally sampled entomophilous community [@milla2022, @devere2017]. Metabarcoding analyses have further informed knowledge of forage seasonality [@leponiemi2023], specialist vs. generalist behaviour and quality vs. quantity based forage preferences [@timberlake2024, @selva2024]. Beyond inferring plant-pollinator interactions, molecular tools have also been used to uncover the exchange of microbial communities between pollinators, facilitated by flower nectar [@donald2023, @donald2022, @devega2021], revealing transmission pathways for microbes shared by social pollinators [@lignon2024, @hammer2021]. Most of the studies applying metagenomics to pollinator-sourced samples for interaction network construction have used external pollen loads of bees as a sample source [@selva2024, @devriese2024, @leponiemi2023, @baksay2022, [@leontidou2021], despite clear limitations of this approach.

The pollen carried on a pollinator's body relates to limited specific interaction types, and is vulnerable to producing false-positive interaction detections. For bees, corbicular pollen is collected with the intent of transport to the colony nest for storage and larval feeding [@vaudo2015]. Prior studies have used external pollen loads to characterize forage networks and interpret forage preferences without acknowledging that these observations are limited to interactions aimed at pollen transport [@selva2024, @timberlake2024, @leponiemi2023, @tanaka2020]. These interpretations based on the standalone use of external pollen as a sample source are problematic because they overstep the interpretative capacity of the sample source. Because the resources needed for adult pollinator nutrition are different from those needed at the larval stage [@leach2018, @vaudo2015], transported pollen may not completely represent the interactions necessary for maintenance of pollinator diets. This is especially true in for bumblebees (*Bombus spp.*), which are able to discern pollen resource quality and make discerning forage choices [@leonhardt2012, @timberlake2024]. Apart from intentional interactions, bumblebees make many trial-and-error floral visits in order to find adequate forage [@selva2024], which may result in pollen transport without consumption. Finally, externally carried pollen can come from beyond floral interactions, even including accumulation of windborne material [@pornon2017, @negri2015]. Accounting for different interaction types is a critical next step in disentangling the network of floral diversity used by pollinators and understanding the functions of network links.

Pollinator intestinal tracts (“guts”) represent an additional source for observing what pollen from which plant species pollinators consume [@li2025, @haag2023, @mayr2021]. The gut content approach can account for the false positive and negative forage interactions apparent in external pollen analyses, and reveal important implications of interactions that are overlooked by most plant-pollinator network studies, such interactions with anemophilous plants. Although these interactions are intentionally excluded from meteagenomic network studies [@tanaka2020, @pornon2017], here is an accumulating body of literature supporting the idea that pollinators must regularly search across functional groups of the plant community to meet their nutritional needs [@selva2024, @milla2022, @timberlake2024, @devere2017, @tanaka2020, @terrell1984, @pojar1973]. This topic has largely been neglected in the literature on pollinator ecology [@saunders2018]. This understudied component of pollinator forage combined with the surprising lack of genetic analyses on pollinator gut contents together represent a clear knowledge gap and opportunity to reveal more about pollinator forage ecology. 

We want to know how metabarcoding of pollinator gut contents can complement or challenge the characterization of plant-pollinator interaction networks described by other more common methodologies, including field surveys of plant pollinator interactions, and external pollen load metabarcoding. To address this, we compare interaction networks constructed from each of these methodologies as constructed for a single model pollinator, *Bombus pascuorum*. We expect that the feeding interactions detected in gut metabarcoding will include a network of plant taxa distinct from those detected by other methodologies. Ideally, the addition of data from pollinator gut contents will elevate our capacity to detect meaningful plant-pollinator interactions, and learn more about interaction types and implications for pollinator health. Our objective is to determine whether this combined methodological approach can provide further insights into pollinator forage ecology and plant-pollinator interaction networks.

**Methods**

*Study area*

Our data come from fieldwork and sample collection conducted in Gorbeia Natural Park (“Gorbeia”), a protected area in the Basque Autonomous Region of Northern Spain. Gorbeia is a temperate mountainous area that receives more than 150 days of rain, and 1,800 - 2,200 mm total of precipitation annually [@diputaciónforaldebizkaia2023]. Mean temperatures during the study period in 2023 ranged from 10 to 20 °C (Pizarro et al. 2021). Altitude, geomorphology, and varying sun exposure together create a diverse plant community across the sites. This diversity is exaggerated by Gorbeia’s location in the transitional margin of the Mediterranean and Atlantic drainages. As a result of this location, the resulting floral community includes a broad mix of species including taxa from Mediterranean, Basque-Cantabrian, and North-central European communities [@loidi2011].

**Discussion**

Bumblebees (*Bombus spp.*) are excellent model pollinators for numerous reasons. On top of the importance of bees in pollen transport relative to other pollinators [@alarcón2010], bumblebees specifically offer the advantage of abundance and ease of observation in the wild, as well as relevance due to their common use in agricultural settings [@velthuis2006] and climate dependence [@sirois-delisle2018] [@maebe2021]. Given the quantity of existing research on bumblebees and their shared ecology with other pollinators, their environmental and nutritional needs are already broadly defined. Bees generally need a combination of pollen and nectar to meet their nutritional needs [@vaudo2015]. Pollen is the main source of protein and fats in the bee diet [@vaudo2015][@steffan2019][@dharampal2019], and diversity of pollen sources promotes healthy individual physiology [@dipasquale2016]. For this reason it is generally understood that plant diversity supports bee populations, a notion that is supported by the observed effect of landscape composition on the bee microbiome and its implications for individual health [@bosmans2018][@jones2018]. While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interaction networks, including the implications of different interaction types for plants and pollinators, interaction patterns over time and space, and the functions of interaction participants. Our analysis uses the bumblebee, *Bombus pascuorum* as a model species for investigating how plant-pollinator interaction observation methodologies can provide insight on interaction specifics.

We evaluated the independent and complementary abilities of three pollinator-perspective based methodologies for observation of *B. pascuorum* plant interactions, contextualized by surveys of floral resource availability. We compared each methodology in terms of the diversity of interactions and our relative sampling effort in obtaining these results. Both metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity. Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Comparing the two metabarcoding methodologies, gut content metabarcoding outperformed corbicular pollen metabarcoding in terms of taxonomic diversity captured for each sampling period, and over all. The number of gut samples in 2023 was equal to the number of individuals captured (n = 126), while not all captured individuals carried sufficient pollen for genomic DNA extraction (n = 25). Given this, gut content metabarcoding was the more efficient methodology. The temporal distribution of taxonomic diversity observed by gut content metabarcoding was also the key dataset in observing seasonal forage patterns. We observed more forage taxa present in the gut contents than were flowering within our transects in the early and late parts of the sampling season (Figure 2), on either end of the mid-summer floral peak. This suggested that the composition of the plant community present in the forage network was likely distinct from that of the floral community present within our transects.

We also compared interaction network topology as described by each methodology, analyzing the difference in the plant taxa included in the interaction network of *B. pascuorum* as detected by each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Between the three interaction methodologies, no statistical differences were found, although the data dispersal differences were significant between the interaction observations and metabarcoding methodologies. **This was likely a result of..**. Both metabarcoding methodologies overlapped minimally with the flower count community and showed a statistical difference to the results of this methodology. This supported the notion that the interaction network included parts of the plant community beyond the floral taxa observed within our transects. This was partially to be expected, given that the metabarcoding methodologies and flower counts had a core difference in purpose. The latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.

Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions [@milla2022], [@baksay2022],[@smart2017], metabarcoding increased observed interaction diversity by more than six-fold compared to interaction transect results. Metabarcoding also detected 93 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This, along with the temporal data, suggests that a large number of taxa within the *B. pascuorum* interaction network lie either beyond the spatial extent of our transects or outside of the plant community traditionally sampled in plant-pollinator network surveys. Our reference database for metabarcode sequences included 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences [@bell2021], allowing metabarcoding to identify taxa from functional groups beyond the floral community sampled in our transects. Through metabarcoding we observed interactions with a variety of taxa outside of the entomophilous meadow and shrublant plant community. Among these were included trees and shrubs (n = 9), grasses (n =15), and other herbaceous plants (n = 7), indicating a different foraging pattern than was expected for *B. pascuorum.* The diversity of plant types observed within our metabarcoding data indicated that *B. pascuorum* forages a greater diversity and spatial range than previously expected.

Our observations of interactions with the anemophilous community are supported by similar previously documented interactions [@selva2024] [@milla2022] [@timberlake2024] [@devere2017] [@tanaka2020] [@terrell1984][@pojar1973], and have especially intriguing implications in the context of gut content metabarcoding observations. Previous studies using external pollen metabarcoding have removed wind-pollinated taxa from their analyses (e.g.[@tanaka2020], [@pornon2017]). This is understandable, given the argument that wind-borne pollen in samples may represent false positive interactions [@pornon2017], [@negri2015]. Our gut content results, however, suggest that the practice removing these taxa as contaminants can be a large oversight, especially when using external pollen loads as standalone proxies for forage networks. Our observation of DNA from anemophilous taxa within gut samples suggest that interactions with these taxa are more than coincidental interactions with wind-borne pollen. Bumblebees are selective foragers that can assess the nutritional content of pollen, and make forage choices based on these assessments [@leonhardt2012]. Multiple studies have concluded that bumblebees use this selective ability to forage for pollen quality, rather than quantity [@ruedenauer2016],[@timberlake2024], which has been used as an explanation for the adaptability of Bumblebee diets in response to environmental variability [@selva2024]. Our detection of DNA from anemophilous pollen sources indicate intentional forage interactions with these taxa as a means of meeting the nutritional needs for adult bumblebees.

Existing hypotheses for pollinator forage adaptations in response to environmental changes have suggested that bees expand forage diversity beyond the flowering community and across habitats in order to survive annual “hunger gaps” [@timberlake2024b] [@becher2024], when blooming floral species are limited [@timberlake2024]. Our observation of high forage diversity in gut contents before and after the floral peak, distinct interaction and flowering taxa network topologies, and consumption of taxa across functional groups, all together support these hypotheses. While the community beyond the physical area of our transects likely played a large role in these observations, the detection of anemophilous taxa in gut contents during the periods where forage diversity was higher than flowering diversity provide evidence for a community driven component as well. These observations show how the broader taxonomic detection capacity of metabarcoding allows for detection of interactions that otherwise would go unobserved by flower visitation surveys. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.

Our comparative analysis understates the resolution of our metabarcoding data. We chose to aggregate our metabarcoding data by sampling day while comparing community composition across methodologies in order to balance sampling effort. By doing so, we disregarded the opportunity to study interactions at the individual level offered by metabarcoding [@gous2019] [@bell2016]. To demonstrate the value of this resolution, we compared the interaction data from individual specimens with both pollen and gut content samples. The low proportion of shared taxa between both sample sources at the individual level indicated a difference in the communities detected within both sample sources that was previously unobserved when methodologies were compared at the sampling day level. To clarify the different indications of these results, we also tested the difference in community composition between both methodologies at the individual level, this time observing a significant difference between the two methodologies (Table \_ ).

The differences and commonalities in the communities detected by each methodology may reflect the result of selective interactions made by *B. pascuorum*. Bumblebees may forage pollen for immediate consumption, or transport pollen on the corbiculae to the colony nest for larval consumption [@vaudo2015]. Focusing on similarities between the two individual level datasets, the repeated detection of certain taxa by both methodologies increased confidence in these identifications, again demonstrating the complementary potential of these methodologies. For example, *Vicia* was observed by both methodologies across all specimens with both sample types in the first three periods. This adds to our previous knowledge from interaction transects of the strong relationship between *Vicia pyrenaica* and *B. pascuorum* (Artamendi, unpublished data), demonstrating the value of field surveys in validating laboratory based methodologies.

Our results suggest complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based [@mantegna2024] [@nanglu2023], and as molecular tools mature in their reliability [@richardson2015] **NOT the best source for this**. Given interaction transects' disadvantage in sampling efficiency, incorporating this methodology as a validation of others may be the best way to integrate this methodology into future studies. Field observations can fill gaps left by metabarcoding methodologies, such as confirmation of pollination efficacy in interactions, or interaction frequency. As the external pollen load is a direct observation of the pollen that can be transferred between flowers, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. Pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval growth, given that external pollen will be returned to the nest to feed developing larvae [@vaudo2015]. In a similar manner, gut content metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity [@li2025]. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions [@cullen2021], including parasite and disease transfer [@lignon2024]. Although they are not equal, our research highlights overall that each methodology offers advantages and disadvantages in terms of sensitivity, sampling effort, and perspective.

Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. [@magrach2023] [@devriese2024]). We present gut content metabarcoding as a promising new methodology for observing plant-pollinator interactions, but we advocate for pairing this methodology with others. Apart from pairing with other plant-pollinator interaction methodologies, the ability to pair gut content metabarcoding with other microbiome studies is highly intriguing for revealing linked interactions within the plant-pollinator-microbe network. The microbial exchange involved in plant pollinator interactions is an important component of pollinator interactions that is yet to be fully explored [@lignon2024] [@cullen2021]. Further application of gut content metabarcoding to plant-pollinator interaction studies will hopefully uncover new cases of synergy with other analyses, as well as further methods of interpreting the results of this methodology. For example, the ability to quantify interaction frequency using metabarcoding at the individual level would be a large advance in the applicability of metabarcoding for gut content or pollen samples. Given our observations, we conclude that the keys to making such methodological advances are likely to lie in finding the connections with similar data sources that can fill the gaps in the information provided by our current methodologies.
