<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>interaction_methodologies</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="interaction_methodologies_files/libs/clipboard/clipboard.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/quarto.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/popper.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="interaction_methodologies_files/libs/quarto-html/anchor.min.js"></script>
<link href="interaction_methodologies_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="interaction_methodologies_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="interaction_methodologies_files/libs/bootstrap/bootstrap-1bc8a17f135ab3d594c857e9f48e611b.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">interaction_methodologies</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<p><strong>Results</strong></p>
<p><em>Interaction plant diversity</em></p>
<p>Each survey type detected a different number of plant taxa within the B. pascuorum interaction network (Figure 1). Flower count surveys identified a total of 117 flowering plant genera available for interactions within our transects across the sampling season. Interaction transect surveys detected B. pascuorum interactions with 27 genera, all of which were also detected by other methodologies. Gut content metabarcoding detected interactions with 132 total plant genera, including 34 taxa uniquely identified by this methodology. Flower counts detected 48% of the taxa detected by gut metabarcoding, and pollen metabarcoding shared 62% of the taxa detected in gut samples. Corbicular pollen metabarcoding detected 123 plant genera, with 31&nbsp; taxa uniquely detected by this methodology. 56% of of the taxa detected by pollen metabarcoding were represented within the flower count survey taxa, and 67% of the pollen metabarcoding taxa were also identified by gut metabarcoding.&nbsp;</p>
<p><code>{fig.venn}</code></p>
<p>While, by design, flower counts and interaction transects detected only taxa from the flowering entomophilous community, both metabarcoding methodologies detected taxa from the anemophilous community as well. Anemophilous plant taxa represented 18% of the total identified plant taxa between the two methodologies, including mainly grasses, but also several tree genera, many detected across multiple samples.&nbsp;</p>
<p>Taxonomic diversity captured by each methodology across periods revealed an overall increase in interaction taxa over the sampling season for each methodology (Figure 2). Divided by periods, detection patterns followed the general results in that metabarcoding methodologies consistently detected more taxa than interaction transects, with gut metabarcoding detecting more taxa than other interaction methodologies in each period. Flower counts detected more taxa than observed in all other sampling methodologies for three of the six periods.</p>
<p>Bumblebees (Bombus spp.) are excellent model pollinators for numerous reasons. On top of the importance of bees in pollen transport relative to other pollinators <span class="citation" data-cites="alarcón2010">(<a href="#ref-alarcón2010" role="doc-biblioref">Alarcón 2010</a>)</span>, bumblebees specifically offer the advantage of abundance and ease of observation in the wild, common use in agricultural settings <span class="citation" data-cites="velthuis2006">(<a href="#ref-velthuis2006" role="doc-biblioref">Velthuis and Doorn 2006</a>)</span> and climate dependence <span class="citation" data-cites="sirois-delisle2018">(<a href="#ref-sirois-delisle2018" role="doc-biblioref">Sirois-Delisle and Kerr 2018</a>)</span> <span class="citation" data-cites="maebe2021">(<a href="#ref-maebe2021" role="doc-biblioref">Maebe et al. 2021</a>)</span>. Given the quantity of existing research on bumblebees, their environmental and nutritional needs are already broadly defined. Bees generally need a combination of pollen and nectar to meet their nutritional needs <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. Pollen is the main source of protein and fats in the bee diet, and the microbial content of pollen is an important source of nutrition during the larval period <span class="citation" data-cites="steffan2019">(<a href="#ref-steffan2019" role="doc-biblioref">Steffan et al. 2019</a>)</span>, <span class="citation" data-cites="dharampal2019">(<a href="#ref-dharampal2019" role="doc-biblioref">Dharampal et al. 2019</a>)</span>. A diversity of pollen sources from floral species promotes healthy individual physiology <span class="citation" data-cites="dipasquale2016">(<a href="#ref-dipasquale2016" role="doc-biblioref">Di Pasquale et al. 2016</a>)</span> and for this reason it is generally understood that plant diversity supports bee populations. In addition to affecting biodiversity available to bees for forage, landscape use has an effect on the pollinator internal microbiome and individual health <span class="citation" data-cites="jones2018">(<a href="#ref-jones2018" role="doc-biblioref">Jones et al. 2018</a>)</span>. While these relationships indicate that a greater diversity of potential plant-pollinator interactions in the environment promotes bee health, it is also important to understand the finer details of interactions, including which are most important to pollinators, and in what spatiotemporal contexts.&nbsp;</p>
<p>We evaluated the independent and complementary abilities of three pollinator-perspective based methodologies for observation of <em>B. pascuorum</em> and plant interactions, contextualized by surveys of floral resource availability. We compared each methodology in terms of their abilities to capture diversity in interaction taxa and their data outputs in relation to sampling effort. In this comparison, metabarcoding methodologies far outperformed interaction transects in terms of detected taxonomic diversity (Figure 1). This advantage was maintained across the sampling season, as interaction networks generally expanded (Figure 2). Considering this, and the effort involved in data collection for both methodologies, metabarcoding also outperformed interactions in terms of efficiency. Interaction transect taxa inherently represented a subset of the taxa observed by flower counts, with cumulative interaction observations including only 23% of the taxa identified across the season by flower counts. Metabarcoding methodologies, however, were not limited to detection of interactions included within the spatial constraint, observation periods, nor the floral diversity of the transects. Both metabarcoding methodologies observed greater total interaction taxa than the floral diversity surveys. During three consecutive sampling periods from May to mid-July, however, more taxa were present within the floral diversity of the transects than metabarcoding detected in <em>B. pascuorum</em> interactions (Figure 2). Finally, comparing the two metabarcoding methodologies, gut metabarcoding outperformed corbicular pollen metabarcoding cumulatively and in each sampling period in terms of taxonomic diversity captured. Comparing the sampling effort of both methodologies, gut metabarcoding also presented an easier methodology, given that the number of samples was equal to the number of individuals captured, while not all captured individuals carried sufficient pollen for genomic DNA extraction. These results suggested that in a raw evaluation of capacity and demanded effort, gut metabarcoding provided the best methodology for detecting interaction networks. However, the seasonal complexities observed between floral resource availability and interaction observations hint that the communities included in these networks are also an important factor in choosing methodologies for future studies.</p>
<p>We also compared community composition as detected by each methodology, analyzing the difference in the plant taxa included in the interaction network of <em>B. pascuorum</em> as detected by the each methodology per sampling day. We observed the expected ordination overlap (Figure 3) between flower counts and interaction transects inherent in our study design, although statistically the communities of the two methodologies differed (Table 1). We attributed the statistical difference to the large difference in number of taxa observed between the two transect methodologies. Community composition analysis found no significant differences between the two metabarcoding methodologies, but both overlapped minimally with the flower count community and showed a statistical difference to the results of this methodology. This reflected an underlying dynamic of the <em>B. pascuorum</em> interaction network structure, first signaled by the differences in taxonomic diversity observed by these methodologies across periods (Figure 2). The core difference between the metabarcoding methodologies and flower counts was that the latter ultimately did not represent a methodology for observing interactions, but rather a survey of entomophilous taxa that could be included in interactions. As with interaction transects, flower counts were also limited in space, time, and taxonomy, while metabarcoding methodologies were unrestricted by these boundaries.</p>
<p>Our results suggest that metabarcoding has a vast ability to expand our existing understanding of pollinator interaction networks. Consistent with previous comparisons between field and metabarcoding observation of plant-pollinator interactions <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla et al. 2022</a>)</span>, <span class="citation" data-cites="baksay2022">(<a href="#ref-baksay2022" role="doc-biblioref">Baksay et al. 2022</a>)</span>,<span class="citation" data-cites="smart2017">(<a href="#ref-smart2017" role="doc-biblioref">Smart et al. 2017</a>)</span>, metabarcoding increased observed interaction diversity by more than six-fold. Metabarcoding detected 94 total plant taxa that were not included in the 117 identified by the floral diversity surveys detected within our transects. This suggests that a large number of taxa within the <em>B. pascuorum</em> interaction network lie either beyond the spatial extent of our transects or outside of entomophilous community traditionally sampled in plant-pollinator network surveys. Our reference database for metabarcode sequences included all 68,420 ITS2 sequences, including all dicotyledons and select additional plant sequences <span class="citation" data-cites="bell2021">(<a href="#ref-bell2021" role="doc-biblioref">K. Bell 2021</a>)</span>, allowing metabarcoding to identify taxa from beyond the entomophilous community. Our metabarcoding data revealed a variety of anemophilous taxa within the <em>B. pascuorum</em> interaction network, including trees and shrubs (n = 9), grasses (n =15), and other herbaceous plants (n = 7), indicating a greater diversity and spatial range of bumblebee foraging than previously expected.&nbsp;Indeed, our transects did not extend beyond grass and shrub meadows, although other habitat types were adjacent or partially integrated into to the transect areas, including forests and wetlands.</p>
<p>Our observations of interactions with the anemophilous community support other previously documented interaction observations <span class="citation" data-cites="selva2024">(<a href="#ref-selva2024" role="doc-biblioref">Selva et al. 2024</a>)</span> <span class="citation" data-cites="milla2022">(<a href="#ref-milla2022" role="doc-biblioref">Milla et al. 2022</a>)</span> <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake et al. 2024</a>)</span> <span class="citation" data-cites="devere2017">(<a href="#ref-devere2017" role="doc-biblioref">Vere et al. 2017</a>)</span> <span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al. 2020</a>)</span> [<span class="citation" data-cites="terrell1984">Terrell and Batra (<a href="#ref-terrell1984" role="doc-biblioref">1984</a>)</span>]<span class="citation" data-cites="pojar1973">(<a href="#ref-pojar1973" role="doc-biblioref">Pojar 1973</a>)</span>, and have especially intriguing implications in the context of gut metabarcoding. Previous studies using external pollen metabarcoding have removed anemophilous taxa, specifically wind-pollinated taxa from their analyses (e.g.<span class="citation" data-cites="tanaka2020">(<a href="#ref-tanaka2020" role="doc-biblioref">Tanaka et al. 2020</a>)</span>, <span class="citation" data-cites="pornon2017">(<a href="#ref-pornon2017" role="doc-biblioref">Pornon et al. 2017</a>)</span>). It is understandable that in an external pollen metabarcoding context with the objective of characterizing pollination interactions, that these sources of pollen could be overwhelmingly represented in results. Our results suggest that in studies intending to use external pollen loads as proxies for forage networks, choosing to omit these taxa could be a large oversight. Gut metabarcoding data are direct observations of consumed plant material, and our finding of pollen or plant material consumed from anemophilous species within or nearby our transects could support existing hypotheses for pollinator forage adaptations in response to environmental changes. For example, expanding forage diversity beyond the floral community and across habitats is likely to explain how Bumblebees survive annual “hunger gaps” <span class="citation" data-cites="timberlake2024b">(<a href="#ref-timberlake2024b" role="doc-biblioref">Timberlake, Tew, and Memmott 2024</a>)</span> <span class="citation" data-cites="becher2024">(<a href="#ref-becher2024" role="doc-biblioref">Becher et al. 2024</a>)</span> , when blooming floral species are limited <span class="citation" data-cites="timberlake2024">(<a href="#ref-timberlake2024" role="doc-biblioref">Timberlake et al. 2024</a>)</span>. Additionally, our observations are complementary to those that have previously characterized bumblees as selective foragers, using an ability to assess nutritional content of pollen to determine forage choices <span class="citation" data-cites="leonhardt2012">(<a href="#ref-leonhardt2012" role="doc-biblioref">Leonhardt and Blüthgen 2012</a>)</span> based on quality, rather than quantity. As the first study to employ metabarcoding as a means of detecting plant taxa represented in pollinator gut contents, the success of our methodology, especially as compared to alternative existing methodologies, offers a promising direction for future plant-pollinator interactions studies. This is especially evident when considering the resolution of the metabarcoding data presented here, which have been simplified for comparative purposes.</p>
<p>In comparing community composition across methodologies, we chose to aggregate our metabarcoding data by sampling day. By doing this, we ensured that sampling effort was balanced across methodologies. However, this understated the resolution of the metabarcoding data. One of the great advantages of applying metabarcoding to pollinator samples is the ability to study interactions at the individual level [<span class="citation" data-cites="gous2019">Gous et al. (<a href="#ref-gous2019" role="doc-biblioref">2019</a>)</span>]<span class="citation" data-cites="bell2016">(<a href="#ref-bell2016" role="doc-biblioref">K. L. Bell et al. 2016</a>)</span>. We compared the interaction data from individual specimens with both between pollen and gut samples. Again, more taxa were detected in gut samples (mean = 20 genera, sd = 10) than in pollen samples (mean = 18 genera, sd = 7). With individual level data, we were able to compare the number of taxa identified for each specimen shared by both sample sources, which was on average only 15% of the detected taxa. This result indicated that both methodologies could be detecting distinct communities related to the roles of the uniquely detected taxa as sources of nutrition for different purposes. forage for immediate or larval consumption <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. The repeated detection of certain taxa between both methodologies increased confidence in these identifications, again demonstrating the complementary potential of these methodologies. For example, <em>Vicia</em> was observed by both methodologies across all specimens with both sample types in the first three periods. This adds to our previous knowledge from interaction transects of the strong relationship between <em>Vicia pyrenaica</em> and <em>B. pascuorum</em> (Artamendi, unpublished data), demonstrating the value of field surveys in validating laboratory based methodologies.</p>
<p>Our results suggest complementary roles between the interaction methodologies. The statistical similarities between observed plant communities suggests robustness between each of them, and the inherent implications of the sample sources of each provide varied means of interpreting different interactions. Interaction transects provide a valuable field-based perspective, which is integral in ecology, and especially important as it becomes increasingly laboratory based, and as molecular tools mature in their reliability (maybe could cite <span class="citation" data-cites="richardson2015">(<a href="#ref-richardson2015" role="doc-biblioref">Richardson et al. 2015</a>)</span> as a paper suggesting need for growth of molecular techniques, but it’s old). Interaction transects have the previously observed disadvantage in sampling efficiency, suggesting that a reduced approach, focused on validation of other methodologies, may be the best way to integrate this methodology into future studies. As the external pollen load is a direct observation of the pollen that can be transferred between flowers, the taxa identified by pollen metabarcoding represent those most likely to benefit from pollination services. Pollen metabarcoding may also be a good starting point for identifying which plants provide pollen with optimal macronutrients for larval growth, given that external pollen will be returned to the nest to feed developing larvae <span class="citation" data-cites="vaudo2015">(<a href="#ref-vaudo2015" role="doc-biblioref">Vaudo 2015</a>)</span>. In a similar manner, gut metabarcoding provides an important perspective on the nutritional needs of actively foraging pollinators, identifying which taxa provide pollen as food for supporting this activity <span class="citation" data-cites="li2025">(<a href="#ref-li2025" role="doc-biblioref">Li et al. 2025</a>)</span>. Knowing which taxa are actually ingested by pollinators is especially useful information for identifying taxa that facilitate microbiota exchange and acquisition during plant interactions <span class="citation" data-cites="cullen2021">(<a href="#ref-cullen2021" role="doc-biblioref">Cullen, Fetters, and Ashman 2021</a>)</span>, including parasite and disease transfer <span class="citation" data-cites="lignon2024">(<a href="#ref-lignon2024" role="doc-biblioref">Lignon et al. 2024</a>)</span>. Given the different information offered by each methodology,</p>
<p>Each interaction methodology, with the exception of gut content metabarcoding, has been used previously as a standalone means for characterizing plant-pollinator interaction networks (e.g. <span class="citation" data-cites="magrach2023">(<a href="#ref-magrach2023" role="doc-biblioref">Magrach et al. 2023</a>)</span>, <span class="citation" data-cites="devriese2024">(<a href="#ref-devriese2024" role="doc-biblioref">Devriese et al. 2024</a>)</span>). While our results suggest that a single methodology approach may have limitations</p>
<p>As a result, the communities detected by metabarcoding were able to include those of the transect methodologies, as well as parts of the Gorbeia plant community beyond the transects, speaking to the complementary nature of the data provided by each of the methodologies together.</p>
<p>As standalone methodologies, each offers advantages and disadvantages in terms of informative capacities, sensitivity, and sampling effort. Interaction transects, fof</p>


<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-alarcón2010" class="csl-entry" role="listitem">
Alarcón, Ruben. 2010. <span>“Congruence Between Visitation and Pollen-Transport Networks in a California Plant<span></span>pollinator Community.”</span> <em>Oikos</em> 119 (1): 35–44. <a href="https://doi.org/10.1111/j.1600-0706.2009.17694.x">https://doi.org/10.1111/j.1600-0706.2009.17694.x</a>.
</div>
<div id="ref-baksay2022" class="csl-entry" role="listitem">
Baksay, Sandra, Christophe Andalo, Didier Galop, Monique Burrus, Nathalie Escaravage, and André Pornon. 2022. <span>“Using Metabarcoding to Investigate the Strength of Plant-Pollinator Interactions from Surveys of Visits to DNA Sequences.”</span> <em>Frontiers in Ecology and Evolution</em> 10 (March). <a href="https://doi.org/10.3389/fevo.2022.735588">https://doi.org/10.3389/fevo.2022.735588</a>.
</div>
<div id="ref-becher2024" class="csl-entry" role="listitem">
Becher, Matthias A., Grace Twiston-Davies, Juliet L. Osborne, and Tonya A. Lander. 2024. <span>“Resource Gaps Pose the Greatest Threat for Bumblebees During the Colony Establishment Phase.”</span> <em>Insect Conservation and Diversity</em> 17 (4): 676–89. <a href="https://doi.org/10.1111/icad.12736">https://doi.org/10.1111/icad.12736</a>.
</div>
<div id="ref-bell2021" class="csl-entry" role="listitem">
Bell, Karen. 2021. <span>“ITS2 July 2021.”</span> figshare. <a href="https://doi.org/10.6084/M9.FIGSHARE.14936004.V1">https://doi.org/10.6084/M9.FIGSHARE.14936004.V1</a>.
</div>
<div id="ref-bell2016" class="csl-entry" role="listitem">
Bell, Karen L., Natasha de Vere, Alexander Keller, Rodney T. Richardson, Annemarie Gous, Kevin S. Burgess, and Berry J. Brosi. 2016. <span>“Pollen DNA Barcoding: Current Applications and Future Prospects.”</span> <em>Genome</em> 59 (9): 629–40. <a href="https://doi.org/10.1139/gen-2015-0200">https://doi.org/10.1139/gen-2015-0200</a>.
</div>
<div id="ref-cullen2021" class="csl-entry" role="listitem">
Cullen, Nevin, Andrea Fetters, and Tia-Lynn Ashman. 2021. <span>“Integrating Microbes into Pollination.”</span> <em>Current Opinion in Insect Science</em> 44 (April): 48–54. <a href="https://doi.org/10.1016/j.cois.2020.11.002">https://doi.org/10.1016/j.cois.2020.11.002</a>.
</div>
<div id="ref-devriese2024" class="csl-entry" role="listitem">
Devriese, Arne, Gerrit Peeters, Rein Brys, and Hans Jacquemyn. 2024. <span>“The Impact of Extraction Method and Pollen Concentration on Community Composition for Pollen Metabarcoding.”</span> <em>Applications in Plant Sciences</em> 12 (5): e11601. <a href="https://doi.org/10.1002/aps3.11601">https://doi.org/10.1002/aps3.11601</a>.
</div>
<div id="ref-dharampal2019" class="csl-entry" role="listitem">
Dharampal, Prarthana S., Caitlin Carlson, Cameron R. Currie, and Shawn A. Steffan. 2019. <span>“Pollen-Borne Microbes Shape Bee Fitness.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 286 (1904): 20182894. <a href="https://doi.org/10.1098/rspb.2018.2894">https://doi.org/10.1098/rspb.2018.2894</a>.
</div>
<div id="ref-dipasquale2016" class="csl-entry" role="listitem">
Di Pasquale, Garance, Cédric Alaux, Yves Le Conte, Jean-François Odoux, Maryline Pioz, Bernard E. Vaissière, Luc P. Belzunces, and Axel Decourtye. 2016. <span>“Variations in the Availability of Pollen Resources Affect Honey Bee Health.”</span> <em>PLOS ONE</em> 11 (9): e0162818. <a href="https://doi.org/10.1371/journal.pone.0162818">https://doi.org/10.1371/journal.pone.0162818</a>.
</div>
<div id="ref-gous2019" class="csl-entry" role="listitem">
Gous, Annemarie, Dirk Z. H. Swanevelder, Connal D. Eardley, and Sandi Willows-Munro. 2019. <span>“Plant<span></span>pollinator Interactions over Time: Pollen Metabarcoding from Bees in a Historic Collection.”</span> <em>Evolutionary Applications</em> 12 (2): 187–97. <a href="https://doi.org/10.1111/eva.12707">https://doi.org/10.1111/eva.12707</a>.
</div>
<div id="ref-jones2018" class="csl-entry" role="listitem">
Jones, Julia C, Carmelo Fruciano, Falk Hildebrand, Hasan Al Toufalilia, Nicholas J Balfour, Peer Bork, Philipp Engel, Francis LW Ratnieks, and William OH Hughes. 2018. <span>“Gut Microbiota Composition Is Associated with Environmental Landscape in Honey Bees.”</span> <em>Ecology and Evolution</em> 8 (1): 441–51. <a href="https://doi.org/10.1002/ece3.3597">https://doi.org/10.1002/ece3.3597</a>.
</div>
<div id="ref-leonhardt2012" class="csl-entry" role="listitem">
Leonhardt, Sara Diana, and Nico Blüthgen. 2012. <span>“The Same, but Different: Pollen Foraging in Honeybee and Bumblebee Colonies.”</span> <em>Apidologie</em> 43 (4): 449–64. <a href="https://doi.org/10.1007/s13592-011-0112-y">https://doi.org/10.1007/s13592-011-0112-y</a>.
</div>
<div id="ref-li2025" class="csl-entry" role="listitem">
Li, Yiran, Chengweiran Liu, Yiran Wang, Muhan Li, Shasha Zou, Xingyu Hu, Zhiwei Chen, et al. 2025. <span>“Urban Wild Bee Well-Being Revealed by Gut Metagenome Data: A Mason Bee Model.”</span> <em>Insect Science</em> n/a (n/a). <a href="https://doi.org/10.1111/1744-7917.70051">https://doi.org/10.1111/1744-7917.70051</a>.
</div>
<div id="ref-lignon2024" class="csl-entry" role="listitem">
Lignon, V. Aiko, Flore Mas, E. Eirian Jones, Clive Kaiser, and Manpreet K. Dhami. 2024. <span>“The Floral Interface: A Playground for Interactions Between Insect Pollinators, Microbes, and Plants.”</span> <em>New Zealand Journal of Zoology</em>, May, 1–20. <a href="https://doi.org/10.1080/03014223.2024.2353285">https://doi.org/10.1080/03014223.2024.2353285</a>.
</div>
<div id="ref-maebe2021" class="csl-entry" role="listitem">
Maebe, Kevin, Alex F. Hart, Leon Marshall, Peter Vandamme, Nicolas J. Vereecken, Denis Michez, and Guy Smagghe. 2021. <span>“Bumblebee Resilience to Climate Change, Through Plastic and Adaptive Responses.”</span> <em>Global Change Biology</em> 27 (18): 4223–37. <a href="https://doi.org/10.1111/gcb.15751">https://doi.org/10.1111/gcb.15751</a>.
</div>
<div id="ref-magrach2023" class="csl-entry" role="listitem">
Magrach, Ainhoa, Maddi Artamendi, Paula Dominguez Lapido, Clara Parejo, and Encarnacion Rubio. 2023. <span>“Indirect Interactions Between Pollinators Drive Interaction Rewiring Through Space.”</span> <em>Ecosphere</em> 14 (6): e4521. <a href="https://doi.org/10.1002/ecs2.4521">https://doi.org/10.1002/ecs2.4521</a>.
</div>
<div id="ref-milla2022" class="csl-entry" role="listitem">
Milla, Liz, Alexander Schmidt-Lebuhn, Jessica Bovill, and Francisco Encinas-Viso. 2022. <span>“Monitoring of Honey Bee Floral Resources with Pollen DNA Metabarcoding as a Complementary Tool to Vegetation Surveys.”</span> <em>Ecological Solutions and Evidence</em> 3 (1): e12120. <a href="https://doi.org/10.1002/2688-8319.12120">https://doi.org/10.1002/2688-8319.12120</a>.
</div>
<div id="ref-pojar1973" class="csl-entry" role="listitem">
Pojar, Jim. 1973. <span>“Pollination of Typically Anemophilous Salt Marsh Plants by Bumble Bees, Bombus Terricola Occidentalis Grne.”</span> <em>The American Midland Naturalist</em> 89 (2): 448–51. <a href="https://doi.org/10.2307/2424049">https://doi.org/10.2307/2424049</a>.
</div>
<div id="ref-pornon2017" class="csl-entry" role="listitem">
Pornon, André, Christophe Andalo, Monique Burrus, and Nathalie Escaravage. 2017. <span>“DNA Metabarcoding Data Unveils Invisible Pollination Networks.”</span> <em>Scientific Reports</em> 7 (1): 16828. <a href="https://doi.org/10.1038/s41598-017-16785-5">https://doi.org/10.1038/s41598-017-16785-5</a>.
</div>
<div id="ref-richardson2015" class="csl-entry" role="listitem">
Richardson, Rodney T., Chia-Hua Lin, Juan O. Quijia, Natalia S. Riusech, Karen Goodell, and Reed M. Johnson. 2015. <span>“Rank-Based Characterization of Pollen Assemblages Collected by Honey Bees Using a Multi-Locus Metabarcoding Approach.”</span> <em>Applications in Plant Sciences</em> 3 (11): 1500043. <a href="https://doi.org/10.3732/apps.1500043">https://doi.org/10.3732/apps.1500043</a>.
</div>
<div id="ref-selva2024" class="csl-entry" role="listitem">
Selva, Simonetta, Marco Moretti, Fabian Ruedenauer, Alexander Keller, Bertrand Fournier, Sara D. Leonhardt, Helen A. Eggenberger, and Joan Casanelles Abella. 2024. <span>“Urban Bumblebees Diversify Their Foraging Strategy to Maintain Nutrient Intake,”</span> October. <a href="https://ecoevorxiv.org/repository/view/7812/">https://ecoevorxiv.org/repository/view/7812/</a>.
</div>
<div id="ref-sirois-delisle2018" class="csl-entry" role="listitem">
Sirois-Delisle, Catherine, and Jeremy T. Kerr. 2018. <span>“Climate Change-Driven Range Losses Among Bumblebee Species Are Poised to Accelerate.”</span> <em>Scientific Reports</em> 8 (1): 14464. <a href="https://doi.org/10.1038/s41598-018-32665-y">https://doi.org/10.1038/s41598-018-32665-y</a>.
</div>
<div id="ref-smart2017" class="csl-entry" role="listitem">
Smart, M. D., R. S. Cornman, D. D. Iwanowicz, M. McDermott-Kubeczko, J. S. Pettis, M. S. Spivak, and C. R. V. Otto. 2017. <span>“A Comparison of Honey Bee-Collected Pollen from Working Agricultural Lands Using Light Microscopy and ITS Metabarcoding.”</span> <em>Environmental Entomology</em> 46 (1): 38–49. <a href="https://doi.org/10.1093/ee/nvw159">https://doi.org/10.1093/ee/nvw159</a>.
</div>
<div id="ref-steffan2019" class="csl-entry" role="listitem">
Steffan, Shawn A., Prarthana S. Dharampal, Bryan N. Danforth, Hannah R. Gaines-Day, Yuko Takizawa, and Yoshito Chikaraishi. 2019. <span>“Omnivory in Bees: Elevated Trophic Positions Among All Major Bee Families.”</span> <em>The American Naturalist</em> 194 (3): 414–21. <a href="https://doi.org/10.1086/704281">https://doi.org/10.1086/704281</a>.
</div>
<div id="ref-tanaka2020" class="csl-entry" role="listitem">
Tanaka, Keisuke, Akinobu Nozaki, Hazuki Nakadai, Yuh Shiwa, and Mariko Shimizu-Kadota. 2020. <span>“Using Pollen DNA Metabarcoding to Profile Nectar Sources of Urban Beekeeping in K<span>ō</span>t<span>ō</span>-Ku, Tokyo.”</span> <em>BMC Research Notes</em> 13 (1): 515. <a href="https://doi.org/10.1186/s13104-020-05361-2">https://doi.org/10.1186/s13104-020-05361-2</a>.
</div>
<div id="ref-terrell1984" class="csl-entry" role="listitem">
Terrell, Edward E., and Suzanne W. T. Batra. 1984. <span>“Insects Collect Pollen of Eastern Wildrice, Zizania Aquatica (Poaceae).”</span> <em>Castanea</em> 49 (1): 31–34. <a href="https://www.jstor.org/stable/4033059">https://www.jstor.org/stable/4033059</a>.
</div>
<div id="ref-timberlake2024b" class="csl-entry" role="listitem">
Timberlake, T. P., N. E. Tew, and J. Memmott. 2024. <span>“Gardens Reduce Seasonal Hunger Gaps for Farmland Pollinators.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 291 (2033). <a href="https://doi.org/10.1098/rspb.2024.1523">https://doi.org/10.1098/rspb.2024.1523</a>.
</div>
<div id="ref-timberlake2024" class="csl-entry" role="listitem">
Timberlake, T. P., N. de Vere, L. E. Jones, I. P. Vaughan, M. Baude, and J. Memmott. 2024. <span>“Ten-a-Day: Bumblebee Pollen Loads Reveal High Consistency in Foraging Breadth Among Species, Sites and Seasons.”</span> <em>Ecological Solutions and Evidence</em> 5 (3): e12360. <a href="https://doi.org/10.1002/2688-8319.12360">https://doi.org/10.1002/2688-8319.12360</a>.
</div>
<div id="ref-vaudo2015" class="csl-entry" role="listitem">
Vaudo, Anthony D. 2015. <span>“Bee Nutrition and Floral Resource Restoration.”</span> <em>Current Opinion in Insect Science</em> 10 (August): 133–41. <a href="https://doi.org/10.1016/j.cois.2015.05.008">https://doi.org/10.1016/j.cois.2015.05.008</a>.
</div>
<div id="ref-velthuis2006" class="csl-entry" role="listitem">
Velthuis, Hayo H. W., and Adriaan van Doorn. 2006. <span>“A Century of Advances in Bumblebee Domestication and the Economic and Environmental Aspects of Its Commercialization for Pollination.”</span> <em>Apidologie</em> 37 (4): 421–51. <a href="https://doi.org/10.1051/apido:2006019">https://doi.org/10.1051/apido:2006019</a>.
</div>
<div id="ref-devere2017" class="csl-entry" role="listitem">
Vere, Natasha de, Laura E. Jones, Tegan Gilmore, Jake Moscrop, Abigail Lowe, Dan Smith, Matthew J. Hegarty, Simon Creer, and Col R. Ford. 2017. <span>“Using DNA Metabarcoding to Investigate Honey Bee Foraging Reveals Limited Flower Use Despite High Floral Availability.”</span> <em>Scientific Reports</em> 7 (1): 42838. <a href="https://doi.org/10.1038/srep42838">https://doi.org/10.1038/srep42838</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>